<span>**Abstract:**</span> Results from Genome-wide association studies
(GWAS) suggest that a complex phenotype is often affected by many
variants with small effects, known as “polygenicity”. Tens of thousands
of samples are often required to ensure statistical power of identifying
these variants with small effects. However, it is often the case that a
research group can only get approval for the access to individual-level
genotype data with a limited sample size (e.g., a few hundreds or
thousands). Meanwhile, pleiotropy is a pervasive phenomenon in genetics
whereby a DNA variant influences multiple traits, and summary statistics
for genetically related traits (e.g., autoimmune diseases or psychiatric
disorders) are becoming publicly available. The sample sizes associated
with the summary statistics data sets are usually quite large. How to
make the most efficient use of existing abundant data resources largely
remains an open problem.

In this study, we propose a statistical approach, LEP, to increasing
statistical power of identifying risk variants and improving accuracy of
risk prediction by integrating individual level genotype data and
summary statistics by veraging leiotropy. An efficient algorithm based
on variational inference is developed to handle the genome-wide
analysis. Through comprehensive simulation studies, we demonstrated the
advantages of LEP over the methods which take either individual-level
data or summary statistics data as input. We applied LEP to perform
integrative analysis of several auto-immune diseases from WTCCC and
summary statistics from other studies. LEP was able to significantly
increase the statistical power of identifying risk variants and improve
the risk prediction accuracy by jointly analyzing autoimmune diseases.

<span>**Keywords:**</span> GWAS, pleiotropy, polygenicity, summary
statistics, variational inference

<span>**References:**</span>

Solovieff N, Cotsapas C, Lee P H, et al. (2013) Pleiotropy in complex
traits: challenges and strategies In: *Nature reviews. Genetics* 14(7):
483.

Carbonetto P, Stephens M. (2012) Scalable variational inference for
Bayesian variable selection in regression, and its accuracy in genetic
association studies In: *Bayesian analysis* 7(1): 73-108.

Chung D, Yang C, Li C, et al. (2014). GPA: a statistical approach to
prioritizing GWAS results by integrating pleiotropy and annotation In:
*PLoS genetics*

Dai M, Ming J, Cai M, et al. (2017). IGESS: a statistical approach to
integrating individual-level genotype data and summary statistics in
genome-wide association studies. In: *Bioinformatics*
