<span>**Abstract:**</span> Thousands of risk variants underlying complex
phenotypes have been identified in genome-wide association studies
(GWAS). However, there are two major challenges towards fully
characterizing the biological basis of complex diseases. First, many
complex traits are suggested to be highly polygenic, whereas a large
proportion of risk variants with small effects remains unknown. Second,
the functional roles of the majority of GWAS hits in the non-coding
region is largely unclear. In this paper, we propose a latent sparse
mixed model (LSMM) to address the challenges by integrating functional
annotations with summary statistics from GWAS. An efficient variational
expectation-maximization (EM) algorithm is developed. We conducted
comprehensive simulation studies and then applied it to 30 GWAS of
complex phenotypes integrating 9 genic annotation categories and 127
tissue-specific functional annotations from the Roadmap project. The
results demonstrate that LSMM is not only able to increase the
statistical power to identify risk variants, but also provide a deeper
understanding of genetic architecture of complex traits by detecting
relevant functional annotations.

<span>**Keywords:**</span> GWAS,
functional annotations, variational inference
