<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017</title>
  <meta name="description" content="Programme and abstracts for NZSA-IASC-ARS Conference 2017">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="pictures/logo.png" />
  <meta property="og:description" content="Programme and abstracts for NZSA-IASC-ARS Conference 2017" />
  <meta name="github-repo" content="jmcurran/NZSA-IASC-Prog" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017" />
  
  <meta name="twitter:description" content="Programme and abstracts for NZSA-IASC-ARS Conference 2017" />
  <meta name="twitter:image" content="pictures/logo.png" />

<meta name="author" content="Auckland New Zealand">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="programme-and-abstracts-for-monday-11th-of-december.html">
<link rel="next" href="programme-and-abstracts-for-tuesday-12th-of-december.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">New Zealand Statistical Association and the International Association of Statistical Computing (Asian Region) Joint Conference 2017</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html"><i class="fa fa-check"></i>Programme At A Glance</a><ul>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#monday"><i class="fa fa-check"></i>Monday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#tuesday"><i class="fa fa-check"></i>Tuesday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#wednesday"><i class="fa fa-check"></i>Wednesday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#thursday"><i class="fa fa-check"></i>Thursday</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-monday-11th-of-december.html#programme-and-abstracts-for-monday-11th-of-december">Programme And Abstracts For Monday 11<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html"><i class="fa fa-check"></i>Effect of area level deprivation on body mass index: analysis of NZ Health Surveys</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#calendar-based-graphics-for-visualising-peoples-daily-schedules"><i class="fa fa-check"></i>Calendar-based graphics for visualising peopleâ€™s daily schedules</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#nonparametric-test-for-volatility-in-clustered-multiple-time-series-abstract"><i class="fa fa-check"></i>Nonparametric Test for Volatility in Clustered Multiple Time Series Abstract</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#igess-a-statistical-approach-to-integrating-individual-level-genotype-data-and-summary-statistics-in-genome-wide-association-studies"><i class="fa fa-check"></i>IGESS: A Statistical Approach to Integrating Individual Level Genotype Data and Summary Statistics in Genome Wide Association Studies</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#clustering-using-nonparametric-mixtures-and-mode-identification"><i class="fa fa-check"></i>Clustering using Nonparametric Mixtures and Mode Identification</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#bayesian-curve-fitting-for-discontinuous-function-using-overcomplete-representation-with-multiple-kernels"><i class="fa fa-check"></i>Bayesian curve fitting for discontinuous function using overcomplete representation with multiple kernels</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#estimation-of-a-semiparametric-spatiotemporal-models-with-mixed-frequency"><i class="fa fa-check"></i>Estimation of a Semiparametric Spatiotemporal Models with Mixed Frequency</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#lsmm-a-statistical-approach-to-integrating-functional-annotations-with-genome-wide-association-studies"><i class="fa fa-check"></i>LSMM: A statistical approach to integrating functional annotations with genome-wide association studies</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#estimation-of-a-high-dimensional-covariance-matrix"><i class="fa fa-check"></i>Estimation of a High-Dimensional Covariance Matrix</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#innovative-bayesian-estimation-in-the-von-mises-distribution"><i class="fa fa-check"></i>Innovative Bayesian Estimation in the von-Mises Distribution</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#evidence-of-climate-change-from-nonparametric-change-point-analysis"><i class="fa fa-check"></i>Evidence of Climate Change from Nonparametric Change-point Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#joint-analysis-of-individual-level-genotype-data-and-summary-statistics-by-leveraging-pleiotropy"><i class="fa fa-check"></i>Joint Analysis of Individual Level Genotype Data and Summary Statistics by Leveraging Pleiotropy</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#genetic-map-estimation-using-hidden-markov-models-in-the-presence-of-partially-observed-information"><i class="fa fa-check"></i>Genetic map estimation using hidden Markov models in the presence of partially observed information</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#a-simple-method-for-grouping-patients-based-on-historical-doses"><i class="fa fa-check"></i>A simple method for grouping patients based on historical doses</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#semiparametric-mixed-analysis-of-covariance-model"><i class="fa fa-check"></i>Semiparametric Mixed Analysis of Covariance Model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#adaptive-false-discovery-rate-regression-with-application-in-integrative-analysis-of-large-scale-genomic-data"><i class="fa fa-check"></i>Adaptive False Discovery Rate regression with application in integrative analysis of large-scale genomic data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#vector-generalized-linear-time-series-models"><i class="fa fa-check"></i>Vector Generalized Linear Time Series Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#local-canonical-correlation-analysis-for-multimodal-labeled-data"><i class="fa fa-check"></i>Local Canonical Correlation Analysis for Multimodal Labeled Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#canonical-covariance-analysis-for-mixed-numerical-and-categorical-three-way-three-mode-data"><i class="fa fa-check"></i>Canonical Covariance Analysis for Mixed Numerical and Categorical Three-way Three-mode Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#variable-selection-algorithms"><i class="fa fa-check"></i>Variable Selection Algorithms</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#estimating-causal-structures-for-continuous-and-discrete-variables"><i class="fa fa-check"></i>Estimating causal structures for continuous and discrete variables</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#incorporating-genetic-networks-into-case-control-association-studies-with-high-dimensional-dna-methylation-data"><i class="fa fa-check"></i>Incorporating genetic networks into case-control association studies with high-dimensional DNA methylation data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#adaptive-model-checking-for-functional-single-index-models"><i class="fa fa-check"></i>Adaptive Model Checking for Functional Single-Index Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#testing-for-genetic-associations-in-arbitrarily-structured-populations"><i class="fa fa-check"></i>Testing for genetic associations in arbitrarily structured populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#on-optimal-group-testing-designs-prevalence-estimation-cost-considerations-and-dilution-effects"><i class="fa fa-check"></i>On Optimal Group Testing Designs: Prevalence Estimation, Cost Considerations, and Dilution Effects</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#the-use-of-bayesian-networks-in-grape-yield-prediction"><i class="fa fa-check"></i>The Use of Bayesian Networks in Grape Yield Prediction</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="abstract.html"><a href="abstract.html"><i class="fa fa-check"></i><b>1</b> Abstract</a><ul>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#pattern-prediction-for-time-series-data-with-change-points"><i class="fa fa-check"></i>Pattern Prediction for Time Series Data with Change Points</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#test-for-genomic-imprinting-effects-on-the-x-chromosome"><i class="fa fa-check"></i>Test for Genomic Imprinting Effects on the X Chromosome</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#fluctuation-reduction-of-value-at-risk-estimation-and-its-applications"><i class="fa fa-check"></i>Fluctuation Reduction of Value-at-Risk Estimation and its Applications</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#meta-analytic-principal-component-analysis-in-integrative-omics-application"><i class="fa fa-check"></i>Meta-analytic Principal Component Analysis in Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#estimation-of-animal-density-from-acoustic-detections"><i class="fa fa-check"></i>Estimation of animal density from acoustic detections</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#mixed-models-for-complex-survey-data"><i class="fa fa-check"></i>Mixed models for complex survey data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#regression-with-random-effects-for-analysing-correlated-survival-data-application-to-disease-recurrences"><i class="fa fa-check"></i>Regression with random effects for analysing correlated survival data: application to disease recurrences</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#genetic-predictors-underlying-long-term-cognitive-recovery-following-mild-traumatic-brain-injury"><i class="fa fa-check"></i>Genetic predictors underlying long-term cognitive recovery following mild traumatic brain injury</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#bayesian-structure-selection-for-vector-autoregression-model"><i class="fa fa-check"></i>Bayesian Structure Selection for Vector Autoregression Model</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#phylogenetic-tree-based-microbiome-association-test"><i class="fa fa-check"></i>Phylogenetic Tree-based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#talk-data-to-me"><i class="fa fa-check"></i>Talk Data To Me</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#smooth-nonparametric-regression-under-shape-restrictions"><i class="fa fa-check"></i>Smooth Nonparametric Regression under Shape Restrictions</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#elastic-band-transform-a-new-approach-to-multiscale-visualization"><i class="fa fa-check"></i>Elastic-Band Transform: A New Approach to Multiscale Visualization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#a-computational-tool-for-detecting-copy-number-variations-from-whole-genome-and-targeted-exome-sequencing"><i class="fa fa-check"></i>A Computational Tool for Detecting Copy Number Variations from Whole Genome and Targeted Exome Sequencing</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#flight-to-relative-safety-learning-from-a-no-arbitrage-network-of-yield-curves-model-of-the-euro-area"><i class="fa fa-check"></i>Flight to relative safety: Learning from a no-arbitrage network of yield curves model of the euro area</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series"><i class="fa fa-check"></i>Wavelet-based power transformation of non-Gaussian long memory time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#robustness-of-temperature-reconstruction-for-past-500-years"><i class="fa fa-check"></i>Robustness of Temperature Reconstruction for Past 500 Years</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#nonparametric-causal-inference-by-the-kernel-method"><i class="fa fa-check"></i>Nonparametric causal inference by the kernel method</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#a-unified-regularized-group-pls-algorithm-scalable-to-big-data"><i class="fa fa-check"></i>A Unified Regularized Group PLS Algorithm Scalable to Big Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#meta-analysis-with-symbolic-data-analysis-and-its-application-for-clinical-data"><i class="fa fa-check"></i>Meta-analysis with symbolic data analysis and its application for clinical data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#scoring-rules-for-prediction-and-classification-challenges"><i class="fa fa-check"></i>Scoring rules for prediction and classification challenges</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#real-time-transit-network-modelling-for-improved-arrival-time-predictions"><i class="fa fa-check"></i>Real-time transit network modelling for improved arrival time predictions</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#sparse-group-subgroup-partial-least-squares-with-application-to-genomic-data"><i class="fa fa-check"></i>Sparse Group-subgroup Partial Least Squares with Application to Genomic Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#big-sir-a-sliced-inverse-regression-approach-for-massive-data"><i class="fa fa-check"></i>BIG-SIR a Sliced Inverse Regression Approach for Massive Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#comparison-of-tests-of-mean-difference-in-longitudinal-data-based-on-block-resampling-methods"><i class="fa fa-check"></i>Comparison of Tests of Mean Difference in Longitudinal Data Based on Block Resampling Methods</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#modeling-of-document-abstraction-using-association-rule-based-characterization"><i class="fa fa-check"></i>Modeling of Document Abstraction Using Association Rule Based Characterization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#bayesian-survival-analysis-of-batsmen-in-test-cricket"><i class="fa fa-check"></i>Bayesian survival analysis of batsmen in Test cricket</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#covariate-discretisation-on-big-data"><i class="fa fa-check"></i>Covariate Discretisation on Big Data</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#visualization-and-statistical-modeling-of-financial-big-data"><i class="fa fa-check"></i>Visualization and Statistical Modeling of Financial Big Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#analysis-of-spatial-data-with-a-gaussian-mixture-markov-random-field-model"><i class="fa fa-check"></i>Analysis of Spatial Data with a Gaussian Mixture Markov Random Field Model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#my-knee-still-hurts-the-statistical-pathway-to-the-development-of-a-clinical-decision-aid"><i class="fa fa-check"></i>My knee still hurts; the statistical pathway to the development of a clinical decision aid</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#programme-and-abstracts-for-tuesday-12th-of-december">Programme And Abstracts For Tuesday 12<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html"><i class="fa fa-check"></i>R&amp;D policy regimes in France: New evidence from a spatio-temporal analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#analysing-scientific-collaborations-of-new-zealand-institutions-using-scopus-bibliometric-data"><i class="fa fa-check"></i>Analysing Scientific Collaborations of New Zealand Institutions using Scopus Bibliometric Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#a-practitioners-guide-to-deep-learning-for-predictive-analytics-on-structured-data"><i class="fa fa-check"></i>A Practitioners Guide to Deep Learning for Predictive Analytics on Structured Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#analysis-of-multivariate-binary-longitudinal-data-metabolic-syndrome-during-menopausal-transition"><i class="fa fa-check"></i>Analysis of Multivariate Binary Longitudinal Data: Metabolic Syndrome During Menopausal Transition</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#clustering-of-curves-on-a-spatial-domain-using-a-bayesian-partitioning-model"><i class="fa fa-check"></i>Clustering of curves on a spatial domain using a Bayesian partitioning model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#the-uncomfortable-entrepreneurs-bad-working-conditions-and-entrepreneurial-commitment"><i class="fa fa-check"></i>The Uncomfortable Entrepreneurs: Bad Working Conditions and Entrepreneurial Commitment</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#spatial-surveillance-with-scan-statistics-by-controlling-the-false-discovery-rate"><i class="fa fa-check"></i>Spatial Surveillance with Scan Statistics by Controlling the False Discovery Rate</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#statistical-models-for-the-source-attribution-of-zoonotic-diseases-a-study-of-campylobacteriosis"><i class="fa fa-check"></i>Statistical models for the source attribution of zoonotic diseases: A study of campylobacteriosis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#family-structure-and-academic-achievements-of-high-school-students-in-tonga"><i class="fa fa-check"></i>Family structure and academic achievements of high school students in Tonga</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#identifying-clusters-of-patients-with-diabetes-using-a-markov-birth-death-process"><i class="fa fa-check"></i>Identifying Clusters Of Patients With Diabetes Using a Markov Birth-Death Process</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-temporal-density-estimation-using-autoregressive-species-sampling-models"><i class="fa fa-check"></i>Bayesian Temporal Density Estimation using Autoregressive Species Sampling Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#intensity-estimation-of-spatial-point-processes-based-on-area-aggregated-data"><i class="fa fa-check"></i>Intensity Estimation of Spatial Point Processes Based on Area-Aggregated Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-inference-for-population-attributable-measures"><i class="fa fa-check"></i>Bayesian Inference for Population Attributable Measures</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#latent-variable-models-and-multivariate-binomial-data"><i class="fa fa-check"></i>Latent variable models and multivariate binomial data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#analysis-of-a-brief-telephone-intervention-for-problem-gambling-and-examining-the-impact-on-co-existing-depression"><i class="fa fa-check"></i>Analysis of a brief telephone intervention for problem gambling and examining the impact on co-existing depression?</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#prior-based-bayesian-information-criterion"><i class="fa fa-check"></i>Prior-based Bayesian Information Criterion</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#correlated-defaults-with-a-distance-to-default"><i class="fa fa-check"></i>Correlated Defaults with a Distance to Default</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#geographically-weighted-principal-component-analysis-for-spatio-temporal-statistical-dataset"><i class="fa fa-check"></i>Geographically Weighted Principal Component Analysis for Spatio-temporal Statistical Dataset</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#dimensionality-reduction-of-multivariate-data-for-bayesian-analysis"><i class="fa fa-check"></i>Dimensionality Reduction of Multivariate Data for Bayesian Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#an-information-criterion-for-prediction-with-auxiliary-variables-under-covariate-shift"><i class="fa fa-check"></i>An information criterion for prediction with auxiliary variables under covariate shift</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#adjusting-for-linkage-bias-in-the-analysis-of-record-linked-data"><i class="fa fa-check"></i>Adjusting for Linkage Bias in the Analysis of Record-Linked Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-semiparametric-hierarchical-models-for-longitudinal-data-analysis-with-application-to-dose-response-studies"><i class="fa fa-check"></i>Bayesian semiparametric hierarchical models for longitudinal data analysis with application to dose-response studies</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#optimizing-junior-rugby-weight-limits"><i class="fa fa-check"></i>Optimizing Junior Rugby Weight Limits</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#spatial-scan-statistics-for-matched-case-control-data"><i class="fa fa-check"></i>Spatial scan statistics for matched case-control data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#whitebait-in-all-its-varieties-one-fish-two-fish-three-four-five-fish."><i class="fa fa-check"></i>Whitebait in all its varieties: one fish, two fish, three, four, five fish.</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#an-advanced-approach-for-time-series-forecasting-using-deep-learning"><i class="fa fa-check"></i>An Advanced Approach for Time Series Forecasting using Deep Learning</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#asking-about-sex-in-general-health-surveys-comparing-the-methods-and-findings-of-the-2010-health-survey-for-england-with-those-of-the-third-national-survey-of-sexual-attitudes-and-lifestyles"><i class="fa fa-check"></i>Asking about Sex in General Health Surveys: Comparing the Methods and Findings of the 2010 Health Survey for England with Those of the Third National Survey of Sexual Attitudes and Lifestyles</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#tolerance-limits-for-the-reliability-of-semiconductor-devices-using-longitudinal-data"><i class="fa fa-check"></i>Tolerance limits for the reliability of semiconductor devices using longitudinal data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#model-checking-for-regressions-a-local-smoothing-based-global-smoothing-test"><i class="fa fa-check"></i>Model-checking for regressions: a local smoothing-based global smoothing test</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#breeding-value-estimation-in-partially-genotyped-populations"><i class="fa fa-check"></i>Breeding Value Estimation in Partially-Genotyped Populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bivas-a-scalable-bayesian-method-for-bi-level-variable-selection"><i class="fa fa-check"></i>BIVAS: A scalable Bayesian method for Bi-level Variable Selection</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#ranking-potential-shoplifters-in-real-time"><i class="fa fa-check"></i>Ranking Potential Shoplifters in Real Time</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#two-stage-approach-to-data-driven-subgroup-identification-in-clinical-trials"><i class="fa fa-check"></i>Two stage approach to data-driven subgroup identification in clinical trials</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#estimation-and-prediction-for-a-distribution-with-bathtub-shape-under-progressive-first-failure-censoring"><i class="fa fa-check"></i>Estimation and prediction for a distribution with bathtub shape under progressive first failure censoring</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#inverse-regression-for-multivariate-functional-data"><i class="fa fa-check"></i>Inverse Regression for Multivariate Functional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#including-covariate-estimation-error-when-predicting-species-distributions-a-simulation-exercise-using-template-model-builder"><i class="fa fa-check"></i>Including covariate estimation error when predicting species distributions: a simulation exercise using Template Model Builder</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#adjusted-adaptive-index-model-for-binary-response"><i class="fa fa-check"></i>Adjusted adaptive index model for binary response</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#factors-influencing-on-growth-of-garments-industry-in-bangladesh"><i class="fa fa-check"></i>Factors Influencing on Growth of Garments Industry in Bangladesh</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#comparison-of-exact-and-approximate-testing-procedures-in-clinical-trials-with-multiple-binary-endpoints"><i class="fa fa-check"></i>Comparison of Exact and Approximate Testing Procedures in Clinical Trials with Multiple Binary Endpoints</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-optimum-warranty-length-under-type-ii-unified-hybrid-censoring-scheme"><i class="fa fa-check"></i>Bayesian optimum warranty length under Type-II unified hybrid censoring scheme</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#multiple-function-on-function-linear-regression-with-application-to-weather-forecast-calibration"><i class="fa fa-check"></i>Multiple function-on-function linear regression with application to weather forecast calibration</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#modelling-the-distribution-of-lifetime-using-compound-time-homogenous-poisson-process"><i class="fa fa-check"></i>Modelling the distribution of lifetime using compound time-homogenous Poisson process</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#detecting-change-points-in-the-stress-strength-reliability-pxy"><i class="fa fa-check"></i>Detecting Change-Points in the Stress-Strength Reliability P(X&lt;Y)</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#new-zealand-crime-and-victims-survey-filling-the-knowledge-gap"><i class="fa fa-check"></i>New Zealand Crime and Victims Survey: filling the knowledge gap</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#missing-data-in-randomised-control-trials-stepped-multiple-imputation"><i class="fa fa-check"></i>Missing Data in Randomised Control Trials: Stepped Multiple Imputation</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#programme-and-abstracts-for-wednesday-13th-of-december">Programme And Abstracts For Wednesday 13<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html"><i class="fa fa-check"></i>Promoting your R package</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-smoothing-filter-modelling-approach-for-time-series"><i class="fa fa-check"></i>A smoothing filter modelling approach for time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#online-learning-for-bayesian-nonparametrics-weakly-conjugate-approximation"><i class="fa fa-check"></i>Online Learning for Bayesian Nonparametrics: Weakly Conjugate Approximation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#improving-the-production-cycle-at-stats-nz-with-rstudio"><i class="fa fa-check"></i>Improving the production cycle at Stats NZ with RStudio</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#sparse-common-component-analysis"><i class="fa fa-check"></i>Sparse Common Component Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#random-search-global-optimization-using-random-forests"><i class="fa fa-check"></i>Random Search Global Optimization using Random Forests</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#gridsvg-then-and-now"><i class="fa fa-check"></i>gridSVG: Then and Now</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#probabilistic-outlier-detection-and-visualization-of-smart-meter-data"><i class="fa fa-check"></i>Probabilistic outlier detection and visualization of smart meter data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#the-joint-models-for-nonlinear-longitudinal-and-time-to-event-data-using-penalized-splines-a-bayesian-approach"><i class="fa fa-check"></i>The joint models for nonlinear longitudinal and time-to-event data using penalized splines: A Bayesian approach</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-a-powerful-analysis-tool-to-improve-official-statistics-in-romania"><i class="fa fa-check"></i>R â€“ A powerful analysis tool to improve Official Statistics in Romania</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#robust-principal-expectile-component-analysis"><i class="fa fa-check"></i>Robust Principal Expectile Component Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bringing-multimix-from-fortran-to-r"><i class="fa fa-check"></i>Bringing Multimix from Fortran to R</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#specification-of-garch-model-under-asymmetric-error-innovations"><i class="fa fa-check"></i>SPECIFICATION OF GARCH MODEL UNDER ASYMMETRIC ERROR INNOVATIONS</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#the-joint-models-for-nonlinear-longitudinal-and-time-to-event-data-using-penalized-splines-a-bayesian-approach-1"><i class="fa fa-check"></i>The joint models for nonlinear longitudinal and time-to-event data using penalized splines: A Bayesian approach</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#statistical-disclosure-control-with-r-traditional-methods-and-synthetic-data"><i class="fa fa-check"></i>Statistical Disclosure Control with R: Traditional Methods and Synthetic Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-max-type-multivariate-two-sample-baumgartner-statistic"><i class="fa fa-check"></i>A Max-Type Multivariate Two-Sample Baumgartner Statistic</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#deep-learning-high-dimensional-covariance-matrices"><i class="fa fa-check"></i>Deep Learning High-Dimensional Covariance Matrices</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-in-industry-application-on-pipe-renewal-planning"><i class="fa fa-check"></i>R in Industry â€“ Application on Pipe Renewal Planning</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#empirical-comparison-of-some-algorithms-for-automatic-univariate-arma-modeling-using-rcmdrplugin.spss"><i class="fa fa-check"></i>Empirical comparison of some algorithms for automatic univariate ARMA modeling using RcmdrPlugin.SPSS</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bayesian-analyses-of-non-homogeneous-gaussian-hidden-markov-models"><i class="fa fa-check"></i>Bayesian Analyses of Non-homogeneous Gaussian Hidden Markov Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#imputation-of-the-2016-economic-census-for-business-activity-in-japan"><i class="fa fa-check"></i>Imputation of the 2016 Economic Census for Business Activity in Japan</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#simultaneous-test-for-mean-vectors-and-covariance-matrices-in-high-dimensional-settings"><i class="fa fa-check"></i>Simultaneous test for mean vectors and covariance matrices in high-dimensional settings</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#applying-active-learning-procedure-to-drug-consumption-data"><i class="fa fa-check"></i>Applying Active Learning Procedure to Drug Consumption Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-for-everything"><i class="fa fa-check"></i>R for Everything</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-package-for-new-two-stage-methods-in-forecasting-time-series-with-multiple-seasonality"><i class="fa fa-check"></i>R Package for New Two-Stage Methods in Forecasting Time Series with Multiple Seasonality</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#performance-of-bayesian-credible-interval-for-binomial-proportion-using-logit-transformation"><i class="fa fa-check"></i>Performance of Bayesian Credible Interval for Binomial Proportion using Logit Transformation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#analysis-of-official-microdata-using-secure-statistical-computation-system"><i class="fa fa-check"></i>Analysis of Official Microdata Using Secure Statistical Computation System</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#high-dimensional-asymptotics-for-the-naive-canonical-correlation-coefficient"><i class="fa fa-check"></i>High Dimensional Asymptotics for the Naive Canonical Correlation Coefficient</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#presenting-flexi-a-statistical-program-for-fitting-variance-models"><i class="fa fa-check"></i>Presenting Flexi, A Statistical Program for Fitting Variance Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#cluster-wise-regression-models-combined-by-a-quasi-linear-function"><i class="fa fa-check"></i>Cluster-wise regression models combined by a quasi-linear function</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#hierarchical-structural-component-analysis-of-gene-environment-interactions"><i class="fa fa-check"></i>Hierarchical Structural Component Analysis of Gene-Environment Interactions</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series-1"><i class="fa fa-check"></i>Wavelet-based power transformation of non-Gaussian long memory time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#cross-covariance-estimation-for-integration-of-multi-omics-data"><i class="fa fa-check"></i>Cross covariance estimation for integration of multi-omics data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#relationships-between-linguistic-characteristics-and-the-use-of-maori-loanwords-in-new-zealand-english."><i class="fa fa-check"></i>Relationships between linguistic characteristics and the use of MÄori loanwords in New Zealand English.</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#transfer-regression-and-predictive-distributions"><i class="fa fa-check"></i>Transfer Regression and Predictive Distributions</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#fluctuation-reduction-of-value-at-risk-estimation-and-its-applications-1"><i class="fa fa-check"></i>Fluctuation Reduction of Value-at-Risk Estimation and its Applications</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#testing-for-genetic-associations-in-arbitrarily-structured-populations-1"><i class="fa fa-check"></i>Testing for genetic associations in arbitrarily structured populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bayesian-analysis-for-fitting-zero-inflated-count-data-with-data-augmentation"><i class="fa fa-check"></i>Bayesian analysis for fitting zero-inflated count data with data augmentation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#regularized-noise-reduction-methodology-for-high-dimensional-data"><i class="fa fa-check"></i>Regularized Noise-Reduction Methodology for High-Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#computation-of-influence-functions-for-robust-statistics"><i class="fa fa-check"></i>Computation of influence functions for robust statistics</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#adaptive-model-averaging-in-high-dimensional-linear-regression"><i class="fa fa-check"></i>Adaptive Model Averaging in High-Dimensional Linear Regression</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#meta-analytic-principal-component-analysis-in-integrative-omics-application-1"><i class="fa fa-check"></i>Meta-analytic Principal Component Analysis in Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#model-based-clustering-for-multivariate-categorical-data-with-dimension-reduction"><i class="fa fa-check"></i>Model-based clustering for multivariate categorical data with dimension reduction</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#selecting-the-number-of-principal-components"><i class="fa fa-check"></i>Selecting the number of principal components</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#rolling-survival-extrapolation-algorithm-for-estimating-life-years-lost-among-subjects-exposed-to-long-term-air-pollution"><i class="fa fa-check"></i>Rolling survival extrapolation algorithm for estimating life years lost among subjects exposed to long-term air pollution</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#enhancing-the-flexibility-of-regression-modeling-by-liquid-association"><i class="fa fa-check"></i>Enhancing the flexibility of regression modeling by liquid association</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#flight-to-relative-safety-learning-from-a-no-arbitrage-network-of-yield-curves-model-of-the-euro-area-1"><i class="fa fa-check"></i>Flight to relative safety: Learning from a no-arbitrage network of yield curves model of the euro area</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#phylogenetic-tree-based-microbiome-association-test-1"><i class="fa fa-check"></i>Phylogenetic Tree-based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#towards-a-sparse-scalable-and-stably-positive-definite-inverse-covariance-estimator"><i class="fa fa-check"></i>Towards a sparse, scalable, and stably positive definite (inverse) covariance estimator</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#high-mortality-predictions-with-lines-or-curves-fitted-to-over-dispersed-exposure-mortality-data"><i class="fa fa-check"></i>High Mortality Predictions with Lines or Curves Fitted to Over-Dispersed Exposure-Mortality Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#lattice-polytope-samplers"><i class="fa fa-check"></i>Lattice Polytope Samplers</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#statistical-modelling-and-analysis-of-cosmic-microwave-background-data"><i class="fa fa-check"></i>Statistical Modelling and Analysis of Cosmic Microwave Background Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#measure-of-departure-from-marginal-average-point-symmetry-for-two-way-contingency-tables-with-ordered-categories"><i class="fa fa-check"></i>Measure of Departure from Marginal Average Point-Symmetry for Two-Way Contingency Tables with Ordered Categories</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#sparse-estimates-from-dense-precision-matrix-posteriors"><i class="fa fa-check"></i>Sparse estimates from dense precision matrix posteriors</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#dimension-reduction-strategies-for-modeling-bi-clustered-high-dimensional-data"><i class="fa fa-check"></i>Dimension Reduction Strategies for Modeling Bi-clustered High Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#computing-entropies-with-nested-sampling"><i class="fa fa-check"></i>Computing Entropies with Nested Sampling</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#spline-based-drift-models-for-high-temperature-operating-life-tests"><i class="fa fa-check"></i>Spline-based Drift Models for High Temperature Operating Life Tests</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-new-approach-to-distribution-free-tests-in-contingency-tables"><i class="fa fa-check"></i>A new approach to distribution free tests in contingency tables</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bayesian-continuous-space-time-model-of-burglaries"><i class="fa fa-check"></i>Bayesian continuous space-time model of Burglaries</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#modified-gene-shaving-algorithm---a-dimension-reduction-and-clustering-method"><i class="fa fa-check"></i>Modified Gene Shaving Algorithm - A Dimension REduction and Clustering Method</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-simple-method-to-construct-confidence-bands-in-functional-linear-regression"><i class="fa fa-check"></i>A simple method to construct confidence bands in functional linear regression</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#separation-of-symmetry-for-square-contingency-tables-with-ordinal-categories"><i class="fa fa-check"></i>Separation of symmetry for square contingency tables with ordinal categories</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#an-overview-of-the-correspondence-analysis-family"><i class="fa fa-check"></i>An overview of the correspondence analysis family</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#testing-for-presence-of-clustering-effect-in-multilevel-model-with-high-dimensional-predictors"><i class="fa fa-check"></i>TESTING FOR PRESENCE OF CLUSTERING EFFECT IN MULTILEVEL MODEL WITH HIGH-DIMENSIONAL PREDICTORS</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-thursday-14th-of-december.html#programme-and-abstracts-for-thursday-14th-of-december">Programme And Abstracts For Thursday 14<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html"><i class="fa fa-check"></i>Penalized Vector Generalized Additive Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#author-name-identification-for-evaluating-research-performance-of-institutes"><i class="fa fa-check"></i>Author Name Identification for Evaluating Research Performance of Institutes</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#cross-covariance-estimation-for-integration-of-multi-omics-data-1"><i class="fa fa-check"></i>Cross covariance estimation for integration of multi-omics data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#evaluation-of-spatial-cluster-detection-method-based-on-all-geographical-linkage-patterns"><i class="fa fa-check"></i>Evaluation of spatial cluster detection method based on all geographical linkage patterns</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#bayesian-static-parameter-inference-for-partially-observed-stochastic-systems"><i class="fa fa-check"></i>Bayesian Static Parameter Inference for Partially Observed Stochastic Systems</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#testing-for-genetic-associations-in-arbitrarily-structured-populations-2"><i class="fa fa-check"></i>Testing for genetic associations in arbitrarily structured populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#consistency-of-linear-mixed-effects-model-selection-with-inconsistent-covariance-parameter-estimators"><i class="fa fa-check"></i>Consistency of Linear Mixed-Effects Model Selection with Inconsistent Covariance Parameter Estimators</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#a-study-of-the-influence-of-articles-in-the-large-scale-citation-network"><i class="fa fa-check"></i>A Study of the Influence of Articles in the Large-Scale Citation Network</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#regularized-noise-reduction-methodology-for-high-dimensional-data-1"><i class="fa fa-check"></i>Regularized Noise-Reduction Methodology for High-Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#genetic-approach-and-statistical-approach-for-association-study-on-dna-data"><i class="fa fa-check"></i>Genetic Approach and Statistical Approach for Association Study on DNA Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#ssrem-a-summary-statistics-based-random-effect-model-to-estimating-heritability-co-heritability-and-effect-sizes-in-gwas-data-analysis"><i class="fa fa-check"></i>SSREM: A Summary-Statistics-based Random Effect Model to Estimating heritability, co-heritability and effect sizes in GWAS data analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#meta-analytic-principal-component-analysis-in-integrative-omics-application-2"><i class="fa fa-check"></i>Meta-analytic Principal Component Analysis in Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#selecting-generalised-linear-models-under-inequality-constraints"><i class="fa fa-check"></i>Selecting Generalised Linear Models Under Inequality Constraints</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#clustering-of-research-subject-based-on-stochastic-block-model"><i class="fa fa-check"></i>Clustering of research subject based on stochastic block model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#selecting-the-number-of-principal-components-1"><i class="fa fa-check"></i>Selecting the number of principal components</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#symbolic-data-analytical-approach-to-unauthorized-access-logs"><i class="fa fa-check"></i>Symbolic data analytical approach to unauthorized-access logs</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#forward-selection-in-regression-models-based-on-robust-estimation"><i class="fa fa-check"></i>Forward selection in regression models based on robust estimation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#phylogenetic-tree-based-microbiome-association-test-2"><i class="fa fa-check"></i>Phylogenetic Tree-based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#statistical-generalized-derivative-applied-to-the-profile-likelihood-estimation-in-a-mixture-of-semiparametric-models"><i class="fa fa-check"></i>Statistical generalized derivative applied to the profile likelihood estimation in a mixture of semiparametric models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#structure-of-members-in-the-organization-to-induce-innovation-quantitatively-analyze-the-capability-of-the-organization"><i class="fa fa-check"></i>Structure of Members in the Organization to Induce Innovation: Quantitatively Analyze the Capability of the Organization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#towards-a-sparse-scalable-and-stably-positive-definite-inverse-covariance-estimator-1"><i class="fa fa-check"></i>Towards a sparse, scalable, and stably positive definite (inverse) covariance estimator</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#statistical-challenges-for-proteogenomic-data-analysis"><i class="fa fa-check"></i>Statistical challenges for proteogenomic data analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#feature-selection-in-high-dimensional-models-with-complex-block-structures"><i class="fa fa-check"></i>Feature selection in high-dimensional models with complex block structures</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series-2"><i class="fa fa-check"></i>Wavelet-based power transformation of non-Gaussian long memory time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#dissimilarities-between-groups-of-data"><i class="fa fa-check"></i>Dissimilarities between groups of data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#a-package-for-multiple-precision-floating-point-computation-on-r"><i class="fa fa-check"></i>A package for multiple precision floating-point computation on R</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#cluster-wise-regression-models-combined-by-a-quasi-linear-function-1"><i class="fa fa-check"></i>Cluster-wise regression models combined by a quasi-linear function</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#mobile-learning-in-teaching-bioinformatics-for-medical-doctors"><i class="fa fa-check"></i>Mobile Learning in teaching Bioinformatics for medical doctors</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#interactive-visualization-of-aggregated-symbolic-data"><i class="fa fa-check"></i>Interactive visualization of aggregated symbolic data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#an-incomplete-data-fisher-scoring-with-an-acceleration-method"><i class="fa fa-check"></i>An incomplete-data Fisher scoring with an acceleration method</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#clusterwise-low-rank-correlation-analysis-based-on-majorization"><i class="fa fa-check"></i>Clusterwise low-rank correlation analysis based on majorization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#towards-an-informal-test-for-goodness-of-fit"><i class="fa fa-check"></i>Towards an informal test for goodness-of-fit</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#an-ewma-chart-for-monitoring-covariance-matrix-based-on-dissimilarity-index"><i class="fa fa-check"></i>An EWMA chart for monitoring covariance matrix based on dissimilarity index</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#improvement-of-computation-for-nonlinear-multivariate-methods"><i class="fa fa-check"></i>Improvement of Computation for Nonlinear Multivariate Methods</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#three-dimensional-data-visualization-education-with-virtual-reality"><i class="fa fa-check"></i>Three-Dimensional Data Visualization Education with Virtual Reality</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="abstract" class="section level1">
<h1><span class="header-section-number"> 1</span> Abstract</h1>
<p>The requirement for predictions to be made earlier in the growing season has become more important, as the opportunity to plan for the wine production and export earlier in the season becomes desirable. The issue with this is there is less information available to those wishing to make early predictions. The analysis in this paper implements a double sigmoidal curve to model the grape growth over the growing season, as this is typically used in agriculture.</p>
In order to conduct prediction in this study, a Bayesian Network is considered. This allows the opportunity to consider the knowledge of experts in the field, where grape growers would know the growth behaviour of the grapes, as well as using new data to update the Bayesian Network. This information is then implemented in the form of priors, which involves estimating the parameters of the aforementioned double sigmoidal model. Sensitivity Analysis is done in this research, which looks at the impact of prior assumptions (or lack thereof) from experts. Examinations are also made of the value of adding information to the model, as it can be determined whether the precision in the predictions improves as a result of adding data. The results in this analysis are based off simulation studies.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:30 OGGB5 (260-051)
</p>
<div id="pattern-prediction-for-time-series-data-with-change-points" class="section level2 unnumbered">
<h2>Pattern Prediction for Time Series Data with Change Points</h2>
<p style="text-align:center">
Satoshi Goto and Hiroshi Yadohisa<br /> Doshisha University<br />
</p>
<p><span><strong>Pattern Prediction for Time Series Data with Change Points</strong></span></p>
<p>Satoshi Goto<span class="math inline">\(^1\)</span> and Hiroshi Yadohisa<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Faculty of Culture and Information Science, Doshisha University, Kyoto, JAPAN</p>
<p><span><strong>Abstract</strong></span>.</p>
<p>Recently, there have been various types of time series data, such as daily stock prices and Web-click logs, that have complicated the structure. In several cases, because of the complexity, time series data cannot satisfy the stationary process assumption. REGIMECAST (Matsubara and Sakura, 2016) has been proposed as a method to forecast time series data. It is useful for capturing changes in time series patterns and representing the non-linear system. However, it cannot adequately represent time series data after radical changes. Generally, radical changes in time series data can be detected using existing methods, such as change-point detection and anomaly detection. These methods are rarely used for forecasting time series data, although these data often show different behaviors after radical changes.</p>
<p>In this study, we propose a method that can forecast future time series data after events involving radical changes. The method has two features: i) appropriate pattern discovery, as it recognizes the appropriate learning section with change-point detection, and ii) flexible representation, as it represents non-stationary processes with a non-linear state space model. We also provide empirical examples using a variety of real datasets.</p>
<p><span><strong>Keywords</strong></span>. anomaly detection, change-point detection, non-linear state space model, pattern discovery, REGIMECAST</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>: Y. Matsubara and Y. Sakurai (2016). Regime shifts in streams: Real-time forecasting of co-evolving time sequences, <span><em>Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining</em></span>, 13â€“17, 2016.</p>
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:30 Case Room 2 (260-057)
</p>
</div>
<div id="test-for-genomic-imprinting-effects-on-the-x-chromosome" class="section level2 unnumbered">
<h2>Test for Genomic Imprinting Effects on the X Chromosome</h2>
<p style="text-align:center">
Wing Kam Fung<br /> Unversity of Hong Kong<br />
</p>
<p><span><strong>Test for Genomic Imprinting Effects on the X Chromosome</strong></span></p>
<p>Wing Kam Fung<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics and Actuarial Science, The University of Hong Kong, Hong Kong</p>
<p><span><strong>Abstract</strong></span>. Genomic imprinting is an epigenetic phenomenon that the expression of an allele copy depends on its parental origin. This mechanism has been found to play an important role in many diseases. Methods for detecting imprinting effects have been developed primarily for autosomal markers. However, no method is available in the literature to test for imprinting effects on the X chromosome. Therefore, it is necessary to suggest methods for detecting such imprinting effects. In this talk, the parental-asymmetry test on X the chromosome (XPAT) is first developed to test for imprinting for qualitative traits in the presence of association, based on family trios each with both parents and their affected daughter. Then, we propose 1-XPAT to tackle parent-daughter pairs, each with one parent and his/her affected daughter. By simultaneously considering family trios and parent-daughter pairs, C-XPAT is constructed to test for imprinting. Further, we extend the proposed methods to accommodate complete (with both parents) and incomplete (with one parent) nuclear families having multiple daughters of which at least one is affected. Simulations are conducted to assess the performance of the proposed methods under different settings. Simulation results demonstrate that the proposed methods control the size well, irrespective of the inbreeding coefficient in females being zero or nonzero. By incorporating incomplete nuclear families, C-XPAT is more powerful than XPAT using only complete nuclear families. For practical use, these proposed methods are applied to analyze the rheumatoid arthritis data.</p>
<span><strong>Keywords</strong></span>. Imprinting effects, X chromosome, qualitative traits, nuclear family
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:30 Case Room 3 (260-055)
</p>
</div>
<div id="fluctuation-reduction-of-value-at-risk-estimation-and-its-applications" class="section level2 unnumbered">
<h2>Fluctuation Reduction of Value-at-Risk Estimation and its Applications</h2>
<p style="text-align:center">
Shih-Feng Huang<br /> National University of Kaohsiung<br />
</p>
<p><span><strong>Fluctuation Reduction of Value-at-Risk Estimation and its Applications</strong></span></p>
<p>Shih-Feng Huang</p>
<p>Department of Applied Mathematics, National University of Kaohsiung, Kaohsiung, 81148, Taiwan</p>
<p><span><strong>Abstract</strong></span>. Value-at-Risk (VaR) is a fundamental tool for risk management and is also associated with the capital requirements of banks. Banks need to adjust their capital levels for satisfying the Basel Capital Accord. On the other hand, managements do not like to change the capital levels too often. To achieve a balance, this study proposes an approach to reduce the fluctuation of VaR estimation. The first step is to fit a time series model to the underlying asset returns and obtain the conventional VaR process. A new VaR (NVaR) estimation of the conventional VaR process is then determined by applying change-point detection algorithms and a proposed combination scheme. The capital levels computed from the NVaR process are capable of satisfying the Basel Accord and reducing the fluctuation of capital levels simultaneously. To apply the proposed method to the calculation of future capital requirements, an innovative approach for NVaR prediction is also proposed by incorporating the concept of CUSUM control charts. The return processes of 30 companies on the list of S<span class="math inline">\(\&amp;\)</span>P 500 from 2005 to 2016 are employed for our empirical investigation. Numerical results indicate that the proposed NVaR prediction is capable of satisfying the Basel Accord and reducing the fluctuation of capital requirements simultaneously by using a comparable average amount of capital requirements to the conventional VaR estimator.</p>
<span><strong>Keywords</strong></span>. Capital requirement, change point detection, CUSUM control chart, fluctuation reduction, Value-at-Risk
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:30 Case Room 4 (260-009)
</p>
</div>
<div id="meta-analytic-principal-component-analysis-in-integrative-omics-application" class="section level2 unnumbered">
<h2>Meta-analytic Principal Component Analysis in Integrative Omics Application</h2>
<p style="text-align:center">
Sunghwan Kim<sup>1</sup> and George Tseng<sup>2</sup><br /> <sup>1</sup>Keimyung University<br /> <sup>2</sup>University of Pittsburgh<br />
</p>
<p><span><strong>Meta-analytic Principal Component Analysis in Integrative Omics Application</strong></span></p>
<p>SungHwan Kim<span class="math inline">\(^1\)</span> and George C. Tseng<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, Keimyung Univerisity, Deagu 42601, South Korea</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Biostatistics, University of Pittsburgh, Pittsburgh 15213, USA</p>
<p><span><strong>Abstract</strong></span>. With the prevalent usage of microarray and massively parallel sequencing, numerous high-throughput omics datasets have become available in the public domain. Integrating abundant information among omics datasets is critical to elucidate biological mechanisms. Due to the high- dimensional nature of the data, methods such as principal component analysis (PCA) have been widely applied, aiming at effective dimension reduction and exploratory visualization. In this paper, we combine multiple omics datasets of identical or similar biological hypothesis and introduce two variations of meta-analytic framework of PCA, namely MetaPCA. Regularization is further incorporated to facilitate sparse feature selection in MetaPCA. We apply MetaPCA and sparse MetaPCA to simulations, three transcriptomic meta-analysis studies in yeast cell cycle, prostate cancer, mouse metabolism, and a TCGA pan-cancer methylation study. The result shows improved accuracy, robustness and exploratory visualization of the proposed framework.</p>
<p><span><strong>Keywords</strong></span>. principal component analysis, meta-analysis, omics data</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Flury (1984) <em>Common principal components in k groups.</em> Journal of the American Statistical Association, 79, 892â€“898.</p>
Krzanowski (1979) <em>Between-groups comparison of principal components.</em> Journal of the American Statistical Association, 74, 703â€“707
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:50 098 Lecture Theatre (260-098)
</p>
</div>
<div id="estimation-of-animal-density-from-acoustic-detections" class="section level2 unnumbered">
<h2>Estimation of animal density from acoustic detections</h2>
<p style="text-align:center">
Ben Stevenson<sup>1</sup> and David Borchers<sup>2</sup><br /> <sup>1</sup>University of Auckland<br /> <sup>2</sup>University of St Andrews<br />
</p>
<p><span><strong>Estimation of animal density from acoustic detections</strong></span></p>
<p>Ben C. Stevenson<span class="math inline">\(^1\)</span> and David L. Borchers<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, University of Auckland, New Zealand</p>
<p><span class="math inline">\(^2 \;\)</span> School of Mathematics and Statistics, University of St Andrews, United Kingdom</p>
<p><span><strong>Abstract</strong></span>. Estimating the density of animal populations is of central importance in ecology, with practical applications that affect decision making in the fields of wildlife management, conservation, and beyond. For species that vocalise, surveys using acoustic detectors such as microphones, hydrophones, or human observers can be vastly cheaper than traditional surveys that physically capture or visually detect animals. In this talk I describe a spatial capture-recapture approach to estimate animal density from acoustic surveys and present a software implementation in the R package <code>ascr</code>, with examples applied to populations of frogs, gibbons, and whales.</p>
<span><strong>Keywords</strong></span>. Ecological statistics, mark-recapture, point process
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:50 OGGB4 (260-073)
</p>
</div>
<div id="mixed-models-for-complex-survey-data" class="section level2 unnumbered">
<h2>Mixed models for complex survey data</h2>
<p style="text-align:center">
Xudong Huang and Thomas Lumley<br /> University of Auckland<br />
</p>
<p><span><strong>Mixed models for complex survey data</strong></span></p>
<p>Xudong Huang <span class="math inline">\(^1\)</span> and Thomas Lumley<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, University of Auckland, New Zealand</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Statistics, University of Auckland, New Zealand</p>
<p><span><strong>Abstract</strong></span>. I want to fit a mixed model to a population distribution, but I have data from a complex (multistage) sample. The sampling is informative, that is, the model holding for the population is different from the model holding for the (biased) sample. Ignoring the sampling design and just fitting the mixed model to the sample distribution will lead to biased inference. Although both the model and sampling involve â€œclustersâ€, the model clusters and sample clusters need not be the same. I will use a pairwise composite likelihood method to estimate the parameters of the population model under this setting. In particular, consistency and asymptotic normality can be established. Variance estimation in this problem is challenging. I will talk about a variance estimator and how to show it is consistent.</p>
<p><span><strong>Keywords</strong></span>. Mixed model, Complex sampling, Pairwise composite likelihood</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Yi, G. , Rao, J. and Li, H.(2016). <em>A weighted composite likelihood approach for analysis of survey data under two-level models.</em> Statistica Sinica, 2016, 26, 569-587
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:50 OGGB5 (260-051)
</p>
</div>
<div id="regression-with-random-effects-for-analysing-correlated-survival-data-application-to-disease-recurrences" class="section level2 unnumbered">
<h2>Regression with random effects for analysing correlated survival data: application to disease recurrences</h2>
<p style="text-align:center">
Richard Tawiah, Suzanne Chambers, and Shu-Kay Ng<br /> Griffith University<br />
</p>
<p><span><strong>Regression with random effects for analysing correlated survival data: application to disease recurrences</strong></span></p>
<p>Richard Tawiah<span class="math inline">\(^1\)</span>, Suzanne Chambers<span class="math inline">\(^1\)</span> and Shu-Kay Ng<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Menzies Health Institute Queensland, Griffith University, QLD, Australia</p>
<p><span><strong>Abstract</strong></span>. Correlated failure time data arise in many biomedical studies, due to multiple occurrences of the same disease in an individual patient. To account for this correlation phenomenon, we formulate a random effect (frailty) survival model with an autoregressive (AR) covariance structure and adopt the generalized linear mixed model (GLMM) methodology for estimation of regression and variance component parameters. A more general case of the problem is also considered via a multilevel random effect approach where the correlation of survival times is induced by a hierarchical clustering structure, such as the appearances of repeated failures in patients from the same hospital in a multicentre clinical trial setting. Our modelling problem is used to investigate prognostic and treatment effects on disease relapses in two data sets, (1) tumour recurrences in bladder cancer patients and (2) recurrent infections in children with chronic granulomatous disease (CGD). Using the first data set, the effect of treatment thiotepa was found to be insignificant but demonstrated an effect in reducing tumour recurrences with adjusted hazard ratio (AHR) of 0.58 (95% CI: 0.29-1.16, p=0.124). The initial number of tumours (AHR: 1.26, 95% CI: 1.08-1.47, p=0.004) had significant positive effect but the effect of the size of the largest initial tumour was insignificant. In the case of the CGD data, treatment gamma interferon showed a significant decreasing effect (AHR: 0.27, 95% CI: 0.13-0.56, p$&lt;$0.001) on the incidence of recurrent infections. In addition, age effect was significant (AHR: 0.90, 95% CI: 0.81-1.0, p=0.042). Pattern of inheritance, height, weight, sex, use of corticosteroids and prophylactic antibiotics did not exhibit significant association with recurrent infections. The appropriateness of our modelling methodology is investigated in a simulation study. The simulation results show that parameters are satisfactorily estimated in the special case where AR random effect is merely used. However, in the multilevel context bias in the variance parameter of random hospital effect increases as the true magnitude of variation in hospital effects increases.</p>
<span><strong>Keywords</strong></span>. Frailty model, random effect, correlated survival times, recurrent event, GLMM, bladder cancer, CGD
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:50 Case Room 2 (260-057)
</p>
</div>
<div id="genetic-predictors-underlying-long-term-cognitive-recovery-following-mild-traumatic-brain-injury" class="section level2 unnumbered">
<h2>Genetic predictors underlying long-term cognitive recovery following mild traumatic brain injury</h2>
<p style="text-align:center">
Priya Parmar<sup>1</sup>, Rob Kydd<sup>2</sup>, Andrew Shelling<sup>2</sup>, Suzanne Barker-Collo<sup>2</sup>, Alice Theadom<sup>1</sup>, and Valery Feigin<sup>1</sup><br /> <sup>1</sup>Auckland University of Technology<br /> <sup>2</sup>University of Auckland<br />
</p>
<p><span><strong>Abstract</strong></span> Traumatic Brain Injury (TBI) is a major cause of death and disability. While moderate and severe forms of TBI develop the most significant impairments even mild TBI may be followed by persisting post-concussion symptoms, neurocognitive problems and mental health disorders such as anxiety. Cognitive impairments can impact on all areas of an individualâ€™s work, home and social life and are important to understand and predict overall recovery. These outcomes may be in part be determined by genetic variants that influence the molecular and physiological response of the brain to damage, as well as determining pre-injury reserve and vulnerability to co-morbidities.</p>
<p>A number of studies have examined the relationship between genetic variants and outcomes following TBI. Most have examined groups with moderate to severe injury and are limited by small sample sizes, selection biases, failed to correct for ethnic factors, and have evaluated outcomes at various time points, making comparison between studies difficult.</p>
<p>Using the population-based study of TBI in NZ (BIONIC) we analysed the association between cognitive outcomes with 18 genetic markers (SNPs-single nucleotide polymorphisms) from 12 genes previously studied in relation with TBI; FAAH, GAD1, WWC1, CHMR2, ANKK1, BDNF, NGB, BCL2, APOE, S100B, HMOX1 and COMT in a sample of 183 European and 76 Maori adults. We used the CNS-Vital Signs (computerised neurocognitive test battery) to provide 11 measures of cognitive functioning, memory and attention collected at baseline, 1-, 6-, 12- and 48 months post-injury.</p>
<p>ANCOVA models were used to identify the association between time, SNP (modelled as major, heterozygous and minor alleles) and SNP by time effect for each CNS-Vital Signs outcome. Statistically significant findings were observed in both European and Maori samples for being associated with the same CNS-Vital Signs outcome for rs8191992 (CHMR2), rs4680 (rs4680), rs2071746 (HMOX1) and rs17071145 (WWC1).</p>
<p>A linear mixed effects model was utilised to analyse each individualâ€™s natural cognitive recovery trajectory over time. The individualsâ€™ age, gender, whether or not this was their first TBI, the severity level of the mild TBI (low, medium or high) and SNP were all included in the model as covariates.</p>
<p>Regression analyses identified the following SNPs to be statistically associated with several CNS-Vital Signs outcomes; rs8191992 (CHMR2) was shown to be associated with attention, neurocognition, composite memory, executive functioning as well as processing and psychomotor speed in Europeans.</p>
<p>Whilst rs3798178 (GAD1) was associated with two domains of attention, neurocognition and three domains of memory (composite, visual and working) in Maori. We found rs3791879 was associated with increased attention and neurocognition in our European sample.</p>
<p>Furthermore, the minor alleles of rs11604671 (ANKK1) were associated with poorer cognitive recovery (compared to those with homozygous major alleles) for two domains of attention, executive functioning, processing speed, social acuity and working memory over time in Maori. We found rs11604671 was associated with reduced executive functioning and processing speed in our European sample</p>
Unlike other genetic studies on TBI patients, our study investigated several different genetic variants in a larger ethnically diverse population sample of individuals with primarily mild TBI. Although our findings agreed with previous literature for genetic associations for cognitive recovery post-injury, for the first time, we were able to identify ethnic differences in specific genetic markers determining specific cognitive outcomes in European and Maori people with TBI. Further large TBI population based cohort studies are warranted to replicate these genetic associations, both locally and globally in order to better understand the differences underlying an individualâ€™s outcome trajectory and inform more effective treatment strategies.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:50 Case Room 3 (260-055)
</p>
</div>
<div id="bayesian-structure-selection-for-vector-autoregression-model" class="section level2 unnumbered">
<h2>Bayesian Structure Selection for Vector Autoregression Model</h2>
<p style="text-align:center">
Chi-Hsiang Chu<sup>1</sup>, Mong-Na Lo Huang<sup>1</sup>, Shih-Feng Huang<sup>2</sup>, and Ray-Bing Chen<sup>3</sup><br /> <sup>1</sup>National Cheng Kung University<br /> <sup>2</sup>National Sun Yat-sen University<br /> <sup>3</sup>National University of Kaohsiung<br />
</p>
<p><span><strong>Bayesian Structure Selection for Vector Autoregression Model</strong></span></p>
<p>Chi-Hsiang Chu<span class="math inline">\(^1\)</span>, Mong-Na Lo Huang<span class="math inline">\(^1\)</span>, Shih-Feng Huang<span class="math inline">\(^2\)</span> and Ray-Bing Chen<span class="math inline">\(^3\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Applied Mathematics, National Sun Yat-sen University, Kaohsiung, Taiwan</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Applied Mathematics, National University of Kaohsiung, Kaohsiung, Taiwan</p>
<p><span class="math inline">\(^3 \;\)</span> Department of Statistics, National Cheng Kung University, Tainan, Taiwan</p>
<p><span><strong>Abstract</strong></span>. Vector autoregression (VAR) model is powerful in economic data analysis because it can be used to analyze several different time series data simultaneously. However, in VAR model, we need to deal with the huge coefficient dimensionality and it would be caused some computational problems for coefficient inference. To reduce the dimensionality, we could take some model structures into account based on the prior knowledge. In this paper, several group structures of the coefficient matrices are considered. Due to different types of VAR structures, corresponding MCMC algorithms are proposed to generate posterior samples for making inference of the structure selection. Simulation studies and a real example are used to show the performances of the proposed Bayesian approaches.</p>
<span><strong>Keywords</strong></span>. Bayesian variable selection, time series, universal grouping, segmentized grouping
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:50 Case Room 4 (260-009)
</p>
</div>
<div id="phylogenetic-tree-based-microbiome-association-test" class="section level2 unnumbered">
<h2>Phylogenetic Tree-based Microbiome Association Test</h2>
<p style="text-align:center">
Sungho Won<br /> Seoul National University<br />
</p>
<p><span><strong>Phylogenetic Tree-based Microbiome Association Test</strong></span></p>
<p>Sungho Won<span class="math inline">\(^{1,2}\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Public Health Science, Seoul National University, Seoul, Korea</p>
<p><span class="math inline">\(^2 \;\)</span> Interdisciplinary Program for Bioinformatics, College of Natural Science, Seoul National University, Seoul</p>
<p><span><strong>Abstract</strong></span>. Microbial metagenomics data has large inter-subject variation and operational taxonomic units (OTU) for each species are usually very sparse. Because of these problems, non-parametric approaches such as Mann-Whitney U test and Wilcoxon rank-sum test have been utilized. However these approaches suffer from low statistical powers for association analyses and thus investigation on efficient statistical analyses is necessary. Main goal in my thesis is to propose phylogenetic Tree-based Microbiome Association Test (TMAT) for association analyses between microbiome abundances of each OTU and disease phenotype. Phylogenetic tree reveals similarity between different OTUs, and thus was used to provide TMAT. TMAT calculates score test statistics for each node and test statistics for all nodes are combined into a single statistics by minimum p-value or Fisherâ€™s combing p-value method. TMAT was compared with existing methods with extensive simulations. Simulation studies show that TMAT preserves the nominal type-1 error and its statistical powers were usually much better than existing methods for considered scenarios. Furthermore it was applied to atopic diseases and found that community profiles of Enterococcus is associated.</p>
<span><strong>Keywords</strong></span>. NGS; phylogenetic treel Microbiome Association Test
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 15:10 098 Lecture Theatre (260-098)
</p>
</div>
<div id="talk-data-to-me" class="section level2 unnumbered">
<h2>Talk Data To Me</h2>
<p style="text-align:center">
Lisa Hall<br /> Fonterra<br />
</p>
<span><strong>Abstract</strong></span> For some scientists, seeing a statistician in the corridor is the equivalent of coming face to face with the grim reaper. They fear being beaten over the head with a Stat101 textbook as you try to hammer into them the importance of replication, randomisation or an appropriate sample size. Many scientists would rather blunder through on their own than admit to the statistician that they donâ€™t understand the ins-and-outs of ANOVA. So we end up with peer reviewed research with badly interpreted p-values and underwhelming Excel graphs despite the existence of more elegant solutions. As ambassadors of statistical rigour, we are the ones who can turn this around. When the scientists wonâ€™t come to us, how can we improve data usage short of chasing scientists down corridors with textbooks? I will give examples of how to Talk Data with scientists to encourage their statistical awakening. In the right context, even the most puritanical scientists can become more comfortable Talking Data. They may even find they enjoy it!
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 15:10 OGGB4 (260-073)
</p>
</div>
<div id="smooth-nonparametric-regression-under-shape-restrictions" class="section level2 unnumbered">
<h2>Smooth Nonparametric Regression under Shape Restrictions</h2>
<p style="text-align:center">
Hongbin Guo and Yong Wang<br /> University of Auckland<br />
</p>
<p><span><strong>Smoothing Nonparametric Regression under Shape Restrictions</strong></span></p>
<p>Hongbin Guo and Yong Wang</p>
<p>Department of Statistics, University of Auckland</p>
<p><span><strong>Abstract</strong></span>. Shape-restricted regression, in particular under isotonicity and convexity(concavity) constraints, has many practical applications. Traditional nonparametric methods to the problem using least squares or maximum likelihood result in discrete step functions or nonsmooth piecewise linear functions, which are unsatisfactory both predictively and visually. In this talk, we describe a new, smooth, nonparametric estimator under the above-mentioned shape restrictions. In particular, the discrete measures that are inherent in the previous estimators are replaced with continuous ones. A new algorithm that can rapidly find the corresponding estimate will also be presented. Numerical studies show that the new estimator outperforms major existing methods in almost all cases.</p>
<p><span><strong>Keywords</strong></span>. Nonparametric regression, smooth, shape restriction, convex, monotonic</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Groeneboom, P., Jongbloed, G. and Wellner, A. (2001). Estimation of a Convex Function: Characterizations and Asymptotic Theory. <em>Ann. Statist</em>. <strong>29</strong>(6), 1653â€“1698.</p>
<p>Wang, Y. (2007). On fast computation of the non-parametric maximum likelihood estimate of a mixing distribution. <em>Journal of the Royal Statistical Society. Series B: Statistical Methodology</em>, <strong>69</strong> (2), 185â€“198.</p>
Meyer, M.(2008). Inference using shape-restricted regression splines. <em>Ann. Appl. Stat.</em> <strong>2</strong>(3), 1013â€“1033.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 15:10 OGGB5 (260-051)
</p>
</div>
<div id="elastic-band-transform-a-new-approach-to-multiscale-visualization" class="section level2 unnumbered">
<h2>Elastic-Band Transform: A New Approach to Multiscale Visualization</h2>
<p style="text-align:center">
Guebin Choi and Hee-Seok Oh<br /> Seoul National University<br />
</p>
<p><span><strong>Elastic-Band Transform : A New Approach to Multiscale Visualization</strong></span></p>
<p>Guebin Choi<span class="math inline">\(^1\)</span> and Hee-Seok Oh<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics Seoul National University Seoul 08826, Korea</p>
<p><span><strong>Abstract</strong></span>. This paper presents a new transformation technique for multiscale visualization of one-dimensional data such as time series and functional data under the concept of the scale-space approach. The proposed method uses a range of regular observations (eye scanning) with varying intervals. The results, termed â€˜elastic-band transformâ€™ can be considered as a collection of observations over different intervals of viewing. It is motivated by a way that human looks at an object such as a sequence of data repeatedly in order to overview a global structure of it as well as find some specific features of it. Some measures based on elastic-bands are discussed for describing characteristics of data, and two-dimensional visualizations induced by the measures are developed for understanding and detecting important structures of data. Furthermore, some statistical applications are studied.</p>
<p><span><strong>Keywords</strong></span>. Transformation; Visualization; Decomposition; Filter; Time Series;</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Chaudhuri, P. and Marron, J.Â S. (1999). SiZer for exploration of structures in curves. <span><em>Journal of the American Statistical Association</em></span>, <span><strong>94</strong></span>, 807â€“823.</p>
<p>Donoho, D. L., and Johnstone, I. M. (1994). Ideal spatial adaptation by wavelet shrinkage. <span><em>Biometrika</em></span>, <span><strong>81</strong></span>, 425-455.</p>
<p>Dragomiretskiy, K. and Zosso, D. (2014). Variational mode decomposition. <span><em>IEEE Transactions on Signal Processing</em></span>, <span><strong>62</strong></span>, 531â€“544.</p>
<p>ErÃ¤stÃ¶, P. and HolmstrÃ¶m, L. (2005). Bayesian multiscale smoothing for making inferences about features in scatter plots. <span><em>Journal of Computational and Graphical Statistics</em></span>, <span><strong>14</strong></span>, 569â€“589.</p>
<p>Fryzlewicz, P. and Oh, H.-S. (2011). Thick pen transformation for time series. <span><em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em></span>, <span><strong>73</strong></span>, 499â€“529.</p>
<p>Hannig, J. and Lee, T. C. M. (2006). Robust SiZer for exploration of regression structures and outlier detection. <span><em>Journal of Computational and Graphical Statistics</em></span>, <span><strong>15</strong></span>, 101â€“117.</p>
<p>Hannig, J., Lee, T. and Park, C. (2013). Metrics for SiZer map comparison. <span><em>Stat</em></span>, <span><strong>2</strong></span>, 49â€“60.</p>
<p>HolmstrÃ¶m, L. (2010a). BSiZer. <span><em>Wiley Interdisciplinary Reviews: Computational Statistics</em></span>, <span><strong>2</strong></span>, 526â€“534.</p>
<p>HolmstrÃ¶ma, L. (2010b). Scale space methods. <span><em>Wiley Interdisciplinary Reviews: Computational Statistics</em></span>, <span><strong>2</strong></span>,150â€“159.</p>
<p>HolmstrÃ¶ma, L. and Pasanena, L. (2017). Statistical scale space methods. <span><em>International Statistical Review</em></span>, <span><strong>85</strong></span>, 1â€“30.</p>
<p>Huang, N. E., Shen, Z., Long, S. R., Wu, M. C., Shih, H. H., Zheng, Q., â€¦ &amp; Liu, H. H. (1998). The empirical mode decomposition and the Hilbert spectrum for nonlinear and non-stationary time series analysis. <span><em>Proceedings of the Royal Society of London A: Mathematical, Physical and Engineering Sciences</em></span>, <span><strong>454</strong></span>, 903â€“995.</p>
<p>Lindeberg, T. (1994). <span><em>Scale-Space Theory in Computer Vision</em></span>, Springer Science &amp; Business Media, New York.</p>
<p>Park, C., Hannig, J. and Kang, K. H. (2009). Improved SiZer for time series. <span><em>Statistica Sinica</em></span>, <span><strong>19</strong></span>, 1511â€“1530.</p>
<p>Park, C, Lee, T. C. and Hannig, J. (2010). Multiscale exploratory analysis of regression quantiles using quantile SiZer. <span><em>Journal of Computational and Graphical Statistics</em></span>, <span><strong>19</strong></span>, 497â€“513.</p>
<p>Rioul, O. and Vetterli, M. (1991). Wavelets and signal processing. <span><em>IEEE Signal Processing Magazine</em></span>, 8(LCAV-ARTICLE-1991-005), 14â€“38.</p>
Vogt, M., &amp; Dette, H. (2015). Detecting gradual changes in locally stationary processes. The Annals of Statistics, 43(2), 713-740.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 15:10 Case Room 2 (260-057)
</p>
</div>
<div id="a-computational-tool-for-detecting-copy-number-variations-from-whole-genome-and-targeted-exome-sequencing" class="section level2 unnumbered">
<h2>A Computational Tool for Detecting Copy Number Variations from Whole Genome and Targeted Exome Sequencing</h2>
<p style="text-align:center">
Yu-Chung Wei<sup>1</sup> and Guan-Hua Huang<sup>2</sup><br /> <sup>1</sup>Feng Chia University<br /> <sup>2</sup>National Chiao Tung University<br />
</p>
<p><span><strong>A Computational Tool for Detecting Copy Number Variations from Whole Genome and Targeted Exome Sequencing</strong></span></p>
<p>Yu-Chung Wei<span class="math inline">\(^1\)</span> and Guan-Hua Huang<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, Feng Chia University, Taichung 40724, TAIWAN</p>
<p><span class="math inline">\(^2 \;\)</span> Institute of Statistics, National Chiao Tung University, Hsinchu 30010, TAIWAN</p>
<p><span><strong>Abstract</strong></span>. Copy number variations (CNVs) are genomic structural mutations with abnormal gene fragment copies. Current CNV detection algorithms for next generation sequencing (NGS) are developed for specific genome targets, including whole genome sequencing and targeted exome sequencing based on the differently data types and corresponding assumptions. Many whole genome tools assume the continuity of search space and reads uniform coverage across the genome. However, these assumptions break down in the exome capture because of discontinuous segments and exome specific functional biases. In order to develop a method adapting to both data types, we specify the large unconsidered genomic fragments as gaps to preserve the truly location information. A Bayesian hierarchical model was built and an efficient reversible jump Markov chain Monte Carlo inference algorithm was utilized to incorporate the gap information. The performance of gap settings for the Bayesian procedure was evaluated and compared with competing approaches using both simulations and real data.</p>
<span><strong>Keywords</strong></span>. Bayesian inference, Bioinformatics, copy number variation, next generation sequencing, reversible jump Markov chain Monte Carlo
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 15:10 Case Room 3 (260-055)
</p>
</div>
<div id="flight-to-relative-safety-learning-from-a-no-arbitrage-network-of-yield-curves-model-of-the-euro-area" class="section level2 unnumbered">
<h2>Flight to relative safety: Learning from a no-arbitrage network of yield curves model of the euro area</h2>
<p style="text-align:center">
Zhiwu Hong<sup>1</sup> and Linlin Niu<sup>2</sup><br /> <sup>1</sup>HKUST Business School<br /> <sup>2</sup>Xiamen University<br />
</p>
<span><strong>Abstract</strong></span> This paper develops a no-arbitrage network of yield curves model for the euro area to study the joint dynamics of and risk transmission across yield curves of different economies. The model nests 1+M yield curves of a benchmark country and M periphery countries with 3+(2+M) state factors. The benchmark curve is spanned by three yield factors, which are also common basis for all curves. Periphery yield spreads are spanned by three factors, including two common risk factors related to market liquidity risk and common sovereign credit risk, and a country-specific risk factor. Parsimony can be guaranteed as M increases with additional yield curve data, and factors can be strongly identified with structural restrictions under the no-arbitrage conditions. We estimate a 1+5 yield curves model for Germany and GIIPS countries using weekly data from 2009 to 2016. The results show that, the German yields enjoy a â€˜flight to liquidityâ€™ effect under a liquidity shock, which pushes down German yields while driving up periphery spreads. However, in response to a shock of common sovereign credit risk, yields of all countries, including Germany, tend to increase immediately and persistently. The weaker is a countryâ€™s economic fundamental, the higher its risk exposure to shocks. Though spillover effects among periphery curves are generally positive, when a Greek shock bursts, the Italian risk factor may be temporarily mitigated due to its relative stronger economy. The network model can be adapted with a time-varying parameter VAR to monitor evolving contributions of factors.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 15:10 Case Room 4 (260-009)
</p>
</div>
<div id="wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series" class="section level2 unnumbered">
<h2>Wavelet-based power transformation of non-Gaussian long memory time series</h2>
<p style="text-align:center">
Kyungduk Ko<sup>1</sup> and Chul Eung Kim<sup>2</sup><br /> <sup>1</sup>Boise State University<br /> <sup>2</sup>Yonsei University<br />
</p>
<p><span><strong>Wavelet-based Power Transformation of non-Gaussian Long Memory Time Series</strong></span></p>
<p>Kyungduk Ko<span class="math inline">\(^1\)</span> and Chul Eung Kim<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Mathematics, Boise State University, Boise, ID 83725, USA</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Applied Statistics, Yeonsei University, Seoul, Korea</p>
<p><span><strong>Abstract</strong></span>. We consider a power transformation through the well-known Box-cox transformation to induce normality from non-Gaussian long memory processes and propose a Bayesian method to simultaneously estimate the transformation parameter and long memory parameter. To ease computational burdens due to the dense variance-covariance matrix of long memory time series, we base our statistical inference on the wavelet domain rather than the original data domain. For a joint estimation of the parameters of interest, posterior estimations are carried out via Markov chain Monte Carlo (MCMC). An application to German stock return data is presented.</p>
<p><span><strong>Keywords</strong></span>. Box-Cox transformation, Discrete wavelet transform, Long memory, MCMC, Normality</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Dahlhaus, R. (1990). Efficient location and regression estimation for long range dependent regression models. <em>Annuals of Statistics</em>, 23, 1029â€“1047.</p>
<p>Ko, K. and Lee, J. (2008). Confidence intervals for long memory regressions. <em>Statistics and Probability Letters</em>, 78, 1894â€“1902.</p>
Lee, J. and Ko, K. (2007). One-way analysis of variance with long memory errors and its application to stock return data. <em>Applied Stochastic Models in Business and Industry</em>, <strong>23</strong>, 493â€“502.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:00 OGGB4 (260-073)
</p>
</div>
<div id="robustness-of-temperature-reconstruction-for-past-500-years" class="section level2 unnumbered">
<h2>Robustness of Temperature Reconstruction for Past 500 Years</h2>
<p style="text-align:center">
Yu Yang, Matthew Schofield, and Richard Barker<br /> University of Otago<br />
</p>
<p><span><strong>Robustness of Temperature Reconstruction for Past 500 Years</strong></span></p>
<p>Yu Yang<span class="math inline">\(^1\)</span>, Matthew Schofield<span class="math inline">\(^1\)</span>, Richard Barker<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^{1} \;\)</span> Department of Mathematics and Statistics, University of Otago, P. O. Box 56 Dunedin 9016, New Zealand</p>
<p><span><strong>Abstract</strong></span>. Temperature reconstruction is vital to studies of climate change. Instrumental records are only available back to 19th century, too short to describe changes that occur over hundreds or thousands of years. Fortunately, nature environmental clues (such as tree rings, pollens and ice cores) can be pieced together to reconstruct unrecorded temperatures. We use tree-ring width to study summer temperature in Northern Sweden for past 500 years. Previous work has shown the predictions to be sensitive to model assumptions. We gain a new insight into this problem by attempting to separately estimate aspects of the process that are robustly estimated. One of these are the years in which the climate is colder or warmer than recent observations. We implement this by considering hidden Markov models on the partially observed temperature series. The model is fitted using Hamiltonian Monte Carlo in Stan.</p>
<p><span><strong>Keywords</strong></span>. temperature reconstruction, robust estimator, hidden Markov model, Bayesian analysis</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Schofield, M. R., Barker, R. J., Gelman, A., Cook, E. R., and Briffa, K. R. (2016). A model-based approach to climate reconstruction using tree-ring data. <em>Journal of the American Statistical Association</em>, 111(513), 93-106.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:00 OGGB5 (260-051)
</p>
</div>
<div id="nonparametric-causal-inference-by-the-kernel-method" class="section level2 unnumbered">
<h2>Nonparametric causal inference by the kernel method</h2>
<p style="text-align:center">
Yuchi Matsuoka and Etsuo Hamada<br /> Osaka University<br />
</p>
<p><span><strong>Nonparametric causal inference by the kernel method</strong></span></p>
<p>Yuchi Matsuoka<span class="math inline">\(^1\)</span> and Etsuo Hamada<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Graduate School of Engineering Science, Osaka University,<br />
1-3 Machikaneyama-cho, Toyonaka, Osaka, Japan</p>
<p><span><strong>Abstract</strong></span>. Rubin causal model is a statistical model to estimate the effect of a treatment on the outcome based on the framework of potential outcomes. To estimate a causal effect based on Rubin causal model, propensity score plays a central role. In particular, matching and weighting methods like Inverse Probability Weighted Estimator (IPWE) and Doubly-Robust estimator based on the estimated propensity score are widely used. Despite its popularity, it was pointed out that model misspecification of the propensity score can result in substantial bias of the resulting estimators of a causal effect and potential outcomes. It is possible to estimate propensity score in nonparametric ways or machine learning methods to avoid model misspecification. However, it doesnâ€™t work well in most situations due to following reasons: 1) Curse of dimensionality. 2) They only aim at an accuracy of classification and donâ€™t optimize the covariate balancing. To overcome the problems above, we propose a new estimator of propensity score using kernel mean embeddings of conditional distributions. Although our proposal is completely nonparametric, our estimator has a dimensionality-independent rate of convergence. Using kernel measures of conditional independence for model selection, our estimator can also correct the bias that arises from the imbalance of covariates. In numerical simulations, we confirm that our method can reduce the bias in misspecified settings. We also describe several asymptotic properties of our estimator.</p>
<span><strong>Keywords</strong></span>. Rubin causal model, Propensity score, Kernel method, Kernel mean embedding, Hilbert-Schmidt Independence Criterion
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:00 Case Room 2 (260-057)
</p>
</div>
<div id="a-unified-regularized-group-pls-algorithm-scalable-to-big-data" class="section level2 unnumbered">
<h2>A Unified Regularized Group PLS Algorithm Scalable to Big Data</h2>
<p style="text-align:center">
Pierre Lafaye de Micheaux<sup>1</sup>, Benoit Liquet<sup>2</sup>, and Matthew Sutton<sup>2</sup><br /> <sup>1</sup>Queensland University of Technology<br /> <sup>2</sup>University of New South Wales<br />
</p>
<p><span><strong>A Unified Regularized Group PLS Algorithm Scalable to Big Data</strong></span></p>
<p>Pierre Lafaye de Micheaux<span class="math inline">\(^1\)</span> and Benoit Liquet<span class="math inline">\(^{2,3}\)</span> and Matthew Sutton <span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> School of Mathematics and Statistics, UNSW Sydney, Australia</p>
<p><span class="math inline">\(^2 \;\)</span> ACEMS, Queensland University of Technology, Brisbane, Australia</p>
<p><span class="math inline">\(^3 \;\)</span> Laboratory of Mathematics and its Applications, University of Pau et Pays de Lâ€™Adour, UMR CNRS 5142, France</p>
<p><span><strong>Abstract</strong></span>. Partial Least Squares (PLS) methods have been heavily exploited to analyse the association between two blocs of data. These powerful approaches can be applied to data sets where the number of variables is greater than the number of observations and in presence of high collinearity between variables. Different sparse versions of PLS have been developed to integrate multiple data sets while simultaneously selecting the contributing variables. Sparse modelling is a key factor in obtaining better estimators and identifying associations between multiple data sets. The cornerstone of the sparsity version of PLS methods is the link between the SVD of a matrix (constructed from deflated versions of the original matrices of data) and least squares minimisation in linear regression. We present here an accurate description of the most popular PLS methods, alongside their mathematical proofs. A unified algorithm is proposed to perform all four types of PLS including their regularised versions. Various approaches to decrease the computation time are offered, and we show how the whole procedure can be scalable to big data sets.</p>
<p><span><strong>Keywords</strong></span>. Big data, High dimensional data, Lasso Penalties, Partial Least Squares, Sparsity, SVD</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Lafaye de Micheaux, P., Liquet, B. &amp; Sutton, M. (2017), <em>A Unified Parallel Algorithm for Regularized Group PLS Scalable to Big Data</em>, ArXiv e-prints .</p>
Liquet, B., Lafaye de Micheaux, P., Hejblum, B. &amp; Thiebaut, R. (2016), <em>Group and sparse group partial least square approaches applied in genomics context</em>, Bioinformatics 32, 35-42.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:00 Case Room 3 (260-055)
</p>
</div>
<div id="meta-analysis-with-symbolic-data-analysis-and-its-application-for-clinical-data" class="section level2 unnumbered">
<h2>Meta-analysis with symbolic data analysis and its application for clinical data</h2>
<p style="text-align:center">
Ryo Takagi, Hiroyuki Minami, and Masahiro Mizuta<br /> Hokkaido University<br />
</p>
<p><span><strong>Meta-analysis with symbolic data analysis and its application for clinical data</strong></span></p>
<p>Ryo Takagi<span class="math inline">\(^1\)</span>, Hiroyuki Minami<span class="math inline">\(^2\)</span>, and Masahiro Mizuta<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Graduate School of Information Science and Technology, Hokkaido University</p>
<p><span class="math inline">\(^2 \;\)</span> Laboratory of Advanced Data Science, Information Initiative Center, Hokkaido University</p>
<p><span><strong>Abstract</strong></span>. We discuss a method of meta-analysis based on symbolic data analysis (SDA). Meta-analysis, mainly used in social and medical science, is a statistical method of combining scientific studies to obtain quantitative results and provides a high level of evidence. Differences between the studies are caused by heterogeneity between the studies. It is useful to detect relationship among scientific studies. A target of analysis on SDA is <span><em>concept</em></span>, a set of individuals. We apply SDA to meta-analysis. In other words, we regard scientific studies as concepts. For example, symbolic clustering or symbolic MDS are useful to preprocess the scientific studies in meta-analysis. In this study, we propose a new approach based on SDA for meta-analysis and show the results of the proposed approach using clinical datasets.</p>
<p><span><strong>Keywords</strong></span>. symbolic clustering, symbolic MDS, concept in SDA</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Edwin Diday and Monique Noirhomme-Fraiture. (2008). <em>Symbolic data analysis and the SODAS software.</em> John Wiley &amp; Sons, Ltd.</p>
David Edward Matthews and Vernon Todd Farewell. (2015). <em>Using and understanding medical statistics</em> (5th, revised and extended edition). Karger Publishers.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:00 Case Room 4 (260-009)
</p>
</div>
<div id="scoring-rules-for-prediction-and-classification-challenges" class="section level2 unnumbered">
<h2>Scoring rules for prediction and classification challenges</h2>
<p style="text-align:center">
Matt Parry<br /> University of Otago<br />
</p>
<p><span><strong>Scoring rules for prediction and classification challenges</strong></span></p>
<p>Matthew Parry<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Mathematics &amp; Statistics, University of Otago, Dunedin, NZ</p>
<p><span><strong>Abstract</strong></span>. Prediction and classification challenges have become an exciting and useful feature of the statistical and machine learning community. For example, Good JudgementÂ Open asks forecasters to predict the probability of particular world events, and Kaggle.com regularly sets classification challenges. Challenge organizers typically publish a ranked list of the leading submissions and, ultimately, announce the winner of the challenge. However, in order for such a competition to be considered worth entering, the challenge organizers must be seen to evaluate the submissions in a fair and open manner. Scoring rules were devised precisely to solve this problem. Crucially, <span><em>proper</em></span> scoring rules elicit honest statements of belief about the outcome. If the challenge organizers use a proper scoring rule to evaluate submissions, a competitorâ€™s expected score under their true belief will be optimized by actually quoting that belief to the organizers. A proper scoring rule therefore rules out any possibility of a competitor gaming the challenge. We discuss a class of proper scoring rules called linear scoring rules that are specifically adapted to probabilistic binary classification. When applied in competition situations, we show that all linear scoring rules essentially balance the needs of organizers and competitors. We also develop scoring rules to score a sequence of predictions that are targeting a single outcome. These scoring rules discount predictions over time and appropriately weight prediction updates.</p>
<p><span><strong>Keywords</strong></span>. Probabilistic forecast, sequence, prequential principle, discounting</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Parry, M. (2016). <em>Linear scoring rules for probabilistic binary classification</em>. Electronic Journal of Statistics, 10 (1), 1596â€“1607.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:20 OGGB4 (260-073)
</p>
</div>
<div id="real-time-transit-network-modelling-for-improved-arrival-time-predictions" class="section level2 unnumbered">
<h2>Real-time transit network modelling for improved arrival time predictions</h2>
<p style="text-align:center">
Tom Elliott and Thomas Lumley<br /> University of Auckland<br />
</p>
<p><span><strong>Real-time transit network modelling for improved arrival time predictions</strong></span></p>
<p>Tom Elliott and Thomas Lumley</p>
<p>Department of Statistics, University of Auckland, Auckland, New Zealand</p>
<p><span><strong>Abstract</strong></span>. The growing availability of GPS tracking devices means that public transport passengers can now check on the real-time location of their bus from their mobile phone, helping them to decide when to leave home, and once at the stop, how long until the bus arrives. A side effect of this technology is that statistical models using vehicle location data to predict arrival times have taken a â€œback seatâ€ in preference for methods that are simpler and faster, but less robust. Auckland Transport, who operate our local public transport network, demonstrate this: the estimated arrival time (ETA) of a bus at a stop is simply the time until scheduled arrival, plus the delay at the busâ€™ most recently visited stop. The most evident problem with this approach is that intermediate stops, traffic lights, and road congestionâ€”all of which affect ETAsâ€”are not considered. We have been developing a modelling framework consisting of (1) a vehicle state model to infer parameters, such as speed, from a sequence of GPS positions; (2) a transit network model that uses information from the vehicle model to estimate traffic conditions along roads in the network; and (3) a predictive model combining vehicle and transit network states to predict arrival times. Since multimodality is commonâ€”for example a bus may or may not stop at a bus stop or traffic lightsâ€”we are using a particle filter to estimate vehicle state, which makes no assumptions about the shape of the distribution, and allows for a more intuitive likelihood function. While this provides a very flexible framework, it is also a computationally intensive one, so computational demands need to be considered to ensure it will be viable as a real-time application for providing passengers with improved, and hopefully reliable, arrival time information.</p>
<span><strong>Keywords</strong></span>. transit, real-time, particle filter
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:20 OGGB5 (260-051)
</p>
</div>
<div id="sparse-group-subgroup-partial-least-squares-with-application-to-genomic-data" class="section level2 unnumbered">
<h2>Sparse Group-subgroup Partial Least Squares with Application to Genomic Data</h2>
<p style="text-align:center">
Matthew Sutton<sup>1</sup>, Benoit Liquet<sup>1</sup>, and Rodolphe Thiebaut<sup>2</sup><br /> <sup>1</sup>Inria, SISTM, Talence and Inserm, U1219, Bordeaux, Bordeaux University, Bordeaux and Vaccine Research Institute<br /> <sup>2</sup>Queensland University of Technology<br />
</p>
<p><span><strong>Sparse Group-subgroup Partial Least Squares with Application to Genomic Data</strong></span></p>
<p>Matthew Sutton<span class="math inline">\(^1\)</span> and Rodolphe ThiÃ©baut<span class="math inline">\(^3\)</span> and Benoit Liquet<span class="math inline">\(^{2,1}\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> ACEMS, Queensland University of Technology, Brisbane, Australia</p>
<p><span class="math inline">\(^2 \;\)</span> Laboratory of Mathematics and its Applications, University of Pau et Pays de Lâ€™Adour, UMR CNRS 5142, France.</p>
<p><span class="math inline">\(^3 \;\)</span> Inria, SISTM, Talence and Inserm, U1219, Bordeaux, Bordeaux University, Bordeaux and Vaccine Research Institute, Creteil, France.</p>
<p><span><strong>Abstract</strong></span>. Integrative analysis of high dimensional omics datasets has been studied by many authors in recent years. By incorporating prior known relationships among the variables, these analyses have been successful in elucidating the relationships between different sets of omics data. In this article, our goal is to identify important relationships between genomic expression and cytokine data from an HIV vaccination trial. We proposed a flexible Partial Least Squares technique which incorporates group and subgroup structure in the modelling process. Our new methodology expands on previous work, by accounting for both grouping of genetic markers (e.g.Â genesets) and temporal effects. The method generalises existing sparse modelling techniques in the PLS methodology and establishes theoretical connections to variable selection methods for supervised and unsupervised problems. Simulation studies are performed to investigate the performance of our methods over alternative sparse approaches. Our method has been implemented in a comprehensive R package called sgsPLS.</p>
<p><span><strong>Keywords</strong></span>. genomics, group variable selection, latent variable modelling, partial least squares, singular value decomposition</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Chaussabel, D. etÂ al. (2008). A modular analysis framework for blood genomics studies: application to systemic lupus erythematosus. <em>Immunity</em> <span><strong>29,</strong></span> 150â€“164.</p>
<p>Chun, H. and KeleÅŸ, S. (2010). Sparse partial least squares regression for simultaneous dimension reduction and variable selection. <em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em> <span><strong>72,</strong></span> 3â€“25.</p>
<p>Garcia, T.Â P., Muller, S., Carroll, R., and Walzem, R. (2014). Identification of important regressor groups, subgroups and individuals via regularization methods: application to gut microbiome data. <em>Bioinformatics</em> <span><strong>30,</strong></span> 35â€“42.</p>
<p>Gomez-Cabrero, D., Abugessaisa, I., Maier, D., Teschendorff, A., Merkenschlager, M., Gisel, A., Ballestar, E., Bongcam-Rudloff, E., Conesa, A., and Tegn<span>Ã©</span>r, J. (2014). Data integration in the era of omics: current and future challenges. <em>BMC Systems Biology</em> <span><strong>8,</strong></span> 1â€“10.</p>
<p>Hejblum, B., Skinner, J., and ThiÃ¨baut, R. (2015). Time-course gene set analysis for longitudinal gene expression data. <em>PLOS Computational Biology</em> <span><strong>11,</strong></span> 1â€“21.</p>
<p><span>Le Cao</span>, K., Rossouw, D., Robert-Granie, C., and Besse, P. (2008). <span>A</span> sparse <span>P</span><span>L</span><span>S</span> for variable selection when integrating omics data. <em>Stat Appl Genet Mol Biol</em> <span><strong>7,</strong></span> 37.</p>
<p>LÃ¨vy, Y., ThiÃ¨baut, R., Montes, M., Lacabaratz, C., Sloan, L., King, B., PÃ¨rusat, S., Harrod, C., Cobb, A., Roberts, L., Surenaud, M., Boucherie, C., Zurawski, S., Delaugerre, C., Richert, L., ChÃªne, G., Banchereau, J., and Palucka, K. (2014). Dendritic cell-based therapeutic vaccine elicits polyfunctional hiv-specific t-cell immunity associated with control of viral load. <em>European Journal of Immunology</em> <span><strong>44,</strong></span> 2802â€“2810.</p>
<p>Lin, D., Zhang, J., Li, J., Calhoun, V., Deng, H., and Wang, Y. (2013). Group sparse canonical correlation analysis for genomic data integration. <em>BMC Bioinformatics</em> <span><strong>14,</strong></span> 1â€“16.</p>
<p>Liquet, B., deÂ Micheaux, P.Â L., Hejblum, B., and ThiÃ©baut, R. (2016). Group and sparse group partial least square approaches applied in genomics context. <em>Bioinformatics</em> <span><strong>32,</strong></span> 35â€“42.</p>
<p>Nowak, G., Hastie, T., Pollack, J., and Tibshirani, R. (2011). A fused lasso latent feature model for analyzing multisample acgh data. <em>Biostatistics</em> <span><strong>12,</strong></span> 776â€“791.</p>
<p>Parkhomenko, E., Tritchler, D., and Beyene, J. (2009). Sparse canonical correlation analysis with application to genomic data integration. <em>Statistical applications in genetics and molecular biology</em> <span> <strong>8,</strong></span> Article 1.</p>
<p>Rosipal, R. and KrÃ¤mer, N. (2006). Overview and recent advances in partial least squares. <em>Subspace, Latent Structure and Feature Selection</em></p>
<p>Safo, S.Â E., Li, S., and Long, Q. (2017). Integrative analysis of transcriptomic and metabolomic data via sparse canonical correlation analysis with incorporation of biological information. <em>Biometrics</em> .</p>
<p>Simon, N., Friedman, J., Tibshirani, R., and Hastie, T. (2013). A <span>S</span>parse-<span>G</span>roup <span>L</span>asso. <em>Journal of Computational and Graphical Statistics</em> <span><strong>22,</strong></span> 213â€“245.</p>
<p>Tenenhaus, A., Philippe, C., Guillemot, V., <span>Le Cao</span>, K., Grill, J., and Frouin, V. (2014). Variable selection for generalized canonical correlation analysis. <em>Biostatistics</em> <span><strong>15,</strong></span> 569â€“583.</p>
<p>Tibshirani, R. (1994). Regression shrinkage and selection via the lasso. <em>Journal of the Royal Statistical Society, Series B</em> <span><strong>58,</strong></span> 267â€“288.</p>
Witten, D., Tibshirani, R., and Hastie, T. (2009). A penalized matrix decomposition, with applications to sparse principal components and canonical correlation analysis. <em>Biostatistics</em> <span><strong>10,</strong></span> 515â€“534.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:20 Case Room 2 (260-057)
</p>
</div>
<div id="big-sir-a-sliced-inverse-regression-approach-for-massive-data" class="section level2 unnumbered">
<h2>BIG-SIR a Sliced Inverse Regression Approach for Massive Data</h2>
<p style="text-align:center">
Benoit Liquet<sup>1</sup> and Jerome Saracco<sup>2</sup><br /> <sup>1</sup>Queensland University of Technology<br /> <sup>2</sup>University of Bordeaux<br />
</p>
<p><span><strong>BIG-SIR a Sliced Inverse Regression approach for massive data</strong></span></p>
<p>Benoit Liquet<span class="math inline">\(^{1,2}\)</span> and Jerome Saracco <span class="math inline">\(^3\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Laboratory of Mathematics and its Applications, University of Pau et Pays de Lâ€™Adour, UMR CNRS 5142, France</p>
<p><span class="math inline">\(^2 \;\)</span> ACEMS, Queensland University of Technology, Brisbane, Australia</p>
<p><span class="math inline">\(^3 \;\)</span> University of Bordeaux, Talence, France</p>
<p><span><strong>Abstract</strong></span>. In a massive data setting, we focus on a semiparametric regression model involving a real dependent variable <span class="math inline">\(Y\)</span> and a <span class="math inline">\(p\)</span>-dimensional covariable <span class="math inline">\(X\)</span>. This model includes a dimension reduction of X via an index <span class="math inline">\(X&#39;\beta\)</span>. The Effective Dimension Reduction (EDR) direction <span class="math inline">\(\beta\)</span> cannot be directly estimated by the Sliced Inverse Regression (SIR) method due to the large volume of the data. To deal with the main challenges of analysing massive datasets which are the storage and computational efficiency, we propose a new SIR estimator of the EDR direction by following the â€œdivide and conquerâ€ strategy. The data is divided into subsets. EDR directions are estimated in each subset which is a small dataset. The recombination step is based on the optimisation of a criterion which assesses the proximity between the EDR directions of each subset. Computations are run in parallel with no communication among them. The consistency of our estimator is established and its asymptotic distribution is given. Extensions to multiple indices models, <span class="math inline">\(q\)</span>-dimensional response variable and/or SIR<span class="math inline">\(_{\alpha}\)</span>-based methods are also discussed. A simulation study using our <code>edrGraphicalTools</code> R package shows that our approach enables us to reduce the computation time and conquer the memory constraint problem posed by massive datasets. A combination of <code>foreach</code> and <code>bigmemory</code> R packages are exploited to offer efficiency of execution in both speed and memory. Results are visualised using the bin-summarise-smooth approach through the <code>bigvis</code> R package. Finally, we illustrate our proposed approach on a massive airline data set.</p>
<p><span><strong>Keywords</strong></span>. High performance computing, Effective Dimension Reduction (EDR), Parallel programming, R software, Sliced Inverse Regression (SIR).</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Liquet, B., &amp; Saracco, J. (2016), <em>BIG-SIR a Sliced Inverse Regression Approach for Massive Data</em>, Statistics and Its Interface. Vol 9, 509-520.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:20 Case Room 3 (260-055)
</p>
</div>
<div id="comparison-of-tests-of-mean-difference-in-longitudinal-data-based-on-block-resampling-methods" class="section level2 unnumbered">
<h2>Comparison of Tests of Mean Difference in Longitudinal Data Based on Block Resampling Methods</h2>
<p style="text-align:center">
Hirohito Sakurai and Masaaki Taguri<br /> National Center for University Entrance Examinations<br />
</p>
<p><span><strong>Comparison of Tests of Mean Difference in Longitudinal Data Based on Block Resampling Methods</strong></span></p>
<p>Hirohito Sakurai<span class="math inline">\(^1\)</span> and Masaaki Taguri<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> National Center for University Entrance Examinations, Tokyo 153-8501, Japan</p>
<p><span><strong>Abstract</strong></span>. Let us consider a two-sample problem in longitudinal data, and discuss comparison of tests of mean difference using block resampling methods. The testing methods are based on moving block bootstrap (MBB), circular block bootstrap (CBB) and stationary bootstrap (SB). These block resampling techniques are used to approximate the null distributions of the following four types of test statistics: sum of absolute values of difference between two mean sequences (<span class="math inline">\(T_1\)</span>), sum of squares of difference between two mean sequences (<span class="math inline">\(T_2\)</span>), area-difference between two mean curves (<span class="math inline">\(T_3\)</span>), and difference of kernel estimators based on two mean sequences (<span class="math inline">\(S_n\)</span>). Our testing algorithm generates blocks of observations in each sample similar to MBB, CBB or SB, and draws resamples <em>with replacement</em> or <em>without replacement</em> from the mixed blocks which are generated by two samples. In the context of block resampling, a resample is usually generated <em>with replacement</em> from blocks of observations, however our discussion also includes block resampling <em>without replacement</em> similar to permutation analogy for MBB, CBB and SB, with <span class="math inline">\(T_1\)</span>, <span class="math inline">\(T_2\)</span>, <span class="math inline">\(T_3\)</span> and <span class="math inline">\(S_n\)</span>, respectively. Monte Carlo simulations are carried out to examine the empirical level and power of the testing methods.</p>
<p><span><strong>Keywords</strong></span>. moving block bootstrap, circular block bootstrap, stationary bootstrap, with/without replacement, empirical level/power</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Lahiri, S.Â N. (2003). <em>Resampling Methods for Dependent Data</em>. New York: Springer.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:20 Case Room 4 (260-009)
</p>
</div>
<div id="modeling-of-document-abstraction-using-association-rule-based-characterization" class="section level2 unnumbered">
<h2>Modeling of Document Abstraction Using Association Rule Based Characterization</h2>
<p style="text-align:center">
Ken Nittono<br /> Hosei University<br />
</p>
<p><span> <strong>Modeling of Document Abstraction Using Association Rule Based Characterization</strong> </span></p>
<p>Ken Nittono</p>
<p>Department of Markets and Management, Hosei University, 102-8160 Tokyo, Japan</p>
<p><span><strong>Abstract</strong></span>. The importance of systems enabling us to extract useful information from enormous text data produced every day along with our social activities in organizations or on the internet and utilize the information immediately and efficiently have been increasing. In this research, an analyzing method which extracts essential parts from a huge document set utilizing association rule analysis as a data mining method is introduced. The method detects typical combinations of terms involved in contexts and regards them as the characterization of text data and it is also combined with information retrieval methods for the sake of further selection as some parts of the essential contexts. This method is considered to enhance its ability of detection for particular contexts that contain some topics and include moderately distributed terminologies. And implementation of the system is discussed in order for utilizing the abstracted documents efficiently as some sort of knowledge such as collective intelligence. An approach for linkage with R is also mentioned in the phase of the implementation of the model.</p>
<p><span><strong>Keywords</strong></span>. Association rule, Text mining, Big data, Information retrieval</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Agrawal, R. Imielinski, T. and Swami, A. (1993). <em>Mining association rules between sets of items in large databases</em>, Proceedings of the ACM SIGMOD Washington, D.C, 207â€“216.</p>
Nittono, K. (2013). <em>Association rule generation and mining approach to concept space for collective documents</em>, Proceedings of the 59th World Statistics Congress of the International Statistical Institute, pp.Â 5515â€“5520.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:40 OGGB4 (260-073)
</p>
</div>
<div id="bayesian-survival-analysis-of-batsmen-in-test-cricket" class="section level2 unnumbered">
<h2>Bayesian survival analysis of batsmen in Test cricket</h2>
<p style="text-align:center">
Oliver Stevenson and Brendon Brewer<br /> University of Auckland<br />
</p>
<p><span><strong>Bayesian survival analysis of batsmen in Test cricket</strong></span></p>
<p>O. G. Stevenson<span class="math inline">\(^1\)</span> and B. J. Brewer<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, University of Auckland, Auckland, New Zealand</p>
<p><span><strong>Abstract</strong></span> It is widely accepted that in the sport of cricket, batting is more difficult early in a playerâ€™s innings, but becomes easier as a player familiarizes themselves with the local conditions. Here we develop a Bayesian survival analysis method to predict and quantify the Test Match batting abilities for international cricketers, at any stage of a playerâ€™s innings. The model is applied in two stages, firstly to individual players, allowing us to quantify playersâ€™ initial and equilibrium batting abilities, and the rate of transition between the two. The results indicate that most players begin a Test match innings batting with between a quarter and a half of their potential batting ability. The model is then implemented using a hierarchical structure, providing us with more general inference concerning a selected group of opening batsmen from New Zealand. Using this hierarchical structure we are able to make predictions for the batting abilities of the next opening batsman to debut for New Zealand. These results are considered in conjunction with other performance based metrics, allowing us to identify players who excel in the role of opening the batting, which has practical implications in terms of batting order and team selection policy.</p>
<p><span><strong>Keywords</strong></span>. Bayesian survival analysis, hierarchical modelling, cricket</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Stevenson, O.G. and Brewer, B.J. (2017). Bayesian survival anaylsis of opening batsmen in Test cricket <em>Journal of Quantitative Analysis in Sports</em>, <em>13</em>(1), 25-36.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:40 OGGB5 (260-051)
</p>
</div>
<div id="covariate-discretisation-on-big-data" class="section level2 unnumbered">
<h2>Covariate Discretisation on Big Data</h2>
<p style="text-align:center">
Hon Hwang<sup>1</sup>, Stephen Wright<sup>2</sup>, and Louise Ryan<sup>1</sup><br /> <sup>1</sup>Australian Red Cross Blood Service<br /> <sup>2</sup>University of Technology Sydney<br />
</p>
<p><span><strong>Covariate Discretisation on Big Data</strong></span></p>
<p>Hon Hwang<span class="math inline">\(^{1,2}\)</span>, Stephen Wright<span class="math inline">\(^{2,3}\)</span>, and Louise Ryan<span class="math inline">\(^{1,2}\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> The University of Technology Sydney (UTS)</p>
<p><span class="math inline">\(^2 \;\)</span> The Australian Research Council (ARC) Centre of Excellence for Mathematical and Statistical Frontiers (ACEMS)</p>
<p><span class="math inline">\(^3 \;\)</span> Australian Red Cross Blood Service</p>
<p><span><strong>Abstract</strong></span>. Distributed Computing Systems such as Hadoop and Spark allow statistical analysis to be performed on arbitrary large datasets. However, when performing statistical analysis on these systems, the data communication between the nodes of a distributed computing system can become a major performance bottleneck. In this work, we outline a novel combination of statistical and computation techniques to address this issue. We first apply data reduction technique such as coarsening (interval-censoring) on large data sets using a distributed computing system. We then perform statistical analysis on the coarsened data. However, performing analysis using coarsened data potentially introduces biases in the results. To address this, we use the Expectation-Maximisation (EM) algorithm to recover the complete (non-coarsened) data model. Our work draws on methods for the analysis of data involving coarsened co-variates using EM by methods of weights. We explore different coarsening strategies (e.g., rounding, quantile and quintile) and discuss how our methods can scale to very large data settings. Through simulation studies, we find our method works especially well when data is coarsened from a wide interval, where there are more loss of information. Compared with naÃ¯vely using the coarsened data, our method is able to estimate regression coefficients that are closer to estimates obtained from using the complete data. In addition, the standard errors from our method reflect more accurately the uncertainty arising from using coarsened data.</p>
<span><strong>Keywords</strong></span>. EM algorithm, coarsened data, regression, big data
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:40 Case Room 2 (260-057)
</p>
</div>
<div id="visualization-and-statistical-modeling-of-financial-big-data" class="section level2 unnumbered">
<h2>Visualization and Statistical Modeling of Financial Big Data</h2>
<p style="text-align:center">
Masayuki Jimichi<sup>1</sup>, Daisuke Miyamoto<sup>2</sup>, Chika Saka<sup>1</sup>, and Syuichi Nagata<sup>1</sup><br /> <sup>1</sup>Kwansei Gakuin University<br /> <sup>2</sup>Nara Institute of Science and Technology<br />
</p>
<p><span><strong>Visualization and Statistical Modeling of Financial Big Data</strong></span></p>
<p>Masayuki Jimichi<span class="math inline">\(^1\)</span>, Daisuke Miyamoto<span class="math inline">\(^2\)</span>, Chika Saka<span class="math inline">\(^1\)</span>, and Syuichi Nagata<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> School of Business Administration, Kwansei Gakuin University, Nishinomiya, Uegahara, 662-8501, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Graduate Schook of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma, Nara 630-0192, Japan</p>
<p><span><strong>Abstract</strong></span>. In this work, we manipulate financial big data of world-wide companies by R packages SparkR and sparklyr, and treate data visualization (e.g.Â Ihaka, 2017; Unwin, 2015) and statistical model (e.g.Â Chambers and Hastie, 1991) based on exploratory data analysis (Tukey, 1977) with R. The dataset is extracted from the database systems by Bureau van Dijk KK which contains information on over 80,000 listed companies. We find that a log-skew-t linear model (e.g.Â Azallini and Capitanio, 2014) is very useful for explaining sales by employees and assets.</p>
<p><span><strong>Keywords</strong></span>. Financial Big Data, Data Visualization, Statistical Modeling, Log-skew-t Linear Model, SparkR, sparklyr</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>-4pt</p>
<p>Azzalini, A. with the collaboration of Capitanio, A. (2014). <em>The Skew-Normal and Related Families</em>. Cambridge University Press. Institute of Mathematical Statistics Monographs.</p>
<p>Chambers, J. M. and Hastie, T. J. ed. (1991). <em>Statistical Models in S</em>. Chapman and Hall/CRC.</p>
<p>Ihaka, R. (2017). Lecture Notes. <a href="https://www.stat.auckland.ac.nz/~ihaka/?Teaching" class="uri">https://www.stat.auckland.ac.nz/~ihaka/?Teaching</a></p>
<p>Tukey, J. W. (1977). <em>Exploratory Data Analysis</em>. Addison-Wesley Publishing Co.</p>
Unwin, A. (2015). <em>Graphical Data Analysis with R</em>, Chapman and Hall/CRC.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:40 Case Room 3 (260-055)
</p>
</div>
<div id="analysis-of-spatial-data-with-a-gaussian-mixture-markov-random-field-model" class="section level2 unnumbered">
<h2>Analysis of Spatial Data with a Gaussian Mixture Markov Random Field Model</h2>
<p style="text-align:center">
Wataru Sakamoto<br /> Okayama University<br />
</p>
<p><span><strong>Analysis of Spatial Data with a Gaussian Mixture Markov Random Field Model</strong> </span></p>
<p>Wataru Sakamoto</p>
<p>Graduate School of Environmental and Life Science, Okayama University, 700-8530, Japan</p>
<p><span><strong>Abstract</strong></span>. In spatial data, detecting regions with higher relative risk is of primary interest. A latent Markov random field model with Gaussian mixture component is introduced, in which the probit or the logit of the mixture weight for each location follows a Gaussian Markov random field such as an intrinsic auto-regressive model (Besag <em>et al.</em>, 1991). A mixture model with spatially correlated weights was proposed by FernÃ¡ndez and Green (2002), and our modeling with Gaussian mixture Markov random field can be extended to the cases of involving covariate and random effects. Parameters are estimated by a Bayesian approach, and the posterior mean of the mixture weight for each location, which varies smoothly, gives meaningful interpretation for spatial structure. Our computation was conducted with R Stan package, in which the Hamiltonian Monte Carlo method is implemented. Some applications to disease mapping data are illustrated.</p>
<p><span><strong>Keywords</strong></span>. Bayesian modeling, spatial cluster detection, spatial correlation</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>FernÃ¡ndez, C. and Green, P. J. (2002). Modelling spatially correlated data via mixtures: a Bayesian approach. <em>J. Roy. Statist. Soc. B</em>, <strong>64</strong>, 805â€“826.</p>
<p>Besag, J., York, J. and MolliÃ©, A. (1991). Bayesian image restoration, with two applications in spatial statistics. <em>Ann. Inst. Statist. Math.</em>, <strong>43</strong>, 1â€“59.</p>
Rue, H. and Held, L. (2005) <em>Gaussian Markov Random Fields: Theory and Applications.</em> Chapman and Hall.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 16:40 Case Room 4 (260-009)
</p>
</div>
<div id="my-knee-still-hurts-the-statistical-pathway-to-the-development-of-a-clinical-decision-aid" class="section level2 unnumbered">
<h2>My knee still hurts; the statistical pathway to the development of a clinical decision aid</h2>
<p style="text-align:center">
Robert Borotkanics<br /> Auckland University of Technology<br />
</p>
<p>NZSA Abstract</p>
<p>My knee still hurts; the statistical pathway to the development of a clinical decision aid</p>
<p>Borotkanics R, Rice DA, Kluger M, McNair PJ, , Lewis G, Somogyi A, Barret, D, Walker M. Auckland University of Technology, Waitemata District Health Board and University of Adelaide</p>
<p>Total knee arthroplasty (TKA) is considered an effective intervention to improve physical function and reduce joint pain in those with end stage knee arthritis, yet up to 34 What is reported herein is the methodological approaches applied to tease out the various nuances of developing such a clinical decision aid. By way of summary, a series of logistic regression models were developed and refined to identify predictors chronic postoperative pain, where pain was reported using the numerical pain rating scale (NRS). Self-reported NRS pain was dichotomized based on functional status using a defined statistical approach. Multivariate models were developed using a stepwise selection approach, accounting for interaction and collinearity. The effect of changing collinearity thresholds on information criterion is illustrated. The sensitivity and specificity were calculated, along with receiver operating characteristic (ROC) analyses for each logistic model. Final models were chosen by a combination of superior area under the curve (AUC) and Akaike Information Criterion (AIC). The stability of Î² coefficients across the top performing models is reported, along with model goodness-of-fit using the Hosmer and Lameshow methodology. Cut-point analyses are reported on models performances, including the effect of changing pain thresholds on model accuracy. Finally, the conversion of a superior logistic model into a probabilistic function, of potential utility for clinicians is illustrated.</p>
Keywords: total knee arthroplasty; logistic regression; clinical decision support
<p class="pagebreak">
</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="programme-and-abstracts-for-monday-11th-of-december.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="programme-and-abstracts-for-tuesday-12th-of-december.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["NZSA-IASC-Programme.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
