<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>New Zealand Statistical Association and the International Association of Statistical Computing (Asian Regional Section) Joint Conference 2017</title>
  <meta name="description" content="Programme And Abstracts For NZSA-IASC-ARS Conference 2017">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="New Zealand Statistical Association and the International Association of Statistical Computing (Asian Regional Section) Joint Conference 2017" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="pictures/logo.png" />
  <meta property="og:description" content="Programme And Abstracts For NZSA-IASC-ARS Conference 2017" />
  <meta name="github-repo" content="jmcurran/NZSA-IASC-Prog" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="New Zealand Statistical Association and the International Association of Statistical Computing (Asian Regional Section) Joint Conference 2017" />
  
  <meta name="twitter:description" content="Programme And Abstracts For NZSA-IASC-ARS Conference 2017" />
  <meta name="twitter:image" content="pictures/logo.png" />

<meta name="author" content="Auckland New Zealand">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="indexbysubmission.html">
<link rel="next" href="Tuesday.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li style = "text-align: center;"><a href="./">New Zealand Statistical Association<br/>and the<br/>International Association<br/>of<br/>Statistical Computing<br/>(Asian Regional Section)<br/>Joint Conference 2017</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcomes</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#welcome-to-the-conference-from-ian-westbrooke-nzsa-president"><i class="fa fa-check"></i>Welcome To The Conference From Ian Westbrooke, NZSA President</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#welcome-address-from-jung-jin-lee-iasc-ars-chair-person"><i class="fa fa-check"></i>Welcome Address From Jung Jin Lee, IASC-ARS Chair-Person</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html"><i class="fa fa-check"></i>Programme At A Glance</a><ul>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#Monday-tbl"><i class="fa fa-check"></i>Monday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#Tuesday-tbl"><i class="fa fa-check"></i>Tuesday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#Wednesday-tbl"><i class="fa fa-check"></i>Wednesday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#Thursday-tbl"><i class="fa fa-check"></i>Thursday</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="am-i-a-session-chair-please-read.html"><a href="am-i-a-session-chair-please-read.html"><i class="fa fa-check"></i>Am I A Session Chair? Please Read!!!</a><ul>
<li class="chapter" data-level="" data-path="am-i-a-session-chair-please-read.html"><a href="am-i-a-session-chair-please-read.html#Monday-chairtbl"><i class="fa fa-check"></i>Monday</a></li>
<li class="chapter" data-level="" data-path="am-i-a-session-chair-please-read.html"><a href="am-i-a-session-chair-please-read.html#Tuesday-chairtbl"><i class="fa fa-check"></i>Tuesday</a></li>
<li class="chapter" data-level="" data-path="am-i-a-session-chair-please-read.html"><a href="am-i-a-session-chair-please-read.html#Wednesday-chairtbl"><i class="fa fa-check"></i>Wednesday</a></li>
<li class="chapter" data-level="" data-path="am-i-a-session-chair-please-read.html"><a href="am-i-a-session-chair-please-read.html#Thursday-chairtbl"><i class="fa fa-check"></i>Thursday</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="indexbyauthor.html"><a href="indexbyauthor.html"><i class="fa fa-check"></i>Index By First Name of Author</a></li>
<li class="chapter" data-level="" data-path="indexbysubmission.html"><a href="indexbysubmission.html"><i class="fa fa-check"></i>Index by Submission Number</a></li>
<li><a href="Monday.html#Monday">Programme And Abstracts For Monday 11<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html"><i class="fa fa-check"></i>R In Times Of Growing User Base And Data Sizes</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#robust-principal-expectile-component-analysis"><i class="fa fa-check"></i>Robust Principal Expectile Component Analysis</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#effect-of-area-level-deprivation-on-body-mass-index-analysis-of-nz-health-surveys"><i class="fa fa-check"></i>Effect Of Area Level Deprivation On Body Mass Index: Analysis Of NZ Health Surveys</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#calendar-based-graphics-for-visualising-peoples-daily-schedules"><i class="fa fa-check"></i>Calendar-Based Graphics For Visualising People’s Daily Schedules</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#nonparametric-test-for-volatility-in-clustered-multiple-time-series"><i class="fa fa-check"></i>Nonparametric Test For Volatility In Clustered Multiple Time Series</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#igess-a-statistical-approach-to-integrating-individual-level-genotype-data-and-summary-statistics-in-genome-wide-association-studies"><i class="fa fa-check"></i>IGESS: A Statistical Approach To Integrating Individual Level Genotype Data And Summary Statistics In Genome Wide Association Studies</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#author-name-identification-for-evaluating-research-performance-of-institutes"><i class="fa fa-check"></i>Author Name Identification For Evaluating Research Performance Of Institutes</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#a-computational-tool-for-detecting-copy-number-variations-from-whole-genome-and-targeted-exome-sequencing"><i class="fa fa-check"></i>A Computational Tool For Detecting Copy Number Variations From Whole Genome And Targeted Exome Sequencing</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#clustering-using-nonparametric-mixtures-and-mode-identification"><i class="fa fa-check"></i>Clustering Using Nonparametric Mixtures And Mode Identification</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#bayesian-curve-fitting-for-discontinuous-function-using-overcomplete-representation-with-multiple-kernels"><i class="fa fa-check"></i>Bayesian Curve Fitting For Discontinuous Function Using Overcomplete Representation With Multiple Kernels</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#estimation-of-a-semiparametric-spatiotemporal-models-with-mixed-frequency"><i class="fa fa-check"></i>Estimation Of A Semiparametric Spatiotemporal Models With Mixed Frequency</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#lsmm-a-statistical-approach-to-integrating-functional-annotations-with-genome-wide-association-studies"><i class="fa fa-check"></i>LSMM: A Statistical Approach To Integrating Functional Annotations With Genome-Wide Association Studies</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#a-study-of-the-influence-of-articles-in-the-large-scale-citation-network"><i class="fa fa-check"></i>A Study Of The Influence Of Articles In The Large-Scale Citation Network</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#estimating-links-of-a-network-from-time-to-event-data"><i class="fa fa-check"></i>Estimating Links Of A Network From Time To Event Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#estimation-of-a-high-dimensional-covariance-matrix"><i class="fa fa-check"></i>Estimation Of A High-Dimensional Covariance Matrix</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#innovative-bayesian-estimation-in-the-von-mises-distribution"><i class="fa fa-check"></i>Innovative Bayesian Estimation In The von Mises Distribution</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#evidence-of-climate-change-from-nonparametric-change-point-analysis"><i class="fa fa-check"></i>Evidence Of Climate Change From Nonparametric Change-Point Analysis</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#joint-analysis-of-individual-level-genotype-data-and-summary-statistics-by-leveraging-pleiotropy"><i class="fa fa-check"></i>Joint Analysis Of Individual Level Genotype Data And Summary Statistics By Leveraging Pleiotropy</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#an-advanced-approach-for-time-series-forecasting-using-deep-learning"><i class="fa fa-check"></i>An Advanced Approach For Time Series Forecasting Using Deep Learning</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#genetic-map-estimation-using-hidden-markov-models-in-the-presence-of-partially-observed-information"><i class="fa fa-check"></i>Genetic Map Estimation Using Hidden Markov Models In The Presence Of Partially Observed Information</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#a-simple-method-for-grouping-patients-based-on-historical-doses"><i class="fa fa-check"></i>A Simple Method For Grouping Patients Based On Historical Doses</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#semiparametric-mixed-analysis-of-covariance-model"><i class="fa fa-check"></i>Semiparametric Mixed Analysis Of Covariance Model</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#adaptive-false-discovery-rate-regression-with-application-in-integrative-analysis-of-large-scale-genomic-data"><i class="fa fa-check"></i>Adaptive False Discovery Rate Regression With Application In Integrative Analysis Of Large-Scale Genomic Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#structure-of-members-in-the-organization-to-induce-innovation-quantitatively-analyze-the-capability-of-the-organization"><i class="fa fa-check"></i>Structure Of Members In The Organization To Induce Innovation: Quantitatively Analyze The Capability Of The Organization</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#vector-generalized-linear-time-series-models"><i class="fa fa-check"></i>Vector Generalized Linear Time Series Models</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#local-canonical-correlation-analysis-for-multimodal-labeled-data"><i class="fa fa-check"></i>Local Canonical Correlation Analysis For Multimodal Labeled Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#a-practitioners-guide-to-deep-learning-for-predictive-analytics-on-structured-data"><i class="fa fa-check"></i>A Practitioners Guide To Deep Learning For Predictive Analytics On Structured Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#clustering-of-research-subject-based-on-stochastic-block-model"><i class="fa fa-check"></i>Clustering Of Research Subject Based On Stochastic Block Model</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#zen-and-the-art-of-workflow-maintenance"><i class="fa fa-check"></i>Zen And The aRt Of Workflow Maintenance</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#canonical-covariance-analysis-for-mixed-numerical-and-categorical-three-way-three-mode-data"><i class="fa fa-check"></i>Canonical Covariance Analysis For Mixed Numerical And Categorical Three-Way Three-Mode Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#variable-selection-algorithms"><i class="fa fa-check"></i>Variable Selection Algorithms</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#estimating-causal-structures-for-continuous-and-discrete-variables"><i class="fa fa-check"></i>Estimating Causal Structures For Continuous And Discrete Variables</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#incorporating-genetic-networks-into-case-control-association-studies-with-high-dimensional-dna-methylation-data"><i class="fa fa-check"></i>Incorporating Genetic Networks Into Case-Control Association Studies With High-Dimensional DNA Methylation Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#adaptive-model-checking-for-functional-single-index-models"><i class="fa fa-check"></i>Adaptive Model Checking For Functional Single-Index Models</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#mobile-learning-in-teaching-bioinformatics-for-medical-doctors"><i class="fa fa-check"></i>Mobile Learning In Teaching Bioinformatics For Medical Doctors</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#on-optimal-group-testing-designs-prevalence-estimation-cost-considerations-and-dilution-effects"><i class="fa fa-check"></i>On Optimal Group Testing Designs: Prevalence Estimation, Cost Considerations, And Dilution Effects</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#the-use-of-bayesian-networks-in-grape-yield-prediction"><i class="fa fa-check"></i>The Use Of Bayesian Networks In Grape Yield Prediction</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#pattern-prediction-for-time-series-data-with-change-points"><i class="fa fa-check"></i>Pattern Prediction For Time Series Data With Change Points</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#test-for-genomic-imprinting-effects-on-the-x-chromosome"><i class="fa fa-check"></i>Test For Genomic Imprinting Effects On The X Chromosome</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#fluctuation-reduction-of-value-at-risk-estimation-and-its-applications"><i class="fa fa-check"></i>Fluctuation Reduction Of Value-At-Risk Estimation And Its Applications</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#e-learning-courses-on-introductory-statistics-using-interactive-educational-tools"><i class="fa fa-check"></i>E-Learning Courses On Introductory Statistics Using Interactive Educational Tools</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#estimation-of-animal-density-from-acoustic-detections"><i class="fa fa-check"></i>Estimation Of Animal Density From Acoustic Detections</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#mixed-models-for-complex-survey-data"><i class="fa fa-check"></i>Mixed Models For Complex Survey Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#regression-with-random-effects-for-analysing-correlated-survival-data-application-to-disease-recurrences"><i class="fa fa-check"></i>Regression With Random Effects For Analysing Correlated Survival Data: Application To Disease Recurrences</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#genetic-predictors-underlying-long-term-cognitive-recovery-following-mild-traumatic-brain-injury"><i class="fa fa-check"></i>Genetic Predictors Underlying Long-Term Cognitive Recovery Following Mild Traumatic Brain Injury</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#bayesian-structure-selection-for-vector-autoregression-model"><i class="fa fa-check"></i>Bayesian Structure Selection For Vector Autoregression Model</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#three-dimensional-data-visualization-education-with-virtual-reality"><i class="fa fa-check"></i>Three-Dimensional Data Visualization Education With Virtual Reality</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#talk-data-to-me"><i class="fa fa-check"></i>Talk Data To Me</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#smooth-nonparametric-regression-under-shape-restrictions"><i class="fa fa-check"></i>Smooth Nonparametric Regression Under Shape Restrictions</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#elastic-band-transform-a-new-approach-to-multiscale-visualization"><i class="fa fa-check"></i>Elastic-Band Transform: A New Approach To Multiscale Visualization</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#meta-analytic-principal-component-analysis-in-integrative-omics-application"><i class="fa fa-check"></i>Meta-Analytic Principal Component Analysis In Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#flight-to-relative-safety-learning-from-a-no-arbitrage-network-of-yield-curves-model-of-the-euro-area"><i class="fa fa-check"></i>Flight To Relative Safety: Learning From A No-Arbitrage Network Of Yield Curves Model Of The Euro Area</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#bayesian-analyses-of-non-homogeneous-gaussian-hidden-markov-models"><i class="fa fa-check"></i>Bayesian Analyses Of Non-Homogeneous Gaussian Hidden Markov Models</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#robustness-of-temperature-reconstruction-for-past-500-years"><i class="fa fa-check"></i>Robustness Of Temperature Reconstruction For Past 500 Years</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#nonparametric-causal-inference-by-the-kernel-method"><i class="fa fa-check"></i>Nonparametric Causal Inference By The Kernel Method</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#a-unified-regularized-group-pls-algorithm-scalable-to-big-data"><i class="fa fa-check"></i>A Unified Regularized Group PLS Algorithm Scalable To Big Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#evaluation-of-spatial-cluster-detection-method-based-on-all-geographical-linkage-patterns"><i class="fa fa-check"></i>Evaluation Of Spatial Cluster Detection Method Based On All Geographical Linkage Patterns</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#scoring-rules-for-prediction-and-classification-challenges"><i class="fa fa-check"></i>Scoring Rules For Prediction And Classification Challenges</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#meta-analysis-with-symbolic-data-analysis-and-its-application-for-clinical-data"><i class="fa fa-check"></i>Meta-Analysis With Symbolic Data Analysis And Its Application For Clinical Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#real-time-transit-network-modelling-for-improved-arrival-time-predictions"><i class="fa fa-check"></i>Real-Time Transit Network Modelling For Improved Arrival Time Predictions</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#visualization-and-statistical-modeling-of-financial-big-data"><i class="fa fa-check"></i>Visualization And Statistical Modeling Of Financial Big Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#sparse-group-subgroup-partial-least-squares-with-application-to-genomic-data"><i class="fa fa-check"></i>Sparse Group-Subgroup Partial Least Squares With Application To Genomic Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#genetic-approach-and-statistical-approach-for-association-study-on-dna-data"><i class="fa fa-check"></i>Genetic Approach And Statistical Approach For Association Study On DNA Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#modeling-of-document-abstraction-using-association-rule-based-characterization"><i class="fa fa-check"></i>Modeling Of Document Abstraction Using Association Rule Based Characterization</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#bayesian-static-parameter-inference-for-partially-observed-stochastic-systems"><i class="fa fa-check"></i>Bayesian Static Parameter Inference For Partially Observed Stochastic Systems</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#bayesian-survival-analysis-of-batsmen-in-test-cricket"><i class="fa fa-check"></i>Bayesian Survival Analysis Of Batsmen In Test Cricket</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#covariate-discretisation-on-big-data"><i class="fa fa-check"></i>Covariate Discretisation On Big Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#big-sir-a-sliced-inverse-regression-approach-for-massive-data"><i class="fa fa-check"></i>BIG-SIR A Sliced Inverse Regression Approach For Massive Data</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#symbolic-data-analytical-approach-to-unauthorized-access-logs"><i class="fa fa-check"></i>Symbolic Data Analytical Approach To Unauthorized-Access Logs</a></li>
<li class="chapter" data-level="" data-path="Monday.html"><a href="Monday.html#my-knee-still-hurts-the-statistical-pathway-to-the-development-of-a-clinical-decision-aid"><i class="fa fa-check"></i>My Knee Still Hurts; The Statistical Pathway To The Development Of A Clinical Decision Aid</a></li>
</ul></li>
<li><a href="Tuesday.html#Tuesday">Programme And Abstracts For Tuesday 12<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html"><i class="fa fa-check"></i>Could Do Better … A Report Card For Statistical Computing</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#rd-policy-regimes-in-france-new-evidence-from-a-spatio-temporal-analysis"><i class="fa fa-check"></i>R&amp;D Policy Regimes In France: New Evidence From A Spatio-Temporal Analysis</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#analysing-scientific-collaborations-of-new-zealand-institutions-using-scopus-bibliometric-data"><i class="fa fa-check"></i>Analysing Scientific Collaborations Of New Zealand Institutions Using Scopus Bibliometric Data</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#family-structure-and-academic-achievements-of-high-school-students-in-tonga"><i class="fa fa-check"></i>Family Structure And Academic Achievements Of High School Students In Tonga</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#analysis-of-multivariate-binary-longitudinal-data-metabolic-syndrome-during-menopausal-transition"><i class="fa fa-check"></i>Analysis Of Multivariate Binary Longitudinal Data: Metabolic Syndrome During Menopausal Transition</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#clustering-of-curves-on-a-spatial-domain-using-a-bayesian-partitioning-model"><i class="fa fa-check"></i>Clustering Of Curves On A Spatial Domain Using A Bayesian Partitioning Model</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#the-uncomfortable-entrepreneurs-bad-working-conditions-and-entrepreneurial-commitment"><i class="fa fa-check"></i>The Uncomfortable Entrepreneurs: Bad Working Conditions And Entrepreneurial Commitment</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#spatial-surveillance-with-scan-statistics-by-controlling-the-false-discovery-rate"><i class="fa fa-check"></i>Spatial Surveillance With Scan Statistics By Controlling The False Discovery Rate</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#statistical-models-for-the-source-attribution-of-zoonotic-diseases-a-study-of-campylobacteriosis"><i class="fa fa-check"></i>Statistical Models For The Source Attribution Of Zoonotic Diseases: A Study Of Campylobacteriosis</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#towards-an-informal-test-for-goodness-of-fit"><i class="fa fa-check"></i>Towards An Informal Test For Goodness-Of-Fit</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#identifying-clusters-of-patients-with-diabetes-using-a-markov-birth-death-process"><i class="fa fa-check"></i>Identifying Clusters Of Patients With Diabetes Using A Markov Birth-Death Process</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#bayesian-temporal-density-estimation-using-autoregressive-species-sampling-models"><i class="fa fa-check"></i>Bayesian Temporal Density Estimation Using Autoregressive Species Sampling Models</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#how-does-the-textile-set-describe-geometric-structures-of-data"><i class="fa fa-check"></i>How Does The Textile Set Describe Geometric Structures Of Data?</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#intensity-estimation-of-spatial-point-processes-based-on-area-aggregated-data"><i class="fa fa-check"></i>Intensity Estimation Of Spatial Point Processes Based On Area-Aggregated Data</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#bayesian-inference-for-population-attributable-measures"><i class="fa fa-check"></i>Bayesian Inference For Population Attributable Measures</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#an-information-criterion-for-prediction-with-auxiliary-variables-under-covariate-shift"><i class="fa fa-check"></i>An Information Criterion For Prediction With Auxiliary Variables Under Covariate Shift</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#analysis-of-a-brief-telephone-intervention-for-problem-gambling-and-examining-the-impact-on-co-existing-depression"><i class="fa fa-check"></i>Analysis Of A Brief Telephone Intervention For Problem Gambling And Examining The Impact On Co-Existing Depression?</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#prior-based-bayesian-information-criterion"><i class="fa fa-check"></i>Prior-Based Bayesian Information Criterion</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#correlated-defaults-with-a-distance-to-default"><i class="fa fa-check"></i>Correlated Defaults With A Distance To Default</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#geographically-weighted-principal-component-analysis-for-spatio-temporal-statistical-dataset"><i class="fa fa-check"></i>Geographically Weighted Principal Component Analysis For Spatio-Temporal Statistical Dataset</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#dimensionality-reduction-of-multivariate-data-for-bayesian-analysis"><i class="fa fa-check"></i>Dimensionality Reduction Of Multivariate Data For Bayesian Analysis</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#an-ewma-chart-for-monitoring-covariance-matrix-based-on-dissimilarity-index"><i class="fa fa-check"></i>An EWMA Chart For Monitoring Covariance Matrix Based On Dissimilarity Index</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#adjusting-for-linkage-bias-in-the-analysis-of-record-linked-data"><i class="fa fa-check"></i>Adjusting For Linkage Bias In The Analysis Of Record-Linked Data</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#bayesian-semiparametric-hierarchical-models-for-longitudinal-data-analysis-with-application-to-dose-response-studies"><i class="fa fa-check"></i>Bayesian Semiparametric Hierarchical Models For Longitudinal Data Analysis With Application To Dose-Response Studies</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#optimizing-junior-rugby-weight-limits"><i class="fa fa-check"></i>Optimizing Junior Rugby Weight Limits</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#spatial-scan-statistics-for-matched-case-control-data"><i class="fa fa-check"></i>Spatial Scan Statistics For Matched Case-Control Data</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#whitebait-in-all-its-varieties-one-fish-two-fish-three-four-five-fish."><i class="fa fa-check"></i>Whitebait In All Its Varieties: One Fish, Two Fish, Three, Four, Five Fish.</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#latent-variable-models-and-multivariate-binomial-data"><i class="fa fa-check"></i>Latent Variable Models And Multivariate Binomial Data</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#asking-about-sex-in-general-health-surveys-comparing-the-methods-and-findings-of-the-2010-health-survey-for-england-with-those-of-the-third-national-survey-of-sexual-attitudes-and-lifestyles"><i class="fa fa-check"></i>Asking About Sex In General Health Surveys: Comparing The Methods And Findings Of The 2010 Health Survey For England With Those Of The Third National Survey Of Sexual Attitudes And Lifestyles</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#bayesian-continuous-space-time-model-of-burglaries"><i class="fa fa-check"></i>Bayesian Continuous Space-Time Model Of Burglaries</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#tolerance-limits-for-the-reliability-of-semiconductor-devices-using-longitudinal-data"><i class="fa fa-check"></i>Tolerance Limits For The Reliability Of Semiconductor Devices Using Longitudinal Data</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#model-checking-for-regressions-a-local-smoothing-based-global-smoothing-test"><i class="fa fa-check"></i>Model-Checking For Regressions: A Local Smoothing-Based Global Smoothing Test</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#breeding-value-estimation-in-partially-genotyped-populations"><i class="fa fa-check"></i>Breeding Value Estimation In Partially-Genotyped Populations</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#bivas-a-scalable-bayesian-method-for-bi-level-variable-selection"><i class="fa fa-check"></i>BIVAS: A Scalable Bayesian Method For Bi-Level Variable Selection</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#ranking-potential-shoplifters-in-real-time"><i class="fa fa-check"></i>Ranking Potential Shoplifters In Real Time</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#two-stage-approach-to-data-driven-subgroup-identification-in-clinical-trials"><i class="fa fa-check"></i>Two Stage Approach To Data-Driven Subgroup Identification In Clinical Trials</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#estimation-and-prediction-for-a-distribution-with-bathtub-shape-under-progressive-first-failure-censoring"><i class="fa fa-check"></i>Estimation And Prediction For A Distribution With Bathtub Shape Under Progressive First Failure Censoring</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#inverse-regression-for-multivariate-functional-data"><i class="fa fa-check"></i>Inverse Regression For Multivariate Functional Data</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#including-covariate-estimation-error-when-predicting-species-distributions-a-simulation-exercise-using-template-model-builder"><i class="fa fa-check"></i>Including Covariate Estimation Error When Predicting Species Distributions: A Simulation Exercise Using Template Model Builder</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#adjusted-adaptive-index-model-for-binary-response"><i class="fa fa-check"></i>Adjusted Adaptive Index Model For Binary Response</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#factors-influencing-on-growth-of-garments-industry-in-bangladesh"><i class="fa fa-check"></i>Factors Influencing On Growth Of Garments Industry In Bangladesh</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#comparison-of-exact-and-approximate-testing-procedures-in-clinical-trials-with-multiple-binary-endpoints"><i class="fa fa-check"></i>Comparison Of Exact And Approximate Testing Procedures In Clinical Trials With Multiple Binary Endpoints</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#bayesian-optimum-warranty-length-under-type-ii-unified-hybrid-censoring-scheme"><i class="fa fa-check"></i>Bayesian Optimum Warranty Length Under Type-II Unified Hybrid Censoring Scheme</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#multiple-function-on-function-linear-regression-with-application-to-weather-forecast-calibration"><i class="fa fa-check"></i>Multiple Function-On-Function Linear Regression With Application To Weather Forecast Calibration</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#modelling-the-distribution-of-lifetime-using-compound-time-homogenous-poisson-process"><i class="fa fa-check"></i>Modelling The Distribution Of Lifetime Using Compound Time-Homogenous Poisson Process</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#detecting-change-points-in-the-stress-strength-reliability-pxy"><i class="fa fa-check"></i>Detecting Change-Points In The Stress-Strength Reliability P(X&lt;Y)</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#new-zealand-crime-and-victims-survey-filling-the-knowledge-gap"><i class="fa fa-check"></i>New Zealand Crime And Victims Survey: Filling The Knowledge Gap</a></li>
<li class="chapter" data-level="" data-path="Tuesday.html"><a href="Tuesday.html#missing-data-in-randomised-control-trials-stepped-multiple-imputation"><i class="fa fa-check"></i>Missing Data In Randomised Control Trials: Stepped Multiple Imputation</a></li>
</ul></li>
<li><a href="Wednesday.html#Wednesday">Programme And Abstracts For Wednesday 13<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html"><i class="fa fa-check"></i>Promoting Your R Package</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#a-smoothing-filter-modelling-approach-for-time-series"><i class="fa fa-check"></i>A Smoothing Filter Modelling Approach For Time Series</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#online-learning-for-bayesian-nonparametrics-weakly-conjugate-approximation"><i class="fa fa-check"></i>Online Learning For Bayesian Nonparametrics: Weakly Conjugate Approximation</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#improving-the-production-cycle-at-stats-nz-with-rstudio"><i class="fa fa-check"></i>Improving The Production Cycle At Stats NZ With RStudio</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#sparse-common-component-analysis"><i class="fa fa-check"></i>Sparse Common Component Analysis</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#random-search-global-optimization-using-random-forests"><i class="fa fa-check"></i>Random Search Global Optimization Using Random Forests</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#gridsvg-then-and-now"><i class="fa fa-check"></i>gridSVG: Then And Now</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#probabilistic-outlier-detection-and-visualization-of-smart-meter-data"><i class="fa fa-check"></i>Probabilistic Outlier Detection And Visualization Of Smart Meter Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#the-joint-models-for-nonlinear-longitudinal-and-time-to-event-data-using-penalized-splines-a-bayesian-approach"><i class="fa fa-check"></i>The Joint Models For Nonlinear Longitudinal And Time-To-Event Data Using Penalized Splines: A Bayesian Approach</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#r-a-powerful-analysis-tool-to-improve-official-statistics-in-romania"><i class="fa fa-check"></i>R – A Powerful Analysis Tool To Improve Official Statistics In Romania</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#a-max-type-multivariate-two-sample-baumgartner-statistic"><i class="fa fa-check"></i>A Max-Type Multivariate Two-Sample Baumgartner Statistic</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#dimension-reduction-for-classification-of-high-dimensional-data-by-stepwise-svm"><i class="fa fa-check"></i>Dimension Reduction For Classification Of High-Dimensional Data By Stepwise SVM</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#bringing-multimix-from-fortran-to-r"><i class="fa fa-check"></i>Bringing Multimix From Fortran To R</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#specification-of-garch-model-under-asymmetric-error-innovations"><i class="fa fa-check"></i>Specification Of GARCH Model Under Asymmetric Error Innovations</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#performance-of-bayesian-credible-interval-for-binomial-proportion-using-logit-transformation"><i class="fa fa-check"></i>Performance Of Bayesian Credible Interval For Binomial Proportion Using Logit Transformation</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#statistical-disclosure-control-with-r-traditional-methods-and-synthetic-data"><i class="fa fa-check"></i>Statistical Disclosure Control With R: Traditional Methods And Synthetic Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#simultaneous-test-for-mean-vectors-and-covariance-matrices-in-high-dimensional-settings"><i class="fa fa-check"></i>Simultaneous Test For Mean Vectors And Covariance Matrices In High-Dimensional Settings</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#deep-learning-high-dimensional-covariance-matrices"><i class="fa fa-check"></i>Deep Learning High-Dimensional Covariance Matrices</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#r-in-industry-application-on-pipe-renewal-planning"><i class="fa fa-check"></i>R In Industry – Application On Pipe Renewal Planning</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#empirical-comparison-of-some-algorithms-for-automatic-univariate-arma-modeling-using-rcmdrplugin.spss"><i class="fa fa-check"></i>Empirical Comparison Of Some Algorithms For Automatic Univariate ARMA Modeling Using RcmdrPlugin.SPSS</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#imputation-of-the-2016-economic-census-for-business-activity-in-japan"><i class="fa fa-check"></i>Imputation Of The 2016 Economic Census For Business Activity In Japan</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#high-dimensional-asymptotics-for-the-naive-canonical-correlation-coefficient"><i class="fa fa-check"></i>High Dimensional Asymptotics For The Naive Canonical Correlation Coefficient</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#applying-active-learning-procedure-to-drug-consumption-data"><i class="fa fa-check"></i>Applying Active Learning Procedure To Drug Consumption Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#r-for-everything"><i class="fa fa-check"></i>R For Everything</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#r-package-for-new-two-stage-methods-in-forecasting-time-series-with-multiple-seasonality"><i class="fa fa-check"></i>R Package For New Two-Stage Methods In Forecasting Time Series With Multiple Seasonality</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#analysis-of-official-microdata-using-secure-statistical-computation-system"><i class="fa fa-check"></i>Analysis Of Official Microdata Using Secure Statistical Computation System</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#presenting-flexi-a-statistical-program-for-fitting-variance-models"><i class="fa fa-check"></i>Presenting Flexi, A Statistical Program For Fitting Variance Models</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#space-and-circular-time-log-gaussian-cox-processes-with-application-to-crime-event-data"><i class="fa fa-check"></i>Space And Circular Time Log Gaussian Cox Processes With Application To Crime Event Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#cluster-wise-regression-models-combined-by-a-quasi-linear-function"><i class="fa fa-check"></i>Cluster-Wise Regression Models Combined By A Quasi-Linear Function</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#hierarchical-structural-component-analysis-of-gene-environment-interactions"><i class="fa fa-check"></i>Hierarchical Structural Component Analysis Of Gene-Environment Interactions</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series"><i class="fa fa-check"></i>Wavelet-Based Power Transformation Of Non-Gaussian Long Memory Time Series</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#cross-covariance-estimation-for-integration-of-multi-omics-data"><i class="fa fa-check"></i>Cross Covariance Estimation For Integration Of Multi-Omics Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#relationships-between-linguistic-characteristics-and-the-use-of-maori-loanwords-in-new-zealand-english."><i class="fa fa-check"></i>Relationships Between Linguistic Characteristics And The Use Of Māori Loanwords In New Zealand English.</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#transfer-regression-and-predictive-distributions"><i class="fa fa-check"></i>Transfer Regression And Predictive Distributions</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#an-overview-of-the-correspondence-analysis-family"><i class="fa fa-check"></i>An Overview Of The Correspondence Analysis Family</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#testing-for-genetic-associations-in-arbitrarily-structured-populations"><i class="fa fa-check"></i>Testing For Genetic Associations In Arbitrarily Structured Populations</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#threshold-determination-for-the-meteorological-data-quality-control-in-korea"><i class="fa fa-check"></i>Threshold Determination For The Meteorological Data Quality Control In Korea</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#regularized-noise-reduction-methodology-for-high-dimensional-data"><i class="fa fa-check"></i>Regularized Noise-Reduction Methodology For High-Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#computation-of-influence-functions-for-robust-statistics"><i class="fa fa-check"></i>Computation Of Influence Functions For Robust Statistics</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#adaptive-model-averaging-in-high-dimensional-linear-regression"><i class="fa fa-check"></i>Adaptive Model Averaging In High-Dimensional Linear Regression</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#model-based-clustering-for-multivariate-categorical-data-with-dimension-reduction"><i class="fa fa-check"></i>Model-Based Clustering For Multivariate Categorical Data With Dimension Reduction</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#phylogenetic-tree-based-microbiome-association-test"><i class="fa fa-check"></i>Phylogenetic Tree-Based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#fitting-additive-hazards-model-using-calibrated-weights-for-case-cohort-data"><i class="fa fa-check"></i>Fitting Additive Hazards Model Using Calibrated Weights For Case-Cohort Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#selecting-the-number-of-principal-components"><i class="fa fa-check"></i>Selecting The Number Of Principal Components</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#rolling-survival-extrapolation-algorithm-for-estimating-life-years-lost-among-subjects-exposed-to-long-term-air-pollution"><i class="fa fa-check"></i>Rolling Survival Extrapolation Algorithm For Estimating Life Years Lost Among Subjects Exposed To Long-Term Air Pollution</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#enhancing-the-flexibility-of-regression-modeling-by-liquid-association"><i class="fa fa-check"></i>Enhancing The Flexibility Of Regression Modeling By Liquid Association</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#clusterwise-low-rank-correlation-analysis-based-on-majorization"><i class="fa fa-check"></i>Clusterwise Low-Rank Correlation Analysis Based On Majorization</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#bayesian-analysis-for-fitting-zero-inflated-count-data-with-data-augmentation"><i class="fa fa-check"></i>Bayesian Analysis For Fitting Zero-Inflated Count Data With Data Augmentation</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#towards-a-sparse-scalable-and-stably-positive-definite-inverse-covariance-estimator"><i class="fa fa-check"></i>Towards A Sparse, Scalable, And Stably Positive Definite (Inverse) Covariance Estimator</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#tick-by-tick-effect-on-the-inference-of-ultra-high-frequency-data"><i class="fa fa-check"></i>Tick-By-Tick Effect On The Inference Of Ultra-High Frequency Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#high-mortality-predictions-with-lines-or-curves-fitted-to-over-dispersed-exposure-mortality-data"><i class="fa fa-check"></i>High Mortality Predictions With Lines Or Curves Fitted To Over-Dispersed Exposure-Mortality Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#lattice-polytope-samplers"><i class="fa fa-check"></i>Lattice Polytope Samplers</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#statistical-modelling-and-analysis-of-cosmic-microwave-background-data"><i class="fa fa-check"></i>Statistical Modelling And Analysis Of Cosmic Microwave Background Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#measure-of-departure-from-marginal-average-point-symmetry-for-two-way-contingency-tables-with-ordered-categories"><i class="fa fa-check"></i>Measure Of Departure From Marginal Average Point-Symmetry For Two-Way Contingency Tables With Ordered Categories</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#sparse-estimates-from-dense-precision-matrix-posteriors"><i class="fa fa-check"></i>Sparse Estimates From Dense Precision Matrix Posteriors</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#dimension-reduction-strategies-for-modeling-bi-clustered-high-dimensional-data"><i class="fa fa-check"></i>Dimension Reduction Strategies For Modeling Bi-Clustered High Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#computing-entropies-with-nested-sampling"><i class="fa fa-check"></i>Computing Entropies With Nested Sampling</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#spline-based-drift-models-for-high-temperature-operating-life-tests"><i class="fa fa-check"></i>Spline-Based Drift Models For High Temperature Operating Life Tests</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#a-new-approach-to-distribution-free-tests-in-contingency-tables"><i class="fa fa-check"></i>A New Approach To Distribution Free Tests In Contingency Tables</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#a-bayesian-inference-for-time-series-via-copula-based-markov-chain-models"><i class="fa fa-check"></i>A Bayesian Inference For Time Series Via Copula-Based Markov Chain Models</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#modified-gene-shaving-algorithm---a-dimension-reduction-and-clustering-method"><i class="fa fa-check"></i>Modified Gene Shaving Algorithm - A Dimension REduction And Clustering Method</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#a-simple-method-to-construct-confidence-bands-in-functional-linear-regression"><i class="fa fa-check"></i>A Simple Method To Construct Confidence Bands In Functional Linear Regression</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#separation-of-symmetry-for-square-contingency-tables-with-ordinal-categories"><i class="fa fa-check"></i>Separation Of Symmetry For Square Contingency Tables With Ordinal Categories</a></li>
<li class="chapter" data-level="" data-path="Wednesday.html"><a href="Wednesday.html#testing-for-presence-of-clustering-effect-in-multilevel-model-with-high-dimensional-predictors"><i class="fa fa-check"></i>Testing For Presence Of Clustering Effect In Multilevel Model With High-Dimensional Predictors</a></li>
</ul></li>
<li><a href="Thursday.html#Thursday">Programme And Abstracts For Thursday 14<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html"><i class="fa fa-check"></i>ALTREP: Alternate Representations Of Basic R Objects</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#penalized-vector-generalized-additive-models"><i class="fa fa-check"></i>Penalized Vector Generalized Additive Models</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#a-package-for-multiple-precision-floating-point-computation-on-r"><i class="fa fa-check"></i>A Package For Multiple Precision Floating-Point Computation On R</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#dissimilarities-between-groups-of-data"><i class="fa fa-check"></i>Dissimilarities Between Groups Of Data</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#comparison-of-tests-of-mean-difference-in-longitudinal-data-based-on-block-resampling-methods"><i class="fa fa-check"></i>Comparison Of Tests Of Mean Difference In Longitudinal Data Based On Block Resampling Methods</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#ssrem-a-summary-statistics-based-random-effect-model-to-estimating-heritability-co-heritability-and-effect-sizes-in-gwas-data-analysis"><i class="fa fa-check"></i>SSREM: A Summary-Statistics-Based Random Effect Model To Estimating Heritability, Co-Heritability And Effect Sizes In GWAS Data Analysis</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#consistency-of-linear-mixed-effects-model-selection-with-inconsistent-covariance-parameter-estimators"><i class="fa fa-check"></i>Consistency Of Linear Mixed-Effects Model Selection With Inconsistent Covariance Parameter Estimators</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#an-incomplete-data-fisher-scoring-with-an-acceleration-method"><i class="fa fa-check"></i>An Incomplete-Data Fisher Scoring With An Acceleration Method</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#interactive-visualization-of-aggregated-symbolic-data"><i class="fa fa-check"></i>Interactive Visualization Of Aggregated Symbolic Data</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#analysis-of-spatial-data-with-a-gaussian-mixture-markov-random-field-model"><i class="fa fa-check"></i>Analysis Of Spatial Data With A Gaussian Mixture Markov Random Field Model</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#forward-selection-in-regression-models-based-on-robust-estimation"><i class="fa fa-check"></i>Forward Selection In Regression Models Based On Robust Estimation</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#selecting-generalised-linear-models-under-inequality-constraints"><i class="fa fa-check"></i>Selecting Generalised Linear Models Under Inequality Constraints</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#improvement-of-computation-for-nonlinear-multivariate-methods"><i class="fa fa-check"></i>Improvement Of Computation For Nonlinear Multivariate Methods</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#feature-selection-in-high-dimensional-models-with-complex-block-structures"><i class="fa fa-check"></i>Feature Selection In High-Dimensional Models With Complex Block Structures</a></li>
<li class="chapter" data-level="" data-path="Thursday.html"><a href="Thursday.html#statistical-generalized-derivative-applied-to-the-profile-likelihood-estimation-in-a-mixture-of-semiparametric-models"><i class="fa fa-check"></i>Statistical Generalized Derivative Applied To The Profile Likelihood Estimation In A Mixture Of Semiparametric Models</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">New Zealand Statistical Association and the International Association of Statistical Computing (Asian Regional Section) Joint Conference 2017</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="Monday" class="section level1 unnumbered">
<h1>Programme And Abstracts For Monday 11<sup>th</sup> Of December</h1>
<div id="talk_201">
<p class="keynoteBanner">
Keynote: Monday 11<sup>th</sup> 9:10 098 Lecture Theatre (260-098)
</p>
</div>
<div id="r-in-times-of-growing-user-base-and-data-sizes" class="section level2 unnumbered">
<h2>R In Times Of Growing User Base And Data Sizes</h2>
<p style="text-align:center">
Simon Urbanek<br /> AT&amp;T Labs<br />
</p>
<p><span><strong>Abstract:</strong></span> R has been historically used mainly on single machines, the analyst performing both analysis and visualization locally. However, the flexible abstraction of graphics in R and its extensibility makes R a great tool to be used remotely and across large clusters. The sizes of datasets as well as the popularity of R have created a demand for extending R’s capabilities beyond single machine. In this talk we will illustrate how R can be used by many users in a collaborative open-source RCloud environment to share data analyses, visualizations and results openly. The design also allows scaling across many instances. At the same time this environment can be combined with distributed computing to scale not only with the number of users but also with the size of datasets. In the second part of the talk we will show several approaches how R can be used very efficiently for Big Data analytics at scale leveraging the Hadoop ecosystem. We will start with hmr - a faster way to use the map/reduce framework from R, introduce ROctopus which allows us to perform arbitrary operations on large data without the constraints of a map/reduce framework and show a general framework for developing and using models in R that can leverage distributed systems. We will illustrate the use of the approaches on real dataset and a large cluster.</p>
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_022">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:30 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="robust-principal-expectile-component-analysis" class="section level2 unnumbered">
<h2>Robust Principal Expectile Component Analysis</h2>
<p style="text-align:center">
Liang-Ching Lin<sup>1</sup>, Ray Bing Chen<sup>1</sup>, Mong-Na Lo Huang<sup>2</sup>, and Meihui Guo<sup>2</sup><br /> <sup>1</sup>National Cheng Kung University<br /> <sup>2</sup>National Sun Yat-sen University<br />
</p>
<p><span><strong>Abstract:</strong></span> Principal component analysis (PCA) is widely used in dimensionality reduction for high-dimensional data. It identifies principal components by sequentially maximizing the component score variance around the mean. However, in many applications, one is interested in capturing the tail variations of the data rather than variation around the center. To capture the tail characteristics, Tran et al. (2016), based on an asymmetric <span class="math inline">\(L_2\)</span> norm, proposed principle expectile components (PECs). In this study, we introduce a new method called Huber-type principal expectile component (HPEC) using an asymmetric Huber norm to produce robust PECs. The statistical properties of HPECs are derived, and a derivative free optimization approach, particle swarm optimization (PSO), is used to find HPECs. As a demonstration, HPEC is applied to real and simulated data with encouraging results.</p>
<p><span><strong>Keywords:</strong></span> asymmetric norm, expectile, Huber’s criterion, particle swarm optimization, principle component</p>
<p><span><strong>References:</strong></span></p>
Tran, N. M., Burdejová, P., Osipenko, M. and Hárdle, W. K. (2016). <em>Principal Component Analysis in an Asymmetric Norm.</em> SFB 649 Discussion Paper 2016-040, Sonderforschungsbereich 649, Humboldt Universitát zu Berlin, Germany.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_002">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:30 OGGB4 (260-073)
</p>
</div>
</div>
<div id="effect-of-area-level-deprivation-on-body-mass-index-analysis-of-nz-health-surveys" class="section level2 unnumbered">
<h2>Effect Of Area Level Deprivation On Body Mass Index: Analysis Of NZ Health Surveys</h2>
<p style="text-align:center">
Andrew Adiguna Halim, Arindam Basu, and Raymond Kirk<br /> Unversity of Canterbury<br />
</p>
<p><span><strong>Abstract:</strong></span> Obesity is a growing public health problem in New Zealand but the trends of its determinants are unclear. We obtained the confidentialised unit record files (CURF) of the New Zealand Health Surveys (NZHS) from the Statistics New Zealand, containing multiple sets of anonymised individual level data from 2002/03 to 2014/15. We assessed the association between deprivation quintile and compliance with the dietary guideline, and the prevalence of overweight/obesity. For adults, we converted Body Mass Index (BMI) variable into tertiles. Then we regressed the BMI tertiles on deprivation level, ethnicity, age, sex, physical activity, education, smoking status, fruit guideline, vegetable guideline, and household income variables using stepwise ordinal logistic regression with complex survey design. We regressed the BMI categories on deprivation level, ethnicity, age, sex, household income, education, fruit guideline, vegetable guideline, soft drink consumption, and fast food consumption in the child data. We found that people living in the highest deprivation quintile were more likely to be in the higher BMI tertile in adults and BMI category in children compared with those living in the lowest deprivation quintile after adjusting for other confounding variables. For adults and children the ORs (95% CI) were 1.349 (95% CI: 1.240-1.468, p&lt;0.001) and 1.803 (95% CI: 1.531-2.125, p&lt;0.001) respectively. In contrast, the ORs (95% CI) for meeting the fruit and vegetable guidelines in adults were 0.968 (95% CI: 0.933-1.005, p: 0.088) and 1.029 (95%CI: 0.988-1.072, p: 0.172) respectively. The ORs (95% CI) for meeting the fruit and vegetable guidelines in children were 0.931 (95% CI: 0.843-1.029, p: 0.164) and 0.994 (95% CI: 0.908-1.088, p: 0.893) respectively. These results suggest that deprivation independently influences BMI, and the effect of meeting dietary guidelines are confounded by deprivation.</p>
<span><strong>Keywords:</strong></span> obesity, BMI, dietary guideline, deprivation, r statistics, proportional odds regression, survey complex design
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_012">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:30 OGGB5 (260-051)
</p>
</div>
</div>
<div id="calendar-based-graphics-for-visualising-peoples-daily-schedules" class="section level2 unnumbered">
<h2>Calendar-Based Graphics For Visualising People’s Daily Schedules</h2>
<p style="text-align:center">
Earo Wang, Dianne Cook, and Rob Hyndman<br /> Monash University<br />
</p>
<p><span><strong>Abstract:</strong></span> This paper describes a <code>frame_calendar</code> function that organises and displays temporal data, collected on sub-daily resolution, into a calendar layout. Calendars are broadly used in society to display temporal information, and events. The <code>frame_calendar</code> uses linear algebra on the date variable to create the layout. It utilises the grammar of graphics to create the plots inside each cell, and thus synchronises neatly with <span><strong>ggplot2</strong></span> graphics. The motivating application is studying pedestrian behaviour in Melbourne, Australia, based on counts which are captured at hourly intervals by sensors scattered around the city. Faceting by the usual features such as day and month, was insufficient to examine the behaviour. Making displays on a monthly calendar format helps to understand pedestrian patterns relative to events such as work days, weekends, holidays, and special events. The layout algorithm has several format options and variations. It is implemented in the R package <span><strong>sugrrants</strong></span>.</p>
<p><span><strong>Keywords:</strong></span> data visualisation, statistical graphics, time series, R package, grammar of graphics</p>
<p><span><strong>References:</strong></span></p>
<p>Van Wijk JJ, Van Selow ER (1999). Cluster and Calendar Based Visualization of Time Series Data. In <em>Information Visualization, 1999.(Info Vis’ 99) Proceedings</em>. 4–9.</p>
<p>Wickham H (2009). <em>ggplot2: Elegant Graphics for Data Analysis.</em> Springer-Verlag New York, New York, NY.</p>
Wickham H, Hofmann H, Wickham C, Cook D (2012). Glyph-maps for Visually Exploring Temporal Patterns in Climate Data and Models. <em>Environmetrics</em>, <strong>23</strong>(5), 382–393.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_036">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:30 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="nonparametric-test-for-volatility-in-clustered-multiple-time-series" class="section level2 unnumbered">
<h2>Nonparametric Test For Volatility In Clustered Multiple Time Series</h2>
<p style="text-align:center">
Paolo Victor Redondo and Erniel Barrios<br /> University of the Philippines Diliman<br />
</p>
<p><span><strong>Abstract:</strong></span> We proposed a test for volatility in clustered multiple time series based on sieve bootstrap. Clustering of observations is intended to capture contagion effect in multiple time series data, assumed to be present in the data generating process where the test is based from. We designed a simulation study to evaluate the test procedure. The method is further illustrated using data on global stock prices and rice production among Asian countries. The test is potentially robust to some distributional assumption but is possibly affected by the nature of volatility.</p>
<span><strong>Keywords:</strong></span> multiple time series; volatility; nonparametric test; Sieve Bootstrap
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_122">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:30 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="igess-a-statistical-approach-to-integrating-individual-level-genotype-data-and-summary-statistics-in-genome-wide-association-studies" class="section level2 unnumbered">
<h2>IGESS: A Statistical Approach To Integrating Individual Level Genotype Data And Summary Statistics In Genome Wide Association Studies</h2>
<p style="text-align:center">
Mingwei Dai<sup>1</sup>, Jingsi Ming<sup>2</sup>, Mingxuan Cai<sup>2</sup>, Jin Liu<sup>3</sup>, Can Yang<sup>4</sup>, Xiang Wan<sup>2</sup>, and Zongben Xu<sup>1</sup><br /> <sup>1</sup>Xi’an Jiaotong University<br /> <sup>2</sup>Hong Kong Baptist University<br /> <sup>3</sup>Duke-NUS Medical School<br /> <sup>4</sup>Hong Kong University of Science and Technology<br />
</p>
<p><span><strong>Abstract:</strong></span> Recent genome-wide association studies (GWAS) suggests that a complex phenotype is often affected by many variants with small effects, known as “polygenicity”. Tens of thousands of samples are often required to ensure statistical power of identifying these variants with small effects. In this study, we propose a statistical approach, IGESS, to increasing statistical power of identifying risk variants and improving accuracy of risk prediction by integrating individual level genotype data and summary statistics. An efficient algorithm based on variational inference is developed to handle genome-wide-scale analysis. Through comprehensive simulation studies, we demonstrated the advantages of IGESS over the methods which take either individual level data or summary statistics data as input. We applied IGESS to perform integrative analysis of Crohn’s Disease from WTCCC and summary statistics from other studies. IGESS was able to significantly increase statistical power of identifying risk variants and improve risk prediction accuracy.</p>
<span><strong>Keywords:</strong></span> GWAS, functional annotations, variational inference
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_128">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:30 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="author-name-identification-for-evaluating-research-performance-of-institutes" class="section level2 unnumbered">
<h2>Author Name Identification For Evaluating Research Performance Of Institutes</h2>
<p style="text-align:center">
Tomokazu Fujino<sup>1</sup>, Keisuke Honda<sup>2</sup>, and Hiroka Hamada<sup>2</sup><br /> <sup>1</sup>Fukuoka Women’s University<br /> <sup>2</sup>Institute of Statistical Mathematics<br />
</p>
<p><span><strong>Abstract:</strong></span> In this paper, we propose a new framework to extract a complete list of the articles written by researchers who belong to a specific research or educational institute from an academic document database such as Web of Science and Scopus. In this framework, it is necessary to perform author name identification because the query for the database is based on the author’s name to extract documents written before the author comming to the current institute. The framework is based on the latent dirichlet allocation (LDA), which is a kind of topic modeling, and some techniques and indices such as synonym retrieval and inverse document frequency (IDF) are used for enhancing the framework.</p>
<p><span><strong>Keywords:</strong></span> Institutional Research, Topic Modeling, Latent Dirichlet Allocation</p>
<p><span><strong>References:</strong></span></p>
<p>Tang, L. and Walsh, J. P. (2010). Bibliometric fingerprints: name disambiguation based on approximate structure equivalence of cognitive maps. <em>Scientometrics</em>, 84(3), 763–784.</p>
<p>Strotmann, A., Zhao, D. and Bubela, T. (2009). Author name disambiguation for collaboration network analysis and visualization. <em>Proc. American Society for Information Science and Technology</em>, 46(1), 1–20.</p>
Soler, J. M. (2007). Separating the articles of authors with the same name. <em>Scientometrics</em>, 72(2), 281–290.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_155">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:50 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="a-computational-tool-for-detecting-copy-number-variations-from-whole-genome-and-targeted-exome-sequencing" class="section level2 unnumbered">
<h2>A Computational Tool For Detecting Copy Number Variations From Whole Genome And Targeted Exome Sequencing</h2>
<p style="text-align:center">
Yu-Chung Wei<sup>1</sup> and Guan-Hua Huang<sup>2</sup><br /> <sup>1</sup>Feng Chia University<br /> <sup>2</sup>National Chiao Tung University<br />
</p>
<p><span><strong>Abstract:</strong></span> Copy number variations (CNVs) are genomic structural mutations with abnormal gene fragment copies. Current CNV detection algorithms for next generation sequencing (NGS) are developed for specific genome targets, including whole genome sequencing and targeted exome sequencing based on the differently data types and corresponding assumptions. Many whole genome tools assume the continuity of search space and reads uniform coverage across the genome. However, these assumptions break down in the exome capture because of discontinuous segments and exome specific functional biases. In order to develop a method adapting to both data types, we specify the large unconsidered genomic fragments as gaps to preserve the truly location information. A Bayesian hierarchical model was built and an efficient reversible jump Markov chain Monte Carlo inference algorithm was utilized to incorporate the gap information. The performance of gap settings for the Bayesian procedure was evaluated and compared with competing approaches using both simulations and real data.</p>
<span><strong>Keywords:</strong></span> Bayesian inference, Bioinformatics, copy number variation, next generation sequencing, reversible jump Markov chain Monte Carlo
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_021">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:50 OGGB4 (260-073)
</p>
</div>
</div>
<div id="clustering-using-nonparametric-mixtures-and-mode-identification" class="section level2 unnumbered">
<h2>Clustering Using Nonparametric Mixtures And Mode Identification</h2>
<p style="text-align:center">
Shengwei Hu and Yong Wang<br /> University of Auckland<br />
</p>
<p><span><strong>Abstract:</strong></span> Clustering aims to partition a set of observations into a proper number of clusters with similar objects allocated to the same group. Current partitioning methods mainly include those based on some measure of distance or probability distribution. Here we propose a mode-based clustering methodology motivated via density estimation and mode identification procedures. The idea is to estimate the data-generating probability distribution using a nonparametric density estimator and then locate the modes of the density obtained. In the nonparametric mixture models, each mode and the observations ascend to it correspond to a single cluster. Thus, the problem of determining the number of clusters can be recast as a mode merging problem. A criterion of measuring the separability between modes is also addressed in this work. The most similar modes would be merged sequentially until the optimal number of clusters is reached. The performance of the proposed method is investigated on both simulated and real datasets.</p>
<p><span><strong>Keywords:</strong></span> Clustering, Nonparametric mixtures, Mode identification</p>
<p><span><strong>References:</strong></span></p>
<p>Wang, X. and Wang, Y.: <em>Nonparametric multivariate density estimation using mixtures</em>. Stat. Comput. <strong>25</strong>, 349–-364 (2015).</p>
Li, J., Ray S. and Lindsay B.G.: <em>A nonparametric statistical approach to clustering via mode identification</em>. Journal of Machine Learning Research. <strong>8</strong>, 1687–-1723 (2007).
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_037">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:50 OGGB5 (260-051)
</p>
</div>
</div>
<div id="bayesian-curve-fitting-for-discontinuous-function-using-overcomplete-representation-with-multiple-kernels" class="section level2 unnumbered">
<h2>Bayesian Curve Fitting For Discontinuous Function Using Overcomplete Representation With Multiple Kernels</h2>
<p style="text-align:center">
Youngseon Lee<sup>1</sup>, Shuhei Mano<sup>2</sup>, and Jaeyong Lee<sup>1</sup><br /> <sup>1</sup>Seoul National University<br /> <sup>2</sup>Institute of Statistical Mathematics<br />
</p>
<p><span><strong>Abstract:</strong></span> We propose a new Bayesian methodology for estimating discontinuous functions. In this model, the estimated function is expressed by the overcomplete representation with multiple kernels. Therefore, the complex shape of functions can be expressed by the much smaller number of parameters due to the nature of the sparseness. It does not need any assumptions about the location of discontinuities, the smoothness of the function, the number of features. The form of the function taking all of these into account is determined naturally by the random Levy measure. Simulation data and real data analysis show that this model is suitable for fitting discontinuous functions. We also proved theoretical properties about the support of the function space having jumps in this paper.</p>
<p><span><strong>Keywords:</strong></span> Bayesian, nonparametric regression, discontinuous curve fitting, overcomplete, multiple kernel, Levy random field</p>
<p><span><strong>References:</strong></span></p>
<p>Chu, J. H., Clyde, M. A., and Liang, F. (2009). Bayesian function estimation using continuous wavelet dictionaries, <em>Statistica Sinica</em>, 1419–1438</p>
<p>Clyde, M. A., and Wolpert, R. L. (2007). Nonparametric function estimation using overcomplete dictionaries, <em>Bayesian Statistics</em>, <strong>8</strong>, 91–114.</p>
<p>Green, Peter J. (1995). Reversible jump Markov chain Monte Carlo computation and Bayesian model determination, <em>Biometrika</em>, <strong>82(4)</strong>, 711–732.</p>
<p>Khinchine, Alexander Ya and L<span>é</span>vy, Paul (1936). Sur les lois stables, <em>CR Acad. Sci. Paris</em>, <strong>202</strong>, 374–376.</p>
<p>M<span>ü</span>ller, P., and Quintana, F. A. (2004). Nonparametric Bayesian data analysis, <em>Statistical science</em>, 95–110</p>
<p>Pillai, N. S., Wu, Q., Liang, F., Mukherjee, S., and Wolpert, R. L. (2007). Characterizing the function space for Bayesian kernel models, <em>Journal of Machine Learning Research</em>, <strong>8</strong>, 1769–1797.</p>
<p>Qiu, Peihua (2011). <em>Jump Regression Analysis</em>. Springer.</p>
Wolpert, R. L., Clyde, M. A., and Tu, C. (2011). Stochastic expansions using continuous dictionaries: L<span>é</span>vy adaptive regression kernels, The <em>Annals of Statistics</em>, 1916–1962.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_030">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:50 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="estimation-of-a-semiparametric-spatiotemporal-models-with-mixed-frequency" class="section level2 unnumbered">
<h2>Estimation Of A Semiparametric Spatiotemporal Models With Mixed Frequency</h2>
<p style="text-align:center">
Vladimir Malabanan, Erniel Barrios, and Joseph Ryan Lansangan<br /> University of the Philippines Diliman<br />
</p>
<p><span><strong>Abstract:</strong></span> A semiparametric spatiotemporal model is postulated with data measured at varying frequency. The model optimizes utilization of information from variables measured at higher frequency by estimating its nonparametric effect on the response through the backfitting algorithm. Simulation studies support the optimality of the model over simple generalized additive model with aggregation of high frequency data. The method is then used in analyzing the spatiotemporal dynamics of corn yield based on some remotely-sensed data as covariates.</p>
<span><strong>Keywords:</strong></span> spatiotemporal model, semiparametric model, backfitting, mixed frequency
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_148">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:50 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="lsmm-a-statistical-approach-to-integrating-functional-annotations-with-genome-wide-association-studies" class="section level2 unnumbered">
<h2>LSMM: A Statistical Approach To Integrating Functional Annotations With Genome-Wide Association Studies</h2>
<p style="text-align:center">
Jingsi Ming<sup>1</sup>, Mingwei Dai<sup>2</sup>, Mingxuan Cai<sup>1</sup>, Xiang Wan<sup>1</sup>, Jin Liu<sup>3</sup>, and Can Yang<sup>4</sup><br /> <sup>1</sup>Hong Kong Baptist University<br /> <sup>2</sup>Xi’an Jiaotong University<br /> <sup>3</sup>Duke-NUS Medical School<br /> <sup>4</sup>Hong Kong University of Science and Technology<br />
</p>
<p><span><strong>Abstract:</strong></span> Thousands of risk variants underlying complex phenotypes have been identified in genome-wide association studies (GWAS). However, there are two major challenges towards fully characterizing the biological basis of complex diseases. First, many complex traits are suggested to be highly polygenic, whereas a large proportion of risk variants with small effects remains unknown. Second, the functional roles of the majority of GWAS hits in the non-coding region is largely unclear. In this paper, we propose a latent sparse mixed model (LSMM) to address the challenges by integrating functional annotations with summary statistics from GWAS. An efficient variational expectation-maximization (EM) algorithm is developed. We conducted comprehensive simulation studies and then applied it to 30 GWAS of complex phenotypes integrating 9 genic annotation categories and 127 tissue-specific functional annotations from the Roadmap project. The results demonstrate that LSMM is not only able to increase the statistical power to identify risk variants, but also provide a deeper understanding of genetic architecture of complex traits by detecting relevant functional annotations.</p>
<span><strong>Keywords:</strong></span> GWAS, functional annotations, variational inference
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_187">
<p class="contribBanner">
Monday 11<sup>th</sup> 10:50 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="a-study-of-the-influence-of-articles-in-the-large-scale-citation-network" class="section level2 unnumbered">
<h2>A Study Of The Influence Of Articles In The Large-Scale Citation Network</h2>
<p style="text-align:center">
Frederick Kin Hing Phoa<sup>1</sup> and Livia Lin Hsuan Chang<sup>2</sup><br /> <sup>1</sup>Academia Sinica<br /> <sup>2</sup>Institute of Statistical Mathematics<br />
</p>
<span><strong>Abstract:</strong></span> Nowadays there are many research metrics at the author-, article-, journal-levels, like the impact factors and many others. However, none of them possess a universally meaningful interpretation on the research influence at all levels, not mentioning that many are subject-biased and consider neighboring relations only. In this work, we introduce a new network-based research metric called the network influence. It utilizes all information in the whole network and it is universal to any levels. Due to its statistical origin, this metric is computationally efficient and statistically interpretable even if one applies it to a large-scale network. This work demonstrates the analysis of networks via network influence using a large-scale citation database called the Web of Science. By just considering the articles among statistics community in 2005-2014, the network influence of all articles are calculated and compared, resulting in a top-ten important articles that are slightly different from the list via impact factors. This metric can be easily extended to author citation network and many similar networks embedded in the Web of Science.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_203">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:10 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="estimating-links-of-a-network-from-time-to-event-data" class="section level2 unnumbered">
<h2>Estimating Links Of A Network From Time To Event Data</h2>
<p style="text-align:center">
Tso-Jung Yen<br /> Academia Sinica<br />
</p>
<span><strong>Abstract:</strong></span> In this paper we develop a statistical method for identifying links of a network from time to event data. This method models the hazard function of a node conditional on event time of other nodes, parameterizing the conditional hazard function with the links of the network. It then estimates the hazard function by maximizing a pseudo partial likelihood function with parameters subject to a user‐specified penalty function and additional constraints. To make such estimation robust, it adopts a prespecified risk control on the number of false discovered links by using the Stability Selection method. Simulation study shows that under this hybrid procedure, the number of false discovered links is tightly controlled while the true links are well recovered. We apply our method to estimate a political cohesion network that drives donation behavior of 146 firms from the data collected during the 2008 Taiwanese legislative election. The results show that firms affiliated with elite organizations or firms of monopoly are more likely to diffuse donation behavior. In contrast, firms belonging to technology industry are more likely to act independently on donation.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_023">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:10 OGGB4 (260-073)
</p>
</div>
</div>
<div id="estimation-of-a-high-dimensional-covariance-matrix" class="section level2 unnumbered">
<h2>Estimation Of A High-Dimensional Covariance Matrix</h2>
<p style="text-align:center">
Xiangjie Xue and Yong Wang<br /> University of Auckland<br />
</p>
<p><span><strong>Abstract:</strong></span> The estimation of covariance or precision (inverse covariance) matrices plays a prominent role in multivariate analysis. The usual estimator, the sample covariance matrix, is known to be unstable and ill-conditioned in high-dimensional setting. In the past two decades, various methods have been developed to give a stable and well-conditioned estimator and they have their own advantages and disadvantages. We will review some of the most popular methods and describe a new method to estimate the correlation matrix and hence the covariance matrix using the empirical Bayes method. Similar to many element-wise methods in the literature, we also assume that the elements in a correlation matrix are independent of each other. We use the fact that the elements in a sample correlation matrix can be approximated by the same one-parameter normal distribution with unknown means , along with the non-parametric maximum likelihood estimation to give a new estimator of the correlation matrix. Preliminary simulation results show that the new estimator has some advantages over various thresholding methods in estimating sparse covariance matrices.</p>
<p><span><strong>Keywords:</strong></span> Big Data, Multivariate Analysis, Statistical Inference</p>
<p><span><strong>References:</strong></span></p>
<p>Efron, B., 2010. <em>Correlated <span class="math inline">\(z\)</span>-values and the accuracy of large-scale statistical estimates</em>. J Am Stat Assoc <strong>105</strong>, 1042 - 1055.</p>
<p>Fan, J., Liao, Y., Liu, H., 2016. <em>An overview of the estimation of large covariance and precision matrices</em>. Econometrics Journal <strong>19</strong>, C1 - C32.</p>
Wang, Y., 2007. <em>On fast computation of the non-parametric maximum likelihood estimate of a mixing distribution</em>. Journal of the Royal Statistical Society: Series B <strong>69</strong>, 185 - 198.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_055">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:10 OGGB5 (260-051)
</p>
</div>
</div>
<div id="innovative-bayesian-estimation-in-the-von-mises-distribution" class="section level2 unnumbered">
<h2>Innovative Bayesian Estimation In The von Mises Distribution</h2>
<p style="text-align:center">
Yuta Kamiya<sup>1</sup>, Toshinari Kamakura<sup>1</sup>, and Takemi Yanagimoto<sup>2</sup><br /> <sup>1</sup>Chuo University<br /> <sup>2</sup>Institute of Statistical Mathematics<br />
</p>
<p><span><strong>Abstract:</strong></span> In spite of recent growing interest in applying the von-Mises distribution to circular data in various scientific fields, researches on the parameter estimation are surprisingly sparse. The standard estimators are the MLE and the maximum marginal likelihood estimator (Schou 1978). Although Bayesian estimators are promising, it looks that they have not been fully developed. We propose the posterior mean of the canonical parameter, instead of the mean parameter, under the reference prior. This estimator satisfies an optimality property, and performs favorably for wide ranges of true parameters. Extensive simulation studies yield that the risks of the proposed estimator are significantly small, compared with the existing estimators. An interesting finding is that the estimating function for the dispersion parameter behaves reasonably. Notable advantages of the present approach are its straightforward extensions to various procedures, including Bayesian estimator under an informative prior based on the reference prior. The proposed estimator is examined by applying to practical datasets.</p>
<p><span><strong>Keywords:</strong></span> von-Mises distribution, bayesian estimation, canonical parameter</p>
<p><span><strong>References:</strong></span></p>
<p>Fisher, Nicholas I. <span><em>Statistical analysis of circular data.</em></span> Cambridge University Press, 1995.</p>
Schou, Geert. “Estimation of the concentration parameter in von Mises–Fisher distributions.” Biometrika 65.2 (1978): 369-377.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_035">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:10 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="evidence-of-climate-change-from-nonparametric-change-point-analysis" class="section level2 unnumbered">
<h2>Evidence Of Climate Change From Nonparametric Change-Point Analysis</h2>
<p style="text-align:center">
Angela Nalica, Paolo Redondo, Erniel Barrios, and Stephen Villejo<br /> University of the Philippines Diliman<br />
</p>
<p><span><strong>Abstract:</strong></span> Suppose that the time series data is sufficiently explained by a model, e.g., autoregressive model, transfer function model. A change-point is considered to exist if any of the model parameters is substantially different in two or more regimes. We proposed a test for existence of a change-point (assuming that location of the change is known) based on nonparametric bootstrap. The method is used in verifying whether the southern oscillation index exhibits change-point which is taken as an evidence of climate change. There is indeed an evidence of climate change in the period.</p>
<span><strong>Keywords:</strong></span> change-point analysis, block bootstrap, southern oscillation index (SOI)
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_161">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:10 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="joint-analysis-of-individual-level-genotype-data-and-summary-statistics-by-leveraging-pleiotropy" class="section level2 unnumbered">
<h2>Joint Analysis Of Individual Level Genotype Data And Summary Statistics By Leveraging Pleiotropy</h2>
<p style="text-align:center">
Mingwei Dai<sup>1</sup>, Jin Liu<sup>2</sup>, and Can Yang<sup>3</sup><br /> <sup>1</sup>Xi’an Jiaotong University<br /> <sup>2</sup>Duke-NUS Medical School<br /> <sup>3</sup>Hong Kong University of Science and Technology<br />
</p>
<p><span><strong>Abstract:</strong></span> Results from Genome-wide association studies (GWAS) suggest that a complex phenotype is often affected by many variants with small effects, known as “polygenicity”. Tens of thousands of samples are often required to ensure statistical power of identifying these variants with small effects. However, it is often the case that a research group can only get approval for the access to individual-level genotype data with a limited sample size (e.g., a few hundreds or thousands). Meanwhile, pleiotropy is a pervasive phenomenon in genetics whereby a DNA variant influences multiple traits, and summary statistics for genetically related traits (e.g., autoimmune diseases or psychiatric disorders) are becoming publicly available. The sample sizes associated with the summary statistics data sets are usually quite large. How to make the most efficient use of existing abundant data resources largely remains an open problem.</p>
<p>In this study, we propose a statistical approach, LEP, to increasing statistical power of identifying risk variants and improving accuracy of risk prediction by integrating individual level genotype data and summary statistics by veraging leiotropy. An efficient algorithm based on variational inference is developed to handle the genome-wide analysis. Through comprehensive simulation studies, we demonstrated the advantages of LEP over the methods which take either individual-level data or summary statistics data as input. We applied LEP to perform integrative analysis of several auto-immune diseases from WTCCC and summary statistics from other studies. LEP was able to significantly increase the statistical power of identifying risk variants and improve the risk prediction accuracy by jointly analyzing autoimmune diseases.</p>
<p><span><strong>Keywords:</strong></span> GWAS, pleiotropy, polygenicity, summary statistics, variational inference</p>
<p><span><strong>References:</strong></span></p>
<p>Solovieff N, Cotsapas C, Lee P H, et al. (2013) Pleiotropy in complex traits: challenges and strategies In: <em>Nature reviews. Genetics</em> 14(7): 483.</p>
<p>Carbonetto P, Stephens M. (2012) Scalable variational inference for Bayesian variable selection in regression, and its accuracy in genetic association studies In: <em>Bayesian analysis</em> 7(1): 73-108.</p>
<p>Chung D, Yang C, Li C, et al. (2014). GPA: a statistical approach to prioritizing GWAS results by integrating pleiotropy and annotation In: <em>PLoS genetics</em></p>
Dai M, Ming J, Cai M, et al. (2017). IGESS: a statistical approach to integrating individual-level genotype data and summary statistics in genome-wide association studies. In: <em>Bioinformatics</em>
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_157">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:10 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="an-advanced-approach-for-time-series-forecasting-using-deep-learning" class="section level2 unnumbered">
<h2>An Advanced Approach For Time Series Forecasting Using Deep Learning</h2>
<p style="text-align:center">
Balaram Panda<br /> Inland Revenue Department<br />
</p>
<p><span><strong>Abstract:</strong></span> Time series forecasting is a decade-long research and which is being evolving day by day. Due to the recent advancement is deep learning technique many of the complex problems have been solved using deep learning. Deep learning techniques have shown tremendous better performance in supervised learning problem. One of the reasons for this success is the ability of deep feedforward network methods to learn multiple feature interaction for a single instance. However, the time-dependent nature not being captured by deep feedforward network till the evolution of RNN(recurrent neural network) and LSTM(long short term memory) network architecture. This paper reveals the success of LSTM time series in comparison with ARIMA and other standard approaches for time series modeling. A sensitivity analysis is also conducted to explore the effect of hyper parameter tuning on LSTM model to reduce the time series forecasting error. We also derive practical advice from our empirical results for those interested in getting most out of LSTM time series for modern time series forecasting.</p>
<p><span><strong>Keywords:</strong></span> Deep Learning, Time Series, LSTM, RNN</p>
<p><span><strong>References:</strong></span></p>
<p>Längkvist, Martin, Lars Karlsson, and Amy Loutfi. “A review of unsupervised feature learning and deep learning for time-series modeling.” Pattern Recognition Letters 42 (2014): 11-24.</p>
Zheng, Yi, et al. “Time series classification using multi-channels deep convolutional neural networks.” International Conference on Web-Age Information Management. Springer, Cham, 2014.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_029">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:30 OGGB4 (260-073)
</p>
</div>
</div>
<div id="genetic-map-estimation-using-hidden-markov-models-in-the-presence-of-partially-observed-information" class="section level2 unnumbered">
<h2>Genetic Map Estimation Using Hidden Markov Models In The Presence Of Partially Observed Information</h2>
<p style="text-align:center">
Timothy Bilton<sup>1,2</sup>, Matthew Schofield<sup>1</sup>, Ken Dodds<sup>2</sup>, and Michael Black<sup>1</sup><br /> <sup>1</sup>University of Otago<br /> <sup>2</sup>AgResearch<br />
</p>
<p><span><strong>Abstract:</strong></span> A genetic linkage map shows the relative position of and genetic distance between markers, positions of the genome that exhibit variation, and underpins the study of species’ genomes in a number of scientific applications. Genetic maps are constructed by tracking the transmission of genetic information from individuals to their offspring, which is frequently modelled using a hidden Markov model (HMM) since only the expression and not the transmission of genetic information is observed. However, constructing genetic maps with data generated using the latest sequencing technology is complicated by the fact that some observations are only partially observed which, if unaccounted for, typically results in inflated estimates. We extend the HMM to model partially observed information by including an additional layer of latent variables. In addition, we investigate several different approaches for computing confidence intervals of the genetic map estimates obtained from the extended HMM. Results show that our model is able to produce accurate genetic map estimates, even in situations where a large proportion of the data is only partially observed. Our methodology has been implemented in the R package GusMap.</p>
<span><strong>Keywords:</strong></span> hidden Markov models, linkage mapping, partially observed data, confidence intervals
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_076">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:30 OGGB5 (260-051)
</p>
</div>
</div>
<div id="a-simple-method-for-grouping-patients-based-on-historical-doses" class="section level2 unnumbered">
<h2>A Simple Method For Grouping Patients Based On Historical Doses</h2>
<p style="text-align:center">
Shengli Tzeng<br /> China Medical University<br />
</p>
<p><span><strong>Abstract:</strong></span> Monitoring dose patterns over time helps physicians and patients learn more about metabolic change, disease evolution, etc. One way to turn such longitudinal data into clinically useful information is through cluster analysis, which aims to separate the “profiles of doses” among patients into homogeneous subgroups. Different doses patterns reflect heterogeneity in patients’ characteristics and effectiveness of therapy. However, not all patients were prescribed at regular time points, and missing values seems ubiquitous if one aligns records at distinct time points. Moreover, a few outliers may heavily influence the estimation for within and/or between variations of clusters, making the distinction among clusters blurred. In this study, a simple method based on a novel pairwise dissimilarity is proposed, which also serves as a screen tool to detect potential outliers. We use smoothing splines, handling data observed either at regular or irregular time points, and measure the dissimilarity between patients based on pairwise varying curve estimates with commutation of smoothing parameters. It takes into account the estimation uncertainty and is not strongly affected by outliers. The effectiveness of our proposal is shown by simulations comparing it to other dissimilarity measures and by a real application to methadone dosage maintenance levels.</p>
<p><span><strong>Keywords:</strong></span> Clustering, longitudinal data, smoothing splines, outliers</p>
<p><span><strong>References:</strong></span></p>
Lin, Chien-Ju, Christian Hennig, and Chieh-Liang Huang. (2016). Clustering and a dissimilarity measure for methadone dosage time series. In <em>Analysis of Large and Complex Data</em>, 31-41. Springer, Switzerland.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_065">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:30 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="semiparametric-mixed-analysis-of-covariance-model" class="section level2 unnumbered">
<h2>Semiparametric Mixed Analysis Of Covariance Model</h2>
<p style="text-align:center">
Virgelio Alao, Erniel Barrios, and Joseph Ryan Lansangan<br /> University of the Philippines Diliman<br />
</p>
<p><span><strong>Abstract:</strong></span> A semiparametric mixed analysis of covariance model is postulated and estimated using the two procedures: based on an imbedded restricted maximum likelihood (REML) and nonparametric regression (smoothing splines) estimation into the backfitting framework (ARMS); and infusing bootstrap into the ARMS (B-ARMS). The heterogeneous effect of covariates across the groups is postulated to affect the response through a nonparametric function to mitigate overparameterization. Using simulation studies, we exhibited the capability of the postulated model (and estimation procedures) in increasing predictive ability and stabilizing variance components estimates even for small sample size and with minimal covariate effect, and regardless of whether the model is correctly specified or there is misspecification error.</p>
<span><strong>Keywords:</strong></span> mixed ANCOVA model, nonparametric regression, backfitting, bootstrap, random effects, variance components
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_125">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:30 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="adaptive-false-discovery-rate-regression-with-application-in-integrative-analysis-of-large-scale-genomic-data" class="section level2 unnumbered">
<h2>Adaptive False Discovery Rate Regression With Application In Integrative Analysis Of Large-Scale Genomic Data</h2>
<p style="text-align:center">
Can Yang<br /> Hong Kong University of Science and Technology<br />
</p>
<p><span><strong>Abstract:</strong></span> Recent international projects, such as the Encyclopedia of DNA Elements (ENCODE) project, the Roadmap project and the Genotype-Tissue Expression (GTEx) project, have generated vast amounts of genomic annotation data, e.g., epigenome and transcriptome. There is great demanding of effective statistical approaches to integrate genomic annotations with the results from genome-wide association studies. In this talk, we introduce a statistical framework, named AdaFDR, for integrating multiple annotations to characterize functional roles of genetic variants that underlie human complex phenotypes. For a given phenotype, AdaFDR can adaptively incorporates relevant annotations for prioritization of genetic risk variants, allowing nonlinear effects among these annotations, such as interaction effects between genomic features. Specifically, we assume that the prior probability of a variant associated with the phenotype is a function of its annotations <span class="math inline">\(F(X)\)</span>, where <span class="math inline">\(X\)</span> is the collection of the annotation status and <span class="math inline">\(F(X)\)</span> is an ensemble of decision trees, i.e., <span class="math inline">\(F(X) = \sum_k f_k(X)\)</span> and <span class="math inline">\(f_k(X)\)</span> is a shallow decision tree. We have developed an efficient EM-Boosting algorithm for model fitting, where a shallow decision tree grows in a gradient-Boosting manner (Friedman J. 2001) at each EM-iteration. Our framework inherits the nice property of gradient boosted trees: (1) The gradient accent property of the Boosting algorithm naturally guarantees the convergence of our EM-Boosting algorithm. (2) Based on the fitted ensemble <span class="math inline">\(\hat{F}(X)\)</span>, we are able to rank the importance of annotations, measure the interaction among annotations and visualize the model via partial plots (Friedman J. 2008). Using AdaFDR, we performed integrative analysis of genome-wide association studies on human complex phenotypes and genome-wide annotation resources, e.g., Roadmap epigenome. The analysis results revealed interesting regulatory patterns of risk variants. These findings deepen our understanding of genetic architectures of complex phenotypes. The statistical framework developed here is also broadly applicable to many other areas for integrative analysis of rich data sets.</p>
<p><span><strong>Keywords:</strong></span> False Discovery Rate, integrative analysis, functional annotation, genomic data</p>
<p><span><strong>References:</strong></span></p>
<p>Friedman, Jerome H (2001). Greedy function approximation: a gradient boosting machine, <em>Annals of statistics</em>, <strong>29:5</strong>,1189–1232.</p>
Jerome H. Friedman and Bogdan E. Popescu (2008) Predictive Learning via Rule Ensembles <em>The Annals of Applied Statistics</em>, <strong>2:3</strong>, 916–954
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_159">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:30 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="structure-of-members-in-the-organization-to-induce-innovation-quantitatively-analyze-the-capability-of-the-organization" class="section level2 unnumbered">
<h2>Structure Of Members In The Organization To Induce Innovation: Quantitatively Analyze The Capability Of The Organization</h2>
<p style="text-align:center">
Yuji Mizukami<sup>1</sup> and Junji Nakano<sup>2</sup><br /> <sup>1</sup>Nihon University<br /> <sup>2</sup>Institute of Statistical Mathematics<br />
</p>
<p><span><strong>Abstract:</strong></span> Innovation is the act of creating new value by using “new connection”, “new point of view”, “new way of thinking”, “new usage method” (Schumpeter 1912). In recent years, the promotion of the Innovation has been strongly encouraged. In the field of research, attempts are also being made to create new value through connection between those fields. Moreover, along with the move to promote integration among these research fields, research is being conducted to grasp and promote the degree of them. In this research, for the purpose of providing indices for measuring the degree of them, we show indices quantitatively indicating the degree of fusion in different fields and the distance between the fields. Also, we have try to present indices for grasping the whole image based on the random graph.</p>
<p><span><strong>Keywords:</strong></span> Research Metrix, Institute Research, Co-author analysis</p>
<p><span><strong>References:</strong></span></p>
<p>Wagner, C. S., Roessner, J. D., Bobb, K., Klein, J. T., Boyack, K. W., Keyton, J. and Börner, K. (2011). <em>Approaches to understanding and measuring interdisciplinary scientific research: A review of the literature, Journal of Informetrics</em>. Vol. 5, No. 1, pp. 14-26.</p>
Mizukami, Y., Mizutani, Y., Honda, K., Suzuki, S., Nakano, J. (2017). <em>An International Research Comparative Study of the Degree of Cooperation between disciplines within mathematics and mathematical sciences, Behaviormetrika</em>, <strong>1</strong>, 19 pages, On-line.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_031">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:50 OGGB4 (260-073)
</p>
</div>
</div>
<div id="vector-generalized-linear-time-series-models" class="section level2 unnumbered">
<h2>Vector Generalized Linear Time Series Models</h2>
<p style="text-align:center">
Victor Miranda and Thomas Yee<br /> University of Auckland<br />
</p>
<p><span><strong>Abstract:</strong></span> Since the introduction of the ARMA class in the early 1970s many time series (TS) extensions have been proposed, e.g., vector ARMA and GARCH-type models for heteroscedasticity. The result has been a plethora of models having pockets of substructure but little overriding framework. In this talk we propose a class of TS models called Vector Generalized Linear Time Series Models (VGLTSM), which can be thought of as multivariate generalized linear models directed towards time series data. The crucial VGLM ideas are constraint matrices, vector responses and covariate-specific linear predictors, and estimation by iteratively reweighted least squares and Fisher scoring. The only addition to the VGLM framework is a log-likelihood that depends on past values. We show how several popular sub-classes of TS models are accommodated as special cases of VGLMs, as well as new work that broadens TS modelling even more. Algorithmic details of its implementation in , and properties such as stationarity, parameters depending on covariates, expected information matrices and cointegrated TS are surveyed.</p>
<p><span><strong>Keywords:</strong></span> VGLM, time series, Fisher scoring.</p>
<p><span><strong>References:</strong></span></p>
Yee, T. W. (2015) <em>Vector Generalized Linear and Additive Models: With an Implementation in R.</em> New York, USA: Springer.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_093">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:50 OGGB5 (260-051)
</p>
</div>
</div>
<div id="local-canonical-correlation-analysis-for-multimodal-labeled-data" class="section level2 unnumbered">
<h2>Local Canonical Correlation Analysis For Multimodal Labeled Data</h2>
<p style="text-align:center">
Seigo Mizutani and Hiroshi Yadohisa<br /> Doshisha University<br />
</p>
<p><span><strong>Abstract:</strong></span> In supervised learning, canonical correlation analysis (CCA) is widely used for dimension reduction problems. When using dimension reduction methods, researchers should always aim to preserve the data structure in a low dimensional space. However, if the obtained data are assumed to be multimodal labeled data, that is, each cluster can be subdivided into several latent clusters, CCA is rarely able to preserve the data structure in a low dimensional space.</p>
<p>In this study, we propose local CCA (LCCA) for multimodal labeled data. This method is based on local Fisher discriminant analysis (LFDA) (Sugiyama, 2007). We do not employ the same local covariance matrix of the explanatory variables as under LFDA, which uses a local between-group variance matrix and a local within-group variance matrix. Instead, in our proposed method, we use a covariance matrix of the explanatory variables as well as a weighted affinity matrix. The usefulness of LCCA in data visualization and clustering is then demonstrated by simulation studies.</p>
<p><span><strong>Keywords:</strong></span> Supervised learning, Dimension reduction, Local Fisher discriminant analysis (LFDA), Weighted affinity matrix</p>
<p><span><strong>References:</strong></span></p>
<p>Sugiyama, M. (2007). Dimensionality reduction of multimodal labeled data by local Fisher discriminant analysis. <em>Journal of Machine Learning Research</em>, <strong>8</strong>, 1027-1061.</p>
<p>Hastie, T. and Buja, A. and Tibshirani, R. (1995) Penalized discriminant analysis., 73-102.</p>
Hotelling, H. (1936). Relations between two sets of variates. <em>Biometrika</em>, <strong>28</strong>, 321-377.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_100">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:50 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="a-practitioners-guide-to-deep-learning-for-predictive-analytics-on-structured-data" class="section level2 unnumbered">
<h2>A Practitioners Guide To Deep Learning For Predictive Analytics On Structured Data</h2>
<p style="text-align:center">
Balaram Panda and Habib Baluwala<br /> Inland Revenue Department<br />
</p>
<p><span><strong>Abstract:</strong></span> Recently, deep learning techniques have shown remarkably strong performance in problems involving unstructured data (ex. text, image, and video). One of the reasons for this success is the ability of deep learning methods to learn multiple levels of abstraction and feature interaction. However, the advantages of using deep learning techniques for structured/ event/transactional data has not been studied in detail. The purpose of this paper is to review the advantages and limitations of using deep feed forward networks on structured data. This is achieved by comparing the performance of deep feed forward networks with conventional machine learning techniques applied on a large structured dataset for classification problem. The paper also describes methodologies for optimizing the deep feed forward networks to achieve better accuracy and different approaches to reduce over fitting for deep feed forward network. A sensitivity analysis is conducted to explore the effect of hyper parameter tuning on model performance. We also derive practical advice from our extensive empirical results for those interested in getting most out of deep feed forward networks for real world settings.</p>
<p><span><strong>Keywords:</strong></span> Deep Learning, deep feed forward networks, machine learning, R, Tensorflow, Python</p>
<p><span><strong>References:</strong></span></p>
<p>Bengio, Yoshua. “Learning deep architectures for AI.” Foundations and trends® in Machine Learning 2.1 (2009): 1-127.</p>
Goodfellow, Ian J., et al. “Maxout networks.” arXiv preprint arXiv:1302.4389 (2013).
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_190">
<p class="contribBanner">
Monday 11<sup>th</sup> 11:50 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="clustering-of-research-subject-based-on-stochastic-block-model" class="section level2 unnumbered">
<h2>Clustering Of Research Subject Based On Stochastic Block Model</h2>
<p style="text-align:center">
Hiroka Hamada<sup>1</sup>, Keisuke Honda<sup>1</sup>, Frederick Kin Hing Phoa<sup>2</sup>, and Junji Nakano<sup>1</sup><br /> <sup>1</sup>Institute of Statistical Mathematics<br /> <sup>2</sup>Academia Sinica<br />
</p>
<p><span><strong>Abstract:</strong></span> In this paper, we propose a new clustering method to measure influence of papers in all areas of science. To see structure of entire relationship we apply stochastic block model (SBM) on big scale citation network data. SBM generates a matrix which divides several blocks which represent relationship among research fields. We show this matrix can be used to visual exploratory analysis. When lists of papers are mapped this matrix we can get useful information by varied locations in visually. Elastic Map is used as dimension reduction method to calculate scalar value onto onto the corresponding principal points of each papers. We demonstrate that this projection score is can be used to evaluate divergence impact of papers across all field. To illustrate one application of our method, we analyze 450k+ articles published between 1981 and 2016 Web of Science data. In this beta version of our system, Edward, probabilistic programming language is used for estimation of SBM parameters and calculation of divergence score of papers.</p>
<p><span><strong>Keywords:</strong></span> Institutional Research, Stochastic Block Model, Elastic Map</p>
<p><span><strong>References:</strong></span></p>
<p>Nowicki,K. and Snijders,T. (2001). Estimation and prediction for stochastic block structures. <em>Journal of the American Statistical Association</em>, 96, 1077–1087.</p>
<p>Gorban,A. and Zinovyev,A. (2005). Elastic Principal Graphs and Manifolds and their Practical Applications. <em>Computing</em>, 75(4), 359–379.</p>
Tran,D., Kucukelbir,A., Dieng, A.B., Rudolph,M., Liang,D. and Blei,D.M. (2016). Edward: A library for probabilistic modeling, inference, and criticism. <em>arXiv preprint arXiv:1610.09787</em>.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_200">
<p class="keynoteBanner">
Keynote: Monday 11<sup>th</sup> 13:20 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="zen-and-the-art-of-workflow-maintenance" class="section level2 unnumbered">
<h2>Zen And The aRt Of Workflow Maintenance</h2>
<p style="text-align:center">
Jenny Bryan<br /> University of British Columbia<br />
</p>
<span><strong>Abstract:</strong></span> My talk deals with two related themes: the ongoing discussion of “data science vs. statistics” and the importance of developing your own data analysis workflow. These topics are related in my mind because I believe, as academic statisticians, we have an unnecessarily narrow view of our discipline. The “data science vs statistics” debate brings this to a head, because the desire to join and even lead data science initiatives provides an incentive to broaden our mandate. What if we embraced the development and teaching of tools – both mental and digital – that address the entire data analysis process? We’ll conclude with something very concrete: a tour of semi-recent developments in the R ecosystem, aimed at people who want to make their workflow more productive and less aggravating.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_079">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:10 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="canonical-covariance-analysis-for-mixed-numerical-and-categorical-three-way-three-mode-data" class="section level2 unnumbered">
<h2>Canonical Covariance Analysis For Mixed Numerical And Categorical Three-Way Three-Mode Data</h2>
<p style="text-align:center">
Jun Tsuchida and Hiroshi Yadohisa<br /> Doshisha University<br />
</p>
<p><span><strong>Abstract:</strong></span> Three-mode three-way data (objects <span class="math inline">\(\times\)</span> variable <span class="math inline">\(\times\)</span> conditions) have been observed in many areas of research. For example, panel data often include values for the same objects and variables at different times. Given two three-mode three-way data sets, we often investigate two types of factors: common factors, which show the relationships between the two data sets, and unique factors, which represent the uniqueness of each data set. In light of this, canonical covariance analysis has been proposed. However, these datasets often have numerical and categorical variables simultaneously. Many multivariate methods for two three-mode thee-way data sets assume that the data has numerical variables only. To overcome this problem, we propose three-mode three-way canonical covariance analysis with numerical and categorical variables. We use an optimal scaling method (for example, Yong (1987)) for the quantification of categorical data because the values of a categorical variable could not be compared with the value of a numerical variable.</p>
<p><span><strong>Keywords:</strong></span> Alternative least squares, Dimensional reduction, Optimal scaling, Quantification method</p>
<p><span><strong>References:</strong></span></p>
Young, F. W. (1981). Quantitative analysis of qualitative data. <em>Psychometrika</em>, <strong>46</strong>, pp. 357–388
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_038">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:10 OGGB4 (260-073)
</p>
</div>
</div>
<div id="variable-selection-algorithms" class="section level2 unnumbered">
<h2>Variable Selection Algorithms</h2>
<p style="text-align:center">
Fangayo Li<sup>1</sup>, Christopher Triggs<sup>1</sup>, Bogdan Dumitrescu<sup>2</sup>, and Ciprian Giurcaneanu<sup>1</sup><br /> <sup>1</sup>University of Auckland<br /> <sup>2</sup>University Politehnica of Bucharest<br />
</p>
<p><span><strong>Abstract:</strong></span> The matching pursuit algorithm (MPA) is an efficient solution for high dimensional variable selection (Bühlmann and van de Geer, 2011). There is, however, no widely accepted stopping rule for MPA. (Li et al., 2017) have given novel stopping rules based on information theoretic criteria (ITC). All of these ITC are based on the degrees of freedom (df) of the hat matrix which maps the data vector to its estimate. We derive some properties of the hat matrix when MPA is used. These allow us to give an upper bound on the possible increase in df between successive MPA iterations. A simulation study with data generated from different models compares the mean integrated square error of the different ITC and cross validation (Sancetta, 2016).</p>
<p><span><strong>Keywords:</strong></span> Matching pursuit algorithm, degrees of freedom, hat matrix</p>
<p><span><strong>References:</strong></span></p>
<p>A.Sancetta (2016). <em>Greedy algorithms for prediction</em>. Bernoulli, vol. 22, pp. 1227 - 1277.</p>
<p>P.Bühlmann and S.van de Geer (2011). <em>Statistics for high-dimensional data. Methods, theory and applications</em>. Springer Science &amp; Business Media.</p>
F.Li, C.Triggs, B.Dumitrescu, and C.D.Giurcăneanu (2017). <em>On the number of iterations for the matching pursuit algorithm</em> . Proceedings of the 25th European Signal Processing Conference (EUSIPCO), pp. 191 - 195. (to appear)
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_094">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:10 OGGB5 (260-051)
</p>
</div>
</div>
<div id="estimating-causal-structures-for-continuous-and-discrete-variables" class="section level2 unnumbered">
<h2>Estimating Causal Structures For Continuous And Discrete Variables</h2>
<p style="text-align:center">
Mako Yamayoshi and Hiroshi Yadohisa<br /> Doshisha University<br />
</p>
<p><span><strong>Abstract:</strong></span> Structural equation models have been used extensively for continuous variable data to find causal structures. In such a framework, the Linear Non- Gaussian Acyclic Model (LiNGAM) could enable finding a whole causal model (Shimizu et al., 2006). However, in many desciplines, the data include both continuous and discrete variables. LiNGAM could fail to capture the actual causal relationship for such data because it handles both discrete and continuous variables as continuous. Therefore, it is necessary to improve the estimation method for causal structures in such conditions.</p>
<p>In this study, we propose a method to find causal structures for continuous and discrete variables. To overcome the problems of the existing method, we use the Link function. Using simulation studies, we show that the proposed method performs more efficiently for data that includes continuous and discrete variables.</p>
<p><span><strong>Keywords:</strong></span> Causal direction, Latent variables, Link function, SEM, LiNGAM</p>
<p><span><strong>References:</strong></span></p>
<p>Barnett, J.A., Payne, R.W. and Yarrow, D. (1990). <em>Yeasts: Characteristics and identification: Second Edition.</em> Cambridge: Cambridge University Press.</p>
<p>S. Shimizu, P.O. Hoyer, A. Hyvärinen, and A. Kerminen (2006). A linear non-Gaussian acyclic model for causal discovery. <em>The Journal of<br />
Machine Learning Research</em>, vol. 7, pp. 2003-2030.</p>
<p>(ed.) Barnett, V., Payne, R. and Steiner, R. (1995). <em>Agricultural Sustainability: Economic, Environmental and Statistical Considerations</em>. Chichester: Wiley.</p>
<p>Payne, R.W. (1997). <em>Algorithm AS314 Inversion of matrices Statistics</em>, <strong>46</strong>, 295–298.</p>
Payne, R.W. and Welham, S.J. (1990). A comparison of algorithms for combination of information in generally balanced designs. In: <em>COMPSTAT90 Proceedings in Computational Statistics</em>, 297–302. Heidelberg: Physica-Verlag.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_010">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:10 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="incorporating-genetic-networks-into-case-control-association-studies-with-high-dimensional-dna-methylation-data" class="section level2 unnumbered">
<h2>Incorporating Genetic Networks Into Case-Control Association Studies With High-Dimensional DNA Methylation Data</h2>
<p style="text-align:center">
Hokeun Sun<br /> Pusan National University<br />
</p>
<p><span><strong>Abstract:</strong></span> In human genetic association studies with high-dimensional microarray data, it has been well known that statistical methods utilizing prior biological network knowledge such as genetic pathways and signaling pathways can outperform other methods that ignore genetic network structures. In recent epigenetic research on case-control association studies, relatively many statistical methods have been proposed to identify cancer-related CpG sites and the corresponding genes from high-dimensional DNA methylation data. However, most of existing methods are not able to utilize genetic networks although methylation levels among linked genes in the networks tend to be highly correlated with each other. In this article, we propose new approach that combines independent component analysis with network-based regularization to identify outcome-related genes for analysis of high-dimensional DNA methylation data. The proposed approach first captures gene-level signals from multiple CpG sites using independent component analysis and then regularizes them to perform gene selection according to given biological network information. In simulation studies, we demonstrated that the proposed approach overwhelms other statistical methods that do not utilize genetic network information in terms of true positive selection. We also applied it to the 450K DNA methylation array data of the four breast invasive carcinoma cancer subtypes from The Cancer Genome Atlas (TCGA) project.</p>
<span><strong>Keywords:</strong></span> Independent component analysis, network-based regularization, genetic network, DNA methylation, high-dimensional data
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_015">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:10 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="adaptive-model-checking-for-functional-single-index-models" class="section level2 unnumbered">
<h2>Adaptive Model Checking For Functional Single-Index Models</h2>
<p style="text-align:center">
Feifei Chen<sup>1</sup>, Qing Jiang<sup>2</sup>, and Zhenghui Feng<sup>3</sup><br /> <sup>1</sup>Renmin University<br /> <sup>2</sup>Beijing Normal University<br /> <sup>3</sup>Xiamen University<br />
</p>
<p><span><strong>Abstract:</strong></span> In this paper, a model-adaptive test statistic is proposed to do model checking for functional single-index models. Dimension reduction methods are included to handle the curse of dimensionality. The test statistic consists of two parts: the first term is a naive one, and the second term is adaptive to the model as if the model were univariate. It is consistent and can detect local alternative at a fast rate. Monte Carlo method is used to find the critical value under null hypothesis. Simulation studies show the performance of our proposed test procedure.</p>
<span><strong>Keywords:</strong></span> Functional single-index models, dimension reduction, model checking
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_177">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:10 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="mobile-learning-in-teaching-bioinformatics-for-medical-doctors" class="section level2 unnumbered">
<h2>Mobile Learning In Teaching Bioinformatics For Medical Doctors</h2>
<p style="text-align:center">
Taerim Lee<sup>1</sup> and Jung Jin Lee<sup>2</sup><br /> <sup>1</sup>Korea National Open University<br /> <sup>2</sup>Soongsil University<br />
</p>
<span><strong>Abstract:</strong></span> This paper describes an implementation of mobile learning initiative in Bioinformatics Training &amp; Education Center (BITEC) for medical doctors supported from Ministry of Welfare and Public Health Korea. This project was initiated by Dept. of Bioinformatics &amp; Statistics KNOU and Dept. of Medical Informatics of SNU Medical College for training medical doctors. The high penetration rates of mobile phone subscriptions and the rapid growing of handheld users show that it is viable for making inroads towards the usage of mobile devices as an alternative learning mode for distance learning. The mobile learning initiative is aimed to encourage learning and interactions in distance learning communities aiming to bridge the transactional distances faced by the learners and adopt mobility as the key tool in Bioinformatics courses delivery. The BITEC m-Learning initiative focuses on introducing Bioinformatics using handheld devices to be made easily accessible through the use of mobile devices for ODL Bioinformatics learners who are very busy medical doctors in ubiquitous learning environment. The m-Learning approach is considered as a learning alternative to support distance learners mainly working doctors and medical researchers in Korea. This research paper discusses the implementation of the mobile e-Book approach which has better affordable, accessible and flexible educational media.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_034">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:30 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="on-optimal-group-testing-designs-prevalence-estimation-cost-considerations-and-dilution-effects" class="section level2 unnumbered">
<h2>On Optimal Group Testing Designs: Prevalence Estimation, Cost Considerations, And Dilution Effects</h2>
<p style="text-align:center">
Shih-Hao Huang<br /> Academia Sinica<br />
</p>
<p><span><strong>Abstract:</strong></span> Group testing has been used for decades to estimate the prevalence of a rare disease when samples from multiple subjects can be pooled and tested as a group. A group testing design is specified by the support points (distinct group sizes) and their corresponding frequencies. In this series of works, we construct locally optimal approximate designs for group testing with uncertain error rates, where the goal is to maximize the precision of the prevalence estimate. We also provide a guaranteed algorithm based on the approximate theory for constructing exact designs for practical use. Our simulated examples based on a Chlamydia study in the United States show that the proposed design outperforms competing designs, and its performance is quite stable to the working parameters. We then extend the framework to accommodate two features likely to be encountered in real-world studies. We develop optimal budgeted-constrained designs, where both subjects and tests incur costs, and the error rates of the the assay are linked to the group sizes, allowing dilution effects to reduce the test performance. (Work done jointly with M.-N. L. Huang, K. Shedden, and W. K. Wong.)</p>
<p><span><strong>Keywords:</strong></span> Budget-constrained design, dilution effect, <span class="math inline">\(D_s\)</span>-optimality, group testing, sensitivity, specificity</p>
<p><span><strong>References:</strong></span></p>
<p>Huang, S.-H., Huang, M.-N. L., Shedden, K. and Wong, W. K. (in press). Optimal group testing designs for estimating prevalence with uncertain testing errors. <em>Journal of the Royal Statistical Society: Series B.</em> DOI: 10.1111/rssb.12223.</p>
Huang, S.-H., Huang, M.-N. L. and Shedden, K. (manuscript). Cost considerations for efficient group testing studies.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_041">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:30 OGGB4 (260-073)
</p>
</div>
</div>
<div id="the-use-of-bayesian-networks-in-grape-yield-prediction" class="section level2 unnumbered">
<h2>The Use Of Bayesian Networks In Grape Yield Prediction</h2>
<p style="text-align:center">
Rory Ellis, Daniel Gerhard, and Elena Moltchanova<br /> University of Canterbury<br />
</p>
<p><span><strong>Abstract:</strong></span> The requirement for predictions to be made earlier in the growing season has become more important, as the opportunity to plan for the wine production and export earlier in the season becomes desirable. The issue with this is there is less information available to those wishing to make early predictions. The analysis in this paper implements a double sigmoidal curve to model the grape growth over the growing season, as this is typically used in agriculture.</p>
In order to conduct prediction in this study, a Bayesian Network is considered. This allows the opportunity to consider the knowledge of experts in the field, where grape growers would know the growth behaviour of the grapes, as well as using new data to update the Bayesian Network. This information is then implemented in the form of priors, which involves estimating the parameters of the aforementioned double sigmoidal model. Sensitivity Analysis is done in this research, which looks at the impact of prior assumptions (or lack thereof) from experts. Examinations are also made of the value of adding information to the model, as it can be determined whether the precision in the predictions improves as a result of adding data. The results in this analysis are based off simulation studies.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_101">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:30 OGGB5 (260-051)
</p>
</div>
</div>
<div id="pattern-prediction-for-time-series-data-with-change-points" class="section level2 unnumbered">
<h2>Pattern Prediction For Time Series Data With Change Points</h2>
<p style="text-align:center">
Satoshi Goto and Hiroshi Yadohisa<br /> Doshisha University<br />
</p>
<p><span><strong>Abstract:</strong></span> Recently, there have been various types of time series data, such as daily stock prices and Web-click logs, that have complicated the structure. In several cases, because of the complexity, time series data cannot satisfy the stationary process assumption. REGIMECAST (Matsubara and Sakura, 2016) has been proposed as a method to forecast time series data. It is useful for capturing changes in time series patterns and representing the non-linear system. However, it cannot adequately represent time series data after radical changes. Generally, radical changes in time series data can be detected using existing methods, such as change-point detection and anomaly detection. These methods are rarely used for forecasting time series data, although these data often show different behaviors after radical changes.</p>
<p>In this study, we propose a method that can forecast future time series data after events involving radical changes. The method has two features: i) appropriate pattern discovery, as it recognizes the appropriate learning section with change-point detection, and ii) flexible representation, as it represents non-stationary processes with a non-linear state space model. We also provide empirical examples using a variety of real datasets.</p>
<p><span><strong>Keywords:</strong></span> anomaly detection, change-point detection, non-linear state space model, pattern discovery, REGIMECAST</p>
<p><span><strong>References:</strong></span></p>
<p>Y. Matsubara and Y. Sakurai (2016). Regime shifts in streams: Real-time forecasting of co-evolving time sequences, <span><em>Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining</em></span>, 13–17, 2016.</p>
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_018">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:30 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="test-for-genomic-imprinting-effects-on-the-x-chromosome" class="section level2 unnumbered">
<h2>Test For Genomic Imprinting Effects On The X Chromosome</h2>
<p style="text-align:center">
Wing Kam Fung<br /> Unversity of Hong Kong<br />
</p>
<p><span><strong>Abstract:</strong></span> Genomic imprinting is an epigenetic phenomenon that the expression of an allele copy depends on its parental origin. This mechanism has been found to play an important role in many diseases. Methods for detecting imprinting effects have been developed primarily for autosomal markers. However, no method is available in the literature to test for imprinting effects on the X chromosome. Therefore, it is necessary to suggest methods for detecting such imprinting effects. In this talk, the parental-asymmetry test on X the chromosome (XPAT) is first developed to test for imprinting for qualitative traits in the presence of association, based on family trios each with both parents and their affected daughter. Then, we propose 1-XPAT to tackle parent-daughter pairs, each with one parent and his/her affected daughter. By simultaneously considering family trios and parent-daughter pairs, C-XPAT is constructed to test for imprinting. Further, we extend the proposed methods to accommodate complete (with both parents) and incomplete (with one parent) nuclear families having multiple daughters of which at least one is affected. Simulations are conducted to assess the performance of the proposed methods under different settings. Simulation results demonstrate that the proposed methods control the size well, irrespective of the inbreeding coefficient in females being zero or nonzero. By incorporating incomplete nuclear families, C-XPAT is more powerful than XPAT using only complete nuclear families. For practical use, these proposed methods are applied to analyze the rheumatoid arthritis data.</p>
<span><strong>Keywords:</strong></span> Imprinting effects, X chromosome, qualitative traits, nuclear family
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_027">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:30 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="fluctuation-reduction-of-value-at-risk-estimation-and-its-applications" class="section level2 unnumbered">
<h2>Fluctuation Reduction Of Value-At-Risk Estimation And Its Applications</h2>
<p style="text-align:center">
Shih-Feng Huang<br /> National University of Kaohsiung<br />
</p>
<p><span><strong>Abstract:</strong></span> Value-at-Risk (VaR) is a fundamental tool for risk management and is also associated with the capital requirements of banks. Banks need to adjust their capital levels for satisfying the Basel Capital Accord. On the other hand, managements do not like to change the capital levels too often. To achieve a balance, this study proposes an approach to reduce the fluctuation of VaR estimation. The first step is to fit a time series model to the underlying asset returns and obtain the conventional VaR process. A new VaR (NVaR) estimation of the conventional VaR process is then determined by applying change-point detection algorithms and a proposed combination scheme. The capital levels computed from the NVaR process are capable of satisfying the Basel Accord and reducing the fluctuation of capital levels simultaneously. To apply the proposed method to the calculation of future capital requirements, an innovative approach for NVaR prediction is also proposed by incorporating the concept of CUSUM control charts. The return processes of 30 companies on the list of S<span class="math inline">\(\&amp;\)</span>P 500 from 2005 to 2016 are employed for our empirical investigation. Numerical results indicate that the proposed NVaR prediction is capable of satisfying the Basel Accord and reducing the fluctuation of capital requirements simultaneously by using a comparable average amount of capital requirements to the conventional VaR estimator.</p>
<span><strong>Keywords:</strong></span> Capital requirement, change point detection, CUSUM control chart, fluctuation reduction, Value-at-Risk
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_204">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:30 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="e-learning-courses-on-introductory-statistics-using-interactive-educational-tools" class="section level2 unnumbered">
<h2>E-Learning Courses On Introductory Statistics Using Interactive Educational Tools</h2>
<p style="text-align:center">
Kazunori Yamaguchi<sup>1</sup>, Kotaro Ohashi<sup>1</sup>, and Michiko Watanabe<sup>2</sup><br /> <sup>1</sup>Rikkyo University<br /> <sup>2</sup>Keio University<br />
</p>
<span><strong>Abstract:</strong></span> This paper introduces e-learning courses for principles and methods of introductory statistics, which is developing for undergarduate students. The course consists of the following contents; Usages and linkages to the official statistics in Japan, videos titled statistics for daily life, and interactive learning contents. For this course, we have developed Japanese versions of interactive Java applets for understanding statistical concepts and a tool for the simulation and data analysis. We expect that combination of these tools and e-learning contents not only makes students easy to understand basic concepts of statistics but also motivate students to study statistics.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_061">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:50 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="estimation-of-animal-density-from-acoustic-detections" class="section level2 unnumbered">
<h2>Estimation Of Animal Density From Acoustic Detections</h2>
<p style="text-align:center">
Ben Stevenson<sup>1</sup> and David Borchers<sup>2</sup><br /> <sup>1</sup>University of Auckland<br /> <sup>2</sup>University of St Andrews<br />
</p>
<p><span><strong>Abstract:</strong></span> Estimating the density of animal populations is of central importance in ecology, with practical applications that affect decision making in the fields of wildlife management, conservation, and beyond. For species that vocalise, surveys using acoustic detectors such as microphones, hydrophones, or human observers can be vastly cheaper than traditional surveys that physically capture or visually detect animals. In this talk I describe a spatial capture-recapture approach to estimate animal density from acoustic surveys and present a software implementation in the R package <code>ascr</code>, with examples applied to populations of frogs, gibbons, and whales.</p>
<span><strong>Keywords:</strong></span> Ecological statistics, mark-recapture, point process
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_045">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:50 OGGB4 (260-073)
</p>
</div>
</div>
<div id="mixed-models-for-complex-survey-data" class="section level2 unnumbered">
<h2>Mixed Models For Complex Survey Data</h2>
<p style="text-align:center">
Xudong Huang and Thomas Lumley<br /> University of Auckland<br />
</p>
<p><span><strong>Abstract:</strong></span> I want to fit a mixed model to a population distribution, but I have data from a complex (multistage) sample. The sampling is informative, that is, the model holding for the population is different from the model holding for the (biased) sample. Ignoring the sampling design and just fitting the mixed model to the sample distribution will lead to biased inference. Although both the model and sampling involve “clusters”, the model clusters and sample clusters need not be the same. I will use a pairwise composite likelihood method to estimate the parameters of the population model under this setting. In particular, consistency and asymptotic normality can be established. Variance estimation in this problem is challenging. I will talk about a variance estimator and how to show it is consistent.</p>
<p><span><strong>Keywords:</strong></span> Mixed model, Complex sampling, Pairwise composite likelihood</p>
<p><span><strong>References:</strong></span></p>
Yi, G. , Rao, J. and Li, H.(2016). <em>A weighted composite likelihood approach for analysis of survey data under two-level models.</em> Statistica Sinica, 2016, 26, 569-587
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_102">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:50 OGGB5 (260-051)
</p>
</div>
</div>
<div id="regression-with-random-effects-for-analysing-correlated-survival-data-application-to-disease-recurrences" class="section level2 unnumbered">
<h2>Regression With Random Effects For Analysing Correlated Survival Data: Application To Disease Recurrences</h2>
<p style="text-align:center">
Richard Tawiah, Suzanne Chambers, and Shu-Kay Ng<br /> Griffith University<br />
</p>
<p><span><strong>Abstract:</strong></span> Correlated failure time data arise in many biomedical studies, due to multiple occurrences of the same disease in an individual patient. To account for this correlation phenomenon, we formulate a random effect (frailty) survival model with an autoregressive (AR) covariance structure and adopt the generalized linear mixed model (GLMM) methodology for estimation of regression and variance component parameters. A more general case of the problem is also considered via a multilevel random effect approach where the correlation of survival times is induced by a hierarchical clustering structure, such as the appearances of repeated failures in patients from the same hospital in a multicentre clinical trial setting. Our modelling problem is used to investigate prognostic and treatment effects on disease relapses in two data sets, (1) tumour recurrences in bladder cancer patients and (2) recurrent infections in children with chronic granulomatous disease (CGD). Using the first data set, the effect of treatment thiotepa was found to be insignificant but demonstrated an effect in reducing tumour recurrences with adjusted hazard ratio (AHR) of 0.58 (95% CI: 0.29-1.16, p=0.124). The initial number of tumours (AHR: 1.26, 95% CI: 1.08-1.47, p=0.004) had significant positive effect but the effect of the size of the largest initial tumour was insignificant. In the case of the CGD data, treatment gamma interferon showed a significant decreasing effect (AHR: 0.27, 95% CI: 0.13-0.56, p$&lt;$0.001) on the incidence of recurrent infections. In addition, age effect was significant (AHR: 0.90, 95% CI: 0.81-1.0, p=0.042). Pattern of inheritance, height, weight, sex, use of corticosteroids and prophylactic antibiotics did not exhibit significant association with recurrent infections. The appropriateness of our modelling methodology is investigated in a simulation study. The simulation results show that parameters are satisfactorily estimated in the special case where AR random effect is merely used. However, in the multilevel context bias in the variance parameter of random hospital effect increases as the true magnitude of variation in hospital effects increases.</p>
<span><strong>Keywords:</strong></span> Frailty model, random effect, correlated survival times, recurrent event, GLMM, bladder cancer, CGD
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_085">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:50 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="genetic-predictors-underlying-long-term-cognitive-recovery-following-mild-traumatic-brain-injury" class="section level2 unnumbered">
<h2>Genetic Predictors Underlying Long-Term Cognitive Recovery Following Mild Traumatic Brain Injury</h2>
<p style="text-align:center">
Priya Parmar<sup>1</sup>, Rob Kydd<sup>2</sup>, Andrew Shelling<sup>2</sup>, Suzanne Barker-Collo<sup>2</sup>, Alice Theadom<sup>1</sup>, and Valery Feigin<sup>1</sup><br /> <sup>1</sup>Auckland University of Technology<br /> <sup>2</sup>University of Auckland<br />
</p>
<p><span><strong>Abstract:</strong></span> Traumatic Brain Injury (TBI) is a major cause of death and disability. While moderate and severe forms of TBI develop the most significant impairments even mild TBI may be followed by persisting post-concussion symptoms, neurocognitive problems and mental health disorders such as anxiety. Cognitive impairments can impact on all areas of an individual’s work, home and social life and are important to understand and predict overall recovery. These outcomes may be in part be determined by genetic variants that influence the molecular and physiological response of the brain to damage, as well as determining pre-injury reserve and vulnerability to co-morbidities.</p>
<p>A number of studies have examined the relationship between genetic variants and outcomes following TBI. Most have examined groups with moderate to severe injury and are limited by small sample sizes, selection biases, failed to correct for ethnic factors, and have evaluated outcomes at various time points, making comparison between studies difficult.</p>
<p>Using the population-based study of TBI in NZ (BIONIC) we analysed the association between cognitive outcomes with 18 genetic markers (SNPs-single nucleotide polymorphisms) from 12 genes previously studied in relation with TBI; FAAH, GAD1, WWC1, CHMR2, ANKK1, BDNF, NGB, BCL2, APOE, S100B, HMOX1 and COMT in a sample of 183 European and 76 Maori adults. We used the CNS-Vital Signs (computerised neurocognitive test battery) to provide 11 measures of cognitive functioning, memory and attention collected at baseline, 1-, 6-, 12- and 48 months post-injury.</p>
<p>ANCOVA models were used to identify the association between time, SNP (modelled as major, heterozygous and minor alleles) and SNP by time effect for each CNS-Vital Signs outcome. Statistically significant findings were observed in both European and Maori samples for being associated with the same CNS-Vital Signs outcome for rs8191992 (CHMR2), rs4680 (rs4680), rs2071746 (HMOX1) and rs17071145 (WWC1).</p>
<p>A linear mixed effects model was utilised to analyse each individual’s natural cognitive recovery trajectory over time. The individuals’ age, gender, whether or not this was their first TBI, the severity level of the mild TBI (low, medium or high) and SNP were all included in the model as covariates.</p>
<p>Regression analyses identified the following SNPs to be statistically associated with several CNS-Vital Signs outcomes; rs8191992 (CHMR2) was shown to be associated with attention, neurocognition, composite memory, executive functioning as well as processing and psychomotor speed in Europeans.</p>
<p>Whilst rs3798178 (GAD1) was associated with two domains of attention, neurocognition and three domains of memory (composite, visual and working) in Maori. We found rs3791879 was associated with increased attention and neurocognition in our European sample.</p>
<p>Furthermore, the minor alleles of rs11604671 (ANKK1) were associated with poorer cognitive recovery (compared to those with homozygous major alleles) for two domains of attention, executive functioning, processing speed, social acuity and working memory over time in Maori. We found rs11604671 was associated with reduced executive functioning and processing speed in our European sample</p>
Unlike other genetic studies on TBI patients, our study investigated several different genetic variants in a larger ethnically diverse population sample of individuals with primarily mild TBI. Although our findings agreed with previous literature for genetic associations for cognitive recovery post-injury, for the first time, we were able to identify ethnic differences in specific genetic markers determining specific cognitive outcomes in European and Maori people with TBI. Further large TBI population based cohort studies are warranted to replicate these genetic associations, both locally and globally in order to better understand the differences underlying an individual’s outcome trajectory and inform more effective treatment strategies.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_049">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:50 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="bayesian-structure-selection-for-vector-autoregression-model" class="section level2 unnumbered">
<h2>Bayesian Structure Selection For Vector Autoregression Model</h2>
<p style="text-align:center">
Chi-Hsiang Chu<sup>1</sup>, Mong-Na Lo Huang<sup>1</sup>, Shih-Feng Huang<sup>2</sup>, and Ray-Bing Chen<sup>3</sup><br /> <sup>1</sup>National Sun Yat-sen University<br /> <sup>2</sup>National University of Kaohsiung<br /> <sup>3</sup>National Cheng Kung University<br />
</p>
<p><span><strong>Abstract:</strong></span> Vector autoregression (VAR) model is powerful in economic data analysis because it can be used to analyze several different time series data simultaneously. However, in VAR model, we need to deal with the huge coefficient dimensionality and it would be caused some computational problems for coefficient inference. To reduce the dimensionality, we could take some model structures into account based on the prior knowledge. In this paper, several group structures of the coefficient matrices are considered. Due to different types of VAR structures, corresponding MCMC algorithms are proposed to generate posterior samples for making inference of the structure selection. Simulation studies and a real example are used to show the performances of the proposed Bayesian approaches.</p>
<span><strong>Keywords:</strong></span> Bayesian variable selection, time series, universal grouping, segmentized grouping
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_182">
<p class="contribBanner">
Monday 11<sup>th</sup> 14:50 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="three-dimensional-data-visualization-education-with-virtual-reality" class="section level2 unnumbered">
<h2>Three-Dimensional Data Visualization Education With Virtual Reality</h2>
<p style="text-align:center">
Dae-Heung Jang, Jae Eun Lee, and Sojin Ahn<br /> Pukyong National University<br />
</p>
<p><span><strong>Abstract:</strong></span> A variety of data visualization methods are utilizing to analyze huge amount of data. Among various methods, a three-dimensional image requires the rotation of the image to show stereo image on the two-dimensional screen. This study discusses data visualization education of two methods (static method and dynamic method) which make it possible to analyze the construct of stereo image to improve the restriction of the three-dimensional image display with virtual reality. This investigation can be useful to explore three-dimensional data structure more clearly.</p>
<p><span><strong>Keywords:</strong></span> Data visualization education, Virtual reality, Stereo image, R package</p>
<p><span><strong>References:</strong></span></p>
<p>Bowman, A. (2015). <em>rpanel</em>: Simple interactive controls for R using the tcltk library. R package version 1.1-3.</p>
<p>Campos, M. M. (2007). Way Cooler: PCA and Visualization Linear Algebra in the Oracle Database 2, <a href="http://oracledmt.blogspot.kr/2007/06/way-cooler-pca-and-visualization-linear.html" class="uri">http://oracledmt.blogspot.kr/2007/06/way-cooler-pca-and-visualization-linear.html</a>.</p>
<p>Ligges, U. (2017). <em>scatterplot3d</em>: 3D Scatter Plot. R package version 0.3-38.</p>
<p>Murdoch, D. (2017). <em>rgl</em>: 3D Visualization Using OpenGL. R package version 0.97.0.</p>
<p>Myers, R. H., Montgomery, D. C. and Anderson-Cook, C. M. (2016). <em>Response Surface Methodology: Process and Product Optimization Using Designed Experiments, 4th ed</em>, Wiley, New York.</p>
<p>Ripley, B. (2016). <em>MASS</em>: Support Functions and Datasets for Venables and Ripley’s MASS. R package version 7.3-47.</p>
<p>Sarkar, D. (2016). <em>lattice</em>: Trellis Graphics for R. R package version 0.20-35.</p>
<p>Soetaert, K. (2016). <em>plot3D</em>: Plotting Multi-Dimensional Data. R package version 1.1.</p>
<p>Wolf, H. P. (2015). <em>aplpack</em>: Another Plot PACKage: stem.leaf, bagplot, faces, spin3R, plotsummary, plothulls, and some slider functions. R package version 1.3.0.</p>
<p><a href="http://astrostatistics.psu.edu/datasets/SDSS" class="uri">http://astrostatistics.psu.edu/datasets/SDSS</a> quasar.html.</p>
<p><a href="http://forbes.com/mlb" class="uri">http://forbes.com/mlb</a> valuations/list.</p>
<a href="http://gartner.com/newsroom/id/3412017" class="uri">http://gartner.com/newsroom/id/3412017</a>.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_146">
<p class="contribBanner">
Monday 11<sup>th</sup> 15:10 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="talk-data-to-me" class="section level2 unnumbered">
<h2>Talk Data To Me</h2>
<p style="text-align:center">
Lisa Hall<br /> Fonterra<br />
</p>
<span><strong>Abstract:</strong></span> For some scientists, seeing a statistician in the corridor is the equivalent of coming face to face with the grim reaper. They fear being beaten over the head with a Stat101 textbook as you try to hammer into them the importance of replication, randomisation or an appropriate sample size. Many scientists would rather blunder through on their own than admit to the statistician that they don’t understand the ins-and-outs of ANOVA. So we end up with peer reviewed research with badly interpreted p-values and underwhelming Excel graphs despite the existence of more elegant solutions. As ambassadors of statistical rigour, we are the ones who can turn this around. When the scientists won’t come to us, how can we improve data usage short of chasing scientists down corridors with textbooks? I will give examples of how to Talk Data with scientists to encourage their statistical awakening. In the right context, even the most puritanical scientists can become more comfortable Talking Data. They may even find they enjoy it!
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_052">
<p class="contribBanner">
Monday 11<sup>th</sup> 15:10 OGGB4 (260-073)
</p>
</div>
</div>
<div id="smooth-nonparametric-regression-under-shape-restrictions" class="section level2 unnumbered">
<h2>Smooth Nonparametric Regression Under Shape Restrictions</h2>
<p style="text-align:center">
Hongbin Guo and Yong Wang<br /> University of Auckland<br />
</p>
<p><span><strong>Abstract:</strong></span> Shape-restricted regression, in particular under isotonicity and convexity(concavity) constraints, has many practical applications. Traditional nonparametric methods to the problem using least squares or maximum likelihood result in discrete step functions or nonsmooth piecewise linear functions, which are unsatisfactory both predictively and visually. In this talk, we describe a new, smooth, nonparametric estimator under the above-mentioned shape restrictions. In particular, the discrete measures that are inherent in the previous estimators are replaced with continuous ones. A new algorithm that can rapidly find the corresponding estimate will also be presented. Numerical studies show that the new estimator outperforms major existing methods in almost all cases.</p>
<p><span><strong>Keywords:</strong></span> Nonparametric regression, smooth, shape restriction, convex, monotonic</p>
<p><span><strong>References:</strong></span></p>
<p>Groeneboom, P., Jongbloed, G. and Wellner, A. (2001). Estimation of a Convex Function: Characterizations and Asymptotic Theory. <em>Ann. Statist</em>. <strong>29</strong>(6), 1653–1698.</p>
<p>Wang, Y. (2007). On fast computation of the non-parametric maximum likelihood estimate of a mixing distribution. <em>Journal of the Royal Statistical Society. Series B: Statistical Methodology</em>, <strong>69</strong> (2), 185–198.</p>
Meyer, M.(2008). Inference using shape-restricted regression splines. <em>Ann. Appl. Stat.</em> <strong>2</strong>(3), 1013–1033.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_130">
<p class="contribBanner">
Monday 11<sup>th</sup> 15:10 OGGB5 (260-051)
</p>
</div>
</div>
<div id="elastic-band-transform-a-new-approach-to-multiscale-visualization" class="section level2 unnumbered">
<h2>Elastic-Band Transform: A New Approach To Multiscale Visualization</h2>
<p style="text-align:center">
Guebin Choi and Hee-Seok Oh<br /> Seoul National University<br />
</p>
<p><span><strong>Abstract:</strong></span> This paper presents a new transformation technique for multiscale visualization of one-dimensional data such as time series and functional data under the concept of the scale-space approach. The proposed method uses a range of regular observations (eye scanning) with varying intervals. The results, termed ‘elastic-band transform’ can be considered as a collection of observations over different intervals of viewing. It is motivated by a way that human looks at an object such as a sequence of data repeatedly in order to overview a global structure of it as well as find some specific features of it. Some measures based on elastic-bands are discussed for describing characteristics of data, and two-dimensional visualizations induced by the measures are developed for understanding and detecting important structures of data. Furthermore, some statistical applications are studied.</p>
<p><span><strong>Keywords:</strong></span> Transformation; Visualization; Decomposition; Filter; Time Series</p>
<p><span><strong>References:</strong></span></p>
<p>Chaudhuri, P. and Marron, J. S. (1999). SiZer for exploration of structures in curves. <span><em>Journal of the American Statistical Association</em></span>, <span><strong>94</strong></span>, 807–823.</p>
<p>Donoho, D. L., and Johnstone, I. M. (1994). Ideal spatial adaptation by wavelet shrinkage. <span><em>Biometrika</em></span>, <span><strong>81</strong></span>, 425-455.</p>
<p>Dragomiretskiy, K. and Zosso, D. (2014). Variational mode decomposition. <span><em>IEEE Transactions on Signal Processing</em></span>, <span><strong>62</strong></span>, 531–544.</p>
<p>Erästö, P. and Holmström, L. (2005). Bayesian multiscale smoothing for making inferences about features in scatter plots. <span><em>Journal of Computational and Graphical Statistics</em></span>, <span><strong>14</strong></span>, 569–589.</p>
<p>Fryzlewicz, P. and Oh, H.-S. (2011). Thick pen transformation for time series. <span><em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em></span>, <span><strong>73</strong></span>, 499–529.</p>
<p>Hannig, J. and Lee, T. C. M. (2006). Robust SiZer for exploration of regression structures and outlier detection. <span><em>Journal of Computational and Graphical Statistics</em></span>, <span><strong>15</strong></span>, 101–117.</p>
<p>Hannig, J., Lee, T. and Park, C. (2013). Metrics for SiZer map comparison. <span><em>Stat</em></span>, <span><strong>2</strong></span>, 49–60.</p>
<p>Holmström, L. (2010a). BSiZer. <span><em>Wiley Interdisciplinary Reviews: Computational Statistics</em></span>, <span><strong>2</strong></span>, 526–534.</p>
<p>Holmströma, L. (2010b). Scale space methods. <span><em>Wiley Interdisciplinary Reviews: Computational Statistics</em></span>, <span><strong>2</strong></span>,150–159.</p>
<p>Holmströma, L. and Pasanena, L. (2017). Statistical scale space methods. <span><em>International Statistical Review</em></span>, <span><strong>85</strong></span>, 1–30.</p>
<p>Huang, N. E., Shen, Z., Long, S. R., Wu, M. C., Shih, H. H., Zheng, Q., … &amp; Liu, H. H. (1998). The empirical mode decomposition and the Hilbert spectrum for nonlinear and non-stationary time series analysis. <span><em>Proceedings of the Royal Society of London A: Mathematical, Physical and Engineering Sciences</em></span>, <span><strong>454</strong></span>, 903–995.</p>
<p>Lindeberg, T. (1994). <span><em>Scale-Space Theory in Computer Vision</em></span>, Springer Science &amp; Business Media, New York.</p>
<p>Park, C., Hannig, J. and Kang, K. H. (2009). Improved SiZer for time series. <span><em>Statistica Sinica</em></span>, <span><strong>19</strong></span>, 1511–1530.</p>
<p>Park, C, Lee, T. C. and Hannig, J. (2010). Multiscale exploratory analysis of regression quantiles using quantile SiZer. <span><em>Journal of Computational and Graphical Statistics</em></span>, <span><strong>19</strong></span>, 497–513.</p>
<p>Rioul, O. and Vetterli, M. (1991). Wavelets and signal processing. <span><em>IEEE Signal Processing Magazine</em></span>, 8(LCAV-ARTICLE-1991-005), 14–38.</p>
Vogt, M., &amp; Dette, H. (2015). Detecting gradual changes in locally stationary processes. The Annals of Statistics, 43(2), 713-740.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_180">
<p class="contribBanner">
Monday 11<sup>th</sup> 15:10 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="meta-analytic-principal-component-analysis-in-integrative-omics-application" class="section level2 unnumbered">
<h2>Meta-Analytic Principal Component Analysis In Integrative Omics Application</h2>
<p style="text-align:center">
Sunghwan Kim<sup>1</sup> and George Tseng<sup>2</sup><br /> <sup>1</sup>Keimyung University<br /> <sup>2</sup>University of Pittsburgh<br />
</p>
<p><span><strong>Abstract:</strong></span> With the prevalent usage of microarray and massively parallel sequencing, numerous high-throughput omics datasets have become available in the public domain. Integrating abundant information among omics datasets is critical to elucidate biological mechanisms. Due to the high- dimensional nature of the data, methods such as principal component analysis (PCA) have been widely applied, aiming at effective dimension reduction and exploratory visualization. In this paper, we combine multiple omics datasets of identical or similar biological hypothesis and introduce two variations of meta-analytic framework of PCA, namely MetaPCA. Regularization is further incorporated to facilitate sparse feature selection in MetaPCA. We apply MetaPCA and sparse MetaPCA to simulations, three transcriptomic meta-analysis studies in yeast cell cycle, prostate cancer, mouse metabolism, and a TCGA pan-cancer methylation study. The result shows improved accuracy, robustness and exploratory visualization of the proposed framework.</p>
<p><span><strong>Keywords:</strong></span> principal component analysis, meta-analysis, omics data</p>
<p><span><strong>References:</strong></span></p>
<p>Flury (1984) <em>Common principal components in k groups.</em> Journal of the American Statistical Association, 79, 892–898.</p>
Krzanowski (1979) <em>Between-groups comparison of principal components.</em> Journal of the American Statistical Association, 74, 703–707
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_081">
<p class="contribBanner">
Monday 11<sup>th</sup> 15:10 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="flight-to-relative-safety-learning-from-a-no-arbitrage-network-of-yield-curves-model-of-the-euro-area" class="section level2 unnumbered">
<h2>Flight To Relative Safety: Learning From A No-Arbitrage Network Of Yield Curves Model Of The Euro Area</h2>
<p style="text-align:center">
Zhiwu Hong<sup>1</sup> and Linlin Niu<sup>2</sup><br /> <sup>1</sup>HKUST Business School<br /> <sup>2</sup>Xiamen University<br />
</p>
<span><strong>Abstract:</strong></span> This paper develops a no-arbitrage network of yield curves model for the euro area to study the joint dynamics of and risk transmission across yield curves of different economies. The model nests 1+M yield curves of a benchmark country and M periphery countries with 3+(2+M) state factors. The benchmark curve is spanned by three yield factors, which are also common basis for all curves. Periphery yield spreads are spanned by three factors, including two common risk factors related to market liquidity risk and common sovereign credit risk, and a country-specific risk factor. Parsimony can be guaranteed as M increases with additional yield curve data, and factors can be strongly identified with structural restrictions under the no-arbitrage conditions. We estimate a 1+5 yield curves model for Germany and GIIPS countries using weekly data from 2009 to 2016. The results show that, the German yields enjoy a ‘flight to liquidity’ effect under a liquidity shock, which pushes down German yields while driving up periphery spreads. However, in response to a shock of common sovereign credit risk, yields of all countries, including Germany, tend to increase immediately and persistently. The weaker is a country’s economic fundamental, the higher its risk exposure to shocks. Though spillover effects among periphery curves are generally positive, when a Greek shock bursts, the Italian risk factor may be temporarily mitigated due to its relative stronger economy. The network model can be adapted with a time-varying parameter VAR to monitor evolving contributions of factors.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_020">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:00 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="bayesian-analyses-of-non-homogeneous-gaussian-hidden-markov-models" class="section level2 unnumbered">
<h2>Bayesian Analyses Of Non-Homogeneous Gaussian Hidden Markov Models</h2>
<p style="text-align:center">
Shin Sato and Darfiana Nur<br /> Flinders University<br />
</p>
<p><span><strong>Abstract:</strong></span> We investigate a non-homogeneous Gaussian hidden Markov model where the model assumes the transition probabilities between the hidden states depend on each discrete-time. The methodology of the statistical inference for the model follows the Bayesian approach implementing the Markov chain Monte Carlo (MCMC) methods for parameter estimation. The methods include: the Metropolis-Hastings, the delayed rejection Metropolis-Hastings, the multiple-try Metropolis-Hastings, and the adaptive Metropolis algorithms.</p>
<p>For simulation studies, we have successfully implemented all the algorithms proposed on the simulated data set that was investigated by Diebold et al. (1994), although we had been faced with the difficulties of estimating each parameter due to the large noises in the data. For a case study, the model was implemented on a data set of the monthly US 3-month treasury bill rates with six financial exogenous variables in which the settings are identical to that of Meligkotsidou and Dellaportas’s (2011), except for the algorithm.</p>
<p><span><strong>Keywords:</strong></span> Non-homogeneous hidden Markov model, Bayesian inference, Markov chain Monte Carlo methods, Metropolis-Hastings algorithms</p>
<p><span><strong>References:</strong></span></p>
<p>Diebold, F.X., Lee, J.-H., and Weinbach, G.C. (1994). Regime switching with time-varying transition probabilities. <em>Business Cycles: Durations, Dynamics, and Forecasting</em>, 144–165.</p>
Spezia, L. (2006). Bayesian analysis of non-homogeneous hidden markov models. <em>Journal of Statistical Computation and Simulation</em>, <strong>76</strong>(8), 713–725.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_060">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:00 OGGB4 (260-073)
</p>
</div>
</div>
<div id="robustness-of-temperature-reconstruction-for-past-500-years" class="section level2 unnumbered">
<h2>Robustness Of Temperature Reconstruction For Past 500 Years</h2>
<p style="text-align:center">
Yu Yang, Matthew Schofield, and Richard Barker<br /> University of Otago<br />
</p>
<p><span><strong>Abstract:</strong></span> Temperature reconstruction is vital to studies of climate change. Instrumental records are only available back to 19th century, too short to describe changes that occur over hundreds or thousands of years. Fortunately, nature environmental clues (such as tree rings, pollens and ice cores) can be pieced together to reconstruct unrecorded temperatures. We use tree-ring width to study summer temperature in Northern Sweden for past 500 years. Previous work has shown the predictions to be sensitive to model assumptions. We gain a new insight into this problem by attempting to separately estimate aspects of the process that are robustly estimated. One of these are the years in which the climate is colder or warmer than recent observations. We implement this by considering hidden Markov models on the partially observed temperature series. The model is fitted using Hamiltonian Monte Carlo in Stan.</p>
<p><span><strong>Keywords:</strong></span> temperature reconstruction, robust estimator, hidden Markov model, Bayesian analysis</p>
<p><span><strong>References:</strong></span></p>
Schofield, M. R., Barker, R. J., Gelman, A., Cook, E. R., and Briffa, K. R. (2016). A model-based approach to climate reconstruction using tree-ring data. <em>Journal of the American Statistical Association</em>, 111(513), 93-106.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_139">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:00 OGGB5 (260-051)
</p>
</div>
</div>
<div id="nonparametric-causal-inference-by-the-kernel-method" class="section level2 unnumbered">
<h2>Nonparametric Causal Inference By The Kernel Method</h2>
<p style="text-align:center">
Yuchi Matsuoka and Etsuo Hamada<br /> Osaka University<br />
</p>
<p><span><strong>Abstract:</strong></span> Rubin causal model is a statistical model to estimate the effect of a treatment on the outcome based on the framework of potential outcomes. To estimate a causal effect based on Rubin causal model, propensity score plays a central role. In particular, matching and weighting methods like Inverse Probability Weighted Estimator (IPWE) and Doubly-Robust estimator based on the estimated propensity score are widely used. Despite its popularity, it was pointed out that model misspecification of the propensity score can result in substantial bias of the resulting estimators of a causal effect and potential outcomes. It is possible to estimate propensity score in nonparametric ways or machine learning methods to avoid model misspecification. However, it doesn’t work well in most situations due to following reasons: 1) Curse of dimensionality. 2) They only aim at an accuracy of classification and don’t optimize the covariate balancing. To overcome the problems above, we propose a new estimator of propensity score using kernel mean embeddings of conditional distributions. Although our proposal is completely nonparametric, our estimator has a dimensionality-independent rate of convergence. Using kernel measures of conditional independence for model selection, our estimator can also correct the bias that arises from the imbalance of covariates. In numerical simulations, we confirm that our method can reduce the bias in misspecified settings. We also describe several asymptotic properties of our estimator.</p>
<span><strong>Keywords:</strong></span> Rubin causal model, Propensity score, Kernel method, Kernel mean embedding, Hilbert-Schmidt Independence Criterion
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_019">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:00 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="a-unified-regularized-group-pls-algorithm-scalable-to-big-data" class="section level2 unnumbered">
<h2>A Unified Regularized Group PLS Algorithm Scalable To Big Data</h2>
<p style="text-align:center">
Pierre Lafaye de Micheaux<sup>1</sup>, Benoit Liquet<sup>2</sup>, and Matthew Sutton<sup>2</sup><br /> <sup>1</sup>University of New South Wales<br /> <sup>2</sup>Queensland University of Technology<br />
</p>
<p><span><strong>Abstract</strong></span>: Partial Least Squares (PLS) methods have been heavily exploited to analyse the association between two blocs of data. These powerful approaches can be applied to data sets where the number of variables is greater than the number of observations and in presence of high collinearity between variables. Different sparse versions of PLS have been developed to integrate multiple data sets while simultaneously selecting the contributing variables. Sparse modelling is a key factor in obtaining better estimators and identifying associations between multiple data sets. The cornerstone of the sparsity version of PLS methods is the link between the SVD of a matrix (constructed from deflated versions of the original matrices of data) and least squares minimisation in linear regression. We present here an accurate description of the most popular PLS methods, alongside their mathematical proofs. A unified algorithm is proposed to perform all four types of PLS including their regularised versions. Various approaches to decrease the computation time are offered, and we show how the whole procedure can be scalable to big data sets.</p>
<p><span><strong>Keywords:</strong></span> Big data, High dimensional data, Lasso Penalties, Partial Least Squares, Sparsity, SVD</p>
<p><span><strong>References:</strong></span></p>
<p>Lafaye de Micheaux, P., Liquet, B. &amp; Sutton, M. (2017), <em>A Unified Parallel Algorithm for Regularized Group PLS Scalable to Big Data</em>, ArXiv e-prints .</p>
Liquet, B., Lafaye de Micheaux, P., Hejblum, B. &amp; Thiebaut, R. (2016), <em>Group and sparse group partial least square approaches applied in genomics context</em>, Bioinformatics 32, 35-42.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_033">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:00 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="evaluation-of-spatial-cluster-detection-method-based-on-all-geographical-linkage-patterns" class="section level2 unnumbered">
<h2>Evaluation Of Spatial Cluster Detection Method Based On All Geographical Linkage Patterns</h2>
<p style="text-align:center">
Fumio Ishioka<sup>1</sup>, Jun Kawahara<sup>2</sup>, and Koji Kurihara<sup>1</sup><br /> <sup>1</sup>Okayama University<br /> <sup>2</sup>Nara Institute of Science and Technology<br />
</p>
<p><span><strong>Abstract:</strong></span> Currently, it is becoming easier to analyze the various types of spatial data and express them visually on a map. However, it is still difficult to estimate the location of spatial clusters based on statistical evidence. The spatial scan statistic (Kulldorff 1997), which is based on the idea of maximizing the likelihood of cluster, has been widely used for spatial cluster detection method. It is important how effectively and efficiently we find a cluster whose likelihood is high, and to find such a cluster, some scan approaches are proposed. However, most of them are limited in the shape of a detected cluster, or need an unrealistic computational time if the data size is too large. The zero-suppressed binary decision diagram (ZDD) (Minato, 1993), one approach to frequent item set mining, enables us to extract all of the potential cluster areas at a realistic computational cost. In this study, we try a new way of spatial cluster detection method to detect a cluster with truly highest likelihood by applying the ZDD, and by using them, we compare and evaluate the performance of the existing scan methods.</p>
<p><span><strong>Keywords:</strong></span> Spatial cluster, Spatial scan statistic, ZDD</p>
<p><span><strong>References:</strong></span></p>
<p>Kulldorff, M. (1997). A spatial scan statistic. <em>Communications in Statistics: Theory and Methods</em>, <strong>26</strong>, 1481–1496.</p>
Minato, S. (1993). Zero-suppressed BDDs for set manipulation in combinatorial problems. <em>In: Proceedings of the 30th ACM/IEEE Design Automation Conference</em>, 272–277.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_134">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:00 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="scoring-rules-for-prediction-and-classification-challenges" class="section level2 unnumbered">
<h2>Scoring Rules For Prediction And Classification Challenges</h2>
<p style="text-align:center">
Matt Parry<br /> University of Otago<br />
</p>
<p><span><strong>Abstract:</strong></span> Prediction and classification challenges have become an exciting and useful feature of the statistical and machine learning community. For example, Good Judgement Open asks forecasters to predict the probability of particular world events, and Kaggle.com regularly sets classification challenges. Challenge organizers typically publish a ranked list of the leading submissions and, ultimately, announce the winner of the challenge. However, in order for such a competition to be considered worth entering, the challenge organizers must be seen to evaluate the submissions in a fair and open manner. Scoring rules were devised precisely to solve this problem. Crucially, <span><em>proper</em></span> scoring rules elicit honest statements of belief about the outcome. If the challenge organizers use a proper scoring rule to evaluate submissions, a competitor’s expected score under their true belief will be optimized by actually quoting that belief to the organizers. A proper scoring rule therefore rules out any possibility of a competitor gaming the challenge. We discuss a class of proper scoring rules called linear scoring rules that are specifically adapted to probabilistic binary classification. When applied in competition situations, we show that all linear scoring rules essentially balance the needs of organizers and competitors. We also develop scoring rules to score a sequence of predictions that are targeting a single outcome. These scoring rules discount predictions over time and appropriately weight prediction updates.</p>
<p><span><strong>Keywords:</strong></span> Probabilistic forecast, sequence, prequential principle, discounting</p>
<p><span><strong>References:</strong></span></p>
Parry, M. (2016). <em>Linear scoring rules for probabilistic binary classification</em>. Electronic Journal of Statistics, 10 (1), 1596–1607.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_095">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:20 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="meta-analysis-with-symbolic-data-analysis-and-its-application-for-clinical-data" class="section level2 unnumbered">
<h2>Meta-Analysis With Symbolic Data Analysis And Its Application For Clinical Data</h2>
<p style="text-align:center">
Ryo Takagi, Hiroyuki Minami, and Masahiro Mizuta<br /> Hokkaido University<br />
</p>
<p><span><strong>Abstract:</strong></span> We discuss a method of meta-analysis based on symbolic data analysis (SDA). Meta-analysis, mainly used in social and medical science, is a statistical method of combining scientific studies to obtain quantitative results and provides a high level of evidence. Differences between the studies are caused by heterogeneity between the studies. It is useful to detect relationship among scientific studies. A target of analysis on SDA is <span><em>concept</em></span>, a set of individuals. We apply SDA to meta-analysis. In other words, we regard scientific studies as concepts. For example, symbolic clustering or symbolic MDS are useful to preprocess the scientific studies in meta-analysis. In this study, we propose a new approach based on SDA for meta-analysis and show the results of the proposed approach using clinical datasets.</p>
<p><span><strong>Keywords:</strong></span> symbolic clustering, symbolic MDS, concept in SDA</p>
<p><span><strong>References:</strong></span></p>
<p>Edwin Diday and Monique Noirhomme-Fraiture. (2008). <em>Symbolic data analysis and the SODAS software.</em> John Wiley &amp; Sons, Ltd.</p>
David Edward Matthews and Vernon Todd Farewell. (2015). <em>Using and understanding medical statistics</em> (5th, revised and extended edition). Karger Publishers.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_077">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:20 OGGB4 (260-073)
</p>
</div>
</div>
<div id="real-time-transit-network-modelling-for-improved-arrival-time-predictions" class="section level2 unnumbered">
<h2>Real-Time Transit Network Modelling For Improved Arrival Time Predictions</h2>
<p style="text-align:center">
Tom Elliott and Thomas Lumley<br /> University of Auckland<br />
</p>
<p><span><strong>Abstract:</strong></span> The growing availability of GPS tracking devices means that public transport passengers can now check on the real-time location of their bus from their mobile phone, helping them to decide when to leave home, and once at the stop, how long until the bus arrives. A side effect of this technology is that statistical models using vehicle location data to predict arrival times have taken a “back seat” in preference for methods that are simpler and faster, but less robust. Auckland Transport, who operate our local public transport network, demonstrate this: the estimated arrival time (ETA) of a bus at a stop is simply the time until scheduled arrival, plus the delay at the bus’ most recently visited stop. The most evident problem with this approach is that intermediate stops, traffic lights, and road congestion—all of which affect ETAs—are not considered. We have been developing a modelling framework consisting of (1) a vehicle state model to infer parameters, such as speed, from a sequence of GPS positions; (2) a transit network model that uses information from the vehicle model to estimate traffic conditions along roads in the network; and (3) a predictive model combining vehicle and transit network states to predict arrival times. Since multimodality is common—for example a bus may or may not stop at a bus stop or traffic lights—we are using a particle filter to estimate vehicle state, which makes no assumptions about the shape of the distribution, and allows for a more intuitive likelihood function. While this provides a very flexible framework, it is also a computationally intensive one, so computational demands need to be considered to ensure it will be viable as a real-time application for providing passengers with improved, and hopefully reliable, arrival time information.</p>
<span><strong>Keywords:</strong></span> transit, real-time, particle filter
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_149">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:20 OGGB5 (260-051)
</p>
</div>
</div>
<div id="visualization-and-statistical-modeling-of-financial-big-data" class="section level2 unnumbered">
<h2>Visualization And Statistical Modeling Of Financial Big Data</h2>
<p style="text-align:center">
Masayuki Jimichi<sup>1</sup>, Daisuke Miyamoto<sup>2</sup>, Chika Saka<sup>1</sup>, and Syuichi Nagata<sup>1</sup><br /> <sup>1</sup>Kwansei Gakuin University<br /> <sup>2</sup>Nara Institute of Science and Technology<br />
</p>
<p><span><strong>Abstract:</strong></span> In this work, we manipulate financial big data of world-wide companies by R packages SparkR and sparklyr, and treate data visualization (e.g. Ihaka, 2017; Unwin, 2015) and statistical model (e.g. Chambers and Hastie, 1991) based on exploratory data analysis (Tukey, 1977) with R. The dataset is extracted from the database systems by Bureau van Dijk KK which contains information on over 80,000 listed companies. We find that a log-skew-t linear model (e.g. Azallini and Capitanio, 2014) is very useful for explaining sales by employees and assets.</p>
<p><span><strong>Keywords:</strong></span> Financial Big Data, Data Visualization, Statistical Modeling, Log-skew-t Linear Model, SparkR, sparklyr</p>
<p><span><strong>References:</strong></span></p>
<p>Azzalini, A. with the collaboration of Capitanio, A. (2014). <em>The Skew-Normal and Related Families</em>. Cambridge University Press. Institute of Mathematical Statistics Monographs.</p>
<p>Chambers, J. M. and Hastie, T. J. ed. (1991). <em>Statistical Models in S</em>. Chapman and Hall/CRC.</p>
<p>Ihaka, R. (2017). Lecture Notes. <a href="https://www.stat.auckland.ac.nz/~ihaka/?Teaching" class="uri">https://www.stat.auckland.ac.nz/~ihaka/?Teaching</a></p>
<p>Tukey, J. W. (1977). <em>Exploratory Data Analysis</em>. Addison-Wesley Publishing Co.</p>
Unwin, A. (2015). <em>Graphical Data Analysis with R</em>, Chapman and Hall/CRC.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_140">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:20 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="sparse-group-subgroup-partial-least-squares-with-application-to-genomic-data" class="section level2 unnumbered">
<h2>Sparse Group-Subgroup Partial Least Squares With Application To Genomic Data</h2>
<p style="text-align:center">
Matthew Sutton<sup>1</sup>, Benoit Liquet<sup>1</sup>, and Rodolphe Thiebaut<sup>2</sup><br /> <sup>1</sup>Queensland University of Technology<br /> <sup>2</sup>Inria, SISTM, Talence and Inserm, U1219, Bordeaux, Bordeaux University, Bordeaux and Vaccine Research Institute<br />
</p>
<p><span><strong>Abstract:</strong></span> Integrative analysis of high dimensional omics datasets has been studied by many authors in recent years. By incorporating prior known relationships among the variables, these analyses have been successful in elucidating the relationships between different sets of omics data. In this article, our goal is to identify important relationships between genomic expression and cytokine data from an HIV vaccination trial. We proposed a flexible Partial Least Squares technique which incorporates group and subgroup structure in the modelling process. Our new methodology expands on previous work, by accounting for both grouping of genetic markers (e.g. genesets) and temporal effects. The method generalises existing sparse modelling techniques in the PLS methodology and establishes theoretical connections to variable selection methods for supervised and unsupervised problems. Simulation studies are performed to investigate the performance of our methods over alternative sparse approaches. Our method has been implemented in a comprehensive R package called sgsPLS.</p>
<p><span><strong>Keywords:</strong></span> genomics, group variable selection, latent variable modelling, partial least squares, singular value decomposition</p>
<p><span><strong>References:</strong></span></p>
<p>Chaussabel, D. et al. (2008). A modular analysis framework for blood genomics studies: application to systemic lupus erythematosus. <em>Immunity</em> <span><strong>29,</strong></span> 150–164.</p>
<p>Chun, H. and Keleş, S. (2010). Sparse partial least squares regression for simultaneous dimension reduction and variable selection. <em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em> <span><strong>72,</strong></span> 3–25.</p>
<p>Garcia, T. P., Muller, S., Carroll, R., and Walzem, R. (2014). Identification of important regressor groups, subgroups and individuals via regularization methods: application to gut microbiome data. <em>Bioinformatics</em> <span><strong>30,</strong></span> 35–42.</p>
<p>Gomez-Cabrero, D., Abugessaisa, I., Maier, D., Teschendorff, A., Merkenschlager, M., Gisel, A., Ballestar, E., Bongcam-Rudloff, E., Conesa, A., and Tegn<span>é</span>r, J. (2014). Data integration in the era of omics: current and future challenges. <em>BMC Systems Biology</em> <span><strong>8,</strong></span> 1–10.</p>
<p>Hejblum, B., Skinner, J., and Thièbaut, R. (2015). Time-course gene set analysis for longitudinal gene expression data. <em>PLOS Computational Biology</em> <span><strong>11,</strong></span> 1–21.</p>
<p><span>Le Cao</span>, K., Rossouw, D., Robert-Granie, C., and Besse, P. (2008). <span>A</span> sparse <span>P</span><span>L</span><span>S</span> for variable selection when integrating omics data. <em>Stat Appl Genet Mol Biol</em> <span><strong>7,</strong></span> 37.</p>
<p>Lèvy, Y., Thièbaut, R., Montes, M., Lacabaratz, C., Sloan, L., King, B., Pèrusat, S., Harrod, C., Cobb, A., Roberts, L., Surenaud, M., Boucherie, C., Zurawski, S., Delaugerre, C., Richert, L., Chêne, G., Banchereau, J., and Palucka, K. (2014). Dendritic cell-based therapeutic vaccine elicits polyfunctional hiv-specific t-cell immunity associated with control of viral load. <em>European Journal of Immunology</em> <span><strong>44,</strong></span> 2802–2810.</p>
<p>Lin, D., Zhang, J., Li, J., Calhoun, V., Deng, H., and Wang, Y. (2013). Group sparse canonical correlation analysis for genomic data integration. <em>BMC Bioinformatics</em> <span><strong>14,</strong></span> 1–16.</p>
<p>Liquet, B., de Micheaux, P. L., Hejblum, B., and Thiébaut, R. (2016). Group and sparse group partial least square approaches applied in genomics context. <em>Bioinformatics</em> <span><strong>32,</strong></span> 35–42.</p>
<p>Nowak, G., Hastie, T., Pollack, J., and Tibshirani, R. (2011). A fused lasso latent feature model for analyzing multisample acgh data. <em>Biostatistics</em> <span><strong>12,</strong></span> 776–791.</p>
<p>Parkhomenko, E., Tritchler, D., and Beyene, J. (2009). Sparse canonical correlation analysis with application to genomic data integration. <em>Statistical applications in genetics and molecular biology</em> <span> <strong>8,</strong></span> Article 1.</p>
<p>Rosipal, R. and Krämer, N. (2006). Overview and recent advances in partial least squares. <em>Subspace, Latent Structure and Feature Selection</em></p>
<p>Safo, S. E., Li, S., and Long, Q. (2017). Integrative analysis of transcriptomic and metabolomic data via sparse canonical correlation analysis with incorporation of biological information. <em>Biometrics</em> .</p>
<p>Simon, N., Friedman, J., Tibshirani, R., and Hastie, T. (2013). A <span>S</span>parse-<span>G</span>roup <span>L</span>asso. <em>Journal of Computational and Graphical Statistics</em> <span><strong>22,</strong></span> 213–245.</p>
<p>Tenenhaus, A., Philippe, C., Guillemot, V., <span>Le Cao</span>, K., Grill, J., and Frouin, V. (2014). Variable selection for generalized canonical correlation analysis. <em>Biostatistics</em> <span><strong>15,</strong></span> 569–583.</p>
<p>Tibshirani, R. (1994). Regression shrinkage and selection via the lasso. <em>Journal of the Royal Statistical Society, Series B</em> <span><strong>58,</strong></span> 267–288.</p>
Witten, D., Tibshirani, R., and Hastie, T. (2009). A penalized matrix decomposition, with applications to sparse principal components and canonical correlation analysis. <em>Biostatistics</em> <span><strong>10,</strong></span> 515–534.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_040">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:20 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="genetic-approach-and-statistical-approach-for-association-study-on-dna-data" class="section level2 unnumbered">
<h2>Genetic Approach And Statistical Approach For Association Study On DNA Data</h2>
<p style="text-align:center">
Makoto Tomita<br /> Tokyo Medical and Dental University<br />
</p>
<p><span><strong>Abstract:</strong></span> Genomic information such as genome-wide association analysis (GWAS) in DNA data is very large, however if the sample size corresponding to it is not enough, as an idea to solve, the author considers by a statistical approach and a genetic approach. The former will be briefly introduced, and the latter will be mainly explained. Basically, the method of focusing genome information becomes the center of presentation.</p>
<p><span><strong>Keywords:</strong></span> genome wide association study, linkage disequilibrium, statistical power</p>
<p><span><strong>References:</strong></span></p>
<p>Tomita, M. (2013). Focusing Approach Using LD Block and Association Study with Haplotype Combination on DNA Data, In: <em>Proceedings 2013 Eleventh International Conference on ICT and Knowledge Engineering</em>, 5–10. Bangkok: IEEE Conference #32165.</p>
<p>Tomita, M. (2015). Haplotype estimation, haplotype block identification and statistical analysis for DNA data, In: <em>Conference Program and Book of Abstracts, Conference of the International Federation of Classication Societies (IFCS-2015)</em>, 227–228, Bologna.</p>
<p>Tomita, M., Hatsumichi, M. and Kurihara, K. (2008). <em>Computational Statistics and Data Analysis</em>, <strong>52</strong>(4), 1806–1820.</p>
<p>Tomita, M., Hashimoto, N. and Tanaka, Y. (2011). <em>Computational Statistics and Data Analysis</em>, <strong>55</strong>(6), 2104–2113.</p>
Tomita, M., Kubota, T. and Ishioka, F. (2015). <em>PLoS ONE</em>, <strong>10</strong>(7), e0127358.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_150">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:20 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="modeling-of-document-abstraction-using-association-rule-based-characterization" class="section level2 unnumbered">
<h2>Modeling Of Document Abstraction Using Association Rule Based Characterization</h2>
<p style="text-align:center">
Ken Nittono<br /> Hosei University<br />
</p>
<p><span><strong>Abstract:</strong></span> The importance of systems enabling us to extract useful information from enormous text data produced every day along with our social activities in organizations or on the internet and utilize the information immediately and efficiently have been increasing. In this research, an analyzing method which extracts essential parts from a huge document set utilizing association rule analysis as a data mining method is introduced. The method detects typical combinations of terms involved in contexts and regards them as the characterization of text data and it is also combined with information retrieval methods for the sake of further selection as some parts of the essential contexts. This method is considered to enhance its ability of detection for particular contexts that contain some topics and include moderately distributed terminologies. And implementation of the system is discussed in order for utilizing the abstracted documents efficiently as some sort of knowledge such as collective intelligence. An approach for linkage with R is also mentioned in the phase of the implementation of the model.</p>
<p><span><strong>Keywords:</strong></span> Association rule, Text mining, Big data, Information retrieval</p>
<p><span><strong>References:</strong></span></p>
<p>Agrawal, R. Imielinski, T. and Swami, A. (1993). <em>Mining association rules between sets of items in large databases</em>, Proceedings of the ACM SIGMOD Washington, D.C, 207–216.</p>
Nittono, K. (2013). <em>Association rule generation and mining approach to concept space for collective documents</em>, Proceedings of the 59th World Statistics Congress of the International Statistical Institute, pp. 5515–5520.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_088">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:40 098 Lecture Theatre (260-098)
</p>
</div>
</div>
<div id="bayesian-static-parameter-inference-for-partially-observed-stochastic-systems" class="section level2 unnumbered">
<h2>Bayesian Static Parameter Inference For Partially Observed Stochastic Systems</h2>
<p style="text-align:center">
Yaxian Xu and Ajay Jasra<br /> National University of Singapore<br />
</p>
<p><span><strong>Abstract:</strong></span> We consider Bayesian static parameter estimation for partially observed stochastic systems with discrete-time observations. This is a very important problem, but is very computationally challenging as the associated posterior distributions are highly complex and one has to resort to discretizing the associated probability law of the underlying stochastic system and advanced Markov chain Monte Carlo (MCMC) techniques to infer the parameters. We are interested in the situation where the discretization is in multiple dimensions. For instance, for partially observed stochastic partial differential equations (SPDEs), where dicretization is in both space and time. In such cases, multi-index Monte Carlo (MIMC) is known to have the potential to reduce the computational cost for a prescribed level of error, relative to i.i.d. sampling from the most precise discretization. We demonstrate how MCMC and particularly particle MCMC can be used in the multi-index framework for Bayesian static parameter inference for the above-mentioned models. The main idea involves constructing an approximate coupling of the posterior density of the joint on the parameter and hidden space and then correcting by an importance sampling method. Our method is illustrated numerically to be preferable for inference of parameters for a partially observed SPDE.</p>
<p><span><strong>Keywords:</strong></span> Multi-index Monte Carlo, Markov chain Monte Carlo, stochastic partial differential equations</p>
<p><span><strong>References:</strong></span></p>
<p>Christophe Andrieu, Arnaud Doucet, and Roman Holenstein. (2010). Particle markov chain monte carlo methods. <em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em>, 72(3):269–342.</p>
Haji-Ali, A. L., Nobile, F. &amp; Tempone, R. (2016). Multi-Index Monte Carlo: When sparsity meets sampling. <em>Numerische Mathematik</em>, 132, 767–806.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_080">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:40 OGGB4 (260-073)
</p>
</div>
</div>
<div id="bayesian-survival-analysis-of-batsmen-in-test-cricket" class="section level2 unnumbered">
<h2>Bayesian Survival Analysis Of Batsmen In Test Cricket</h2>
<p style="text-align:center">
Oliver Stevenson and Brendon Brewer<br /> University of Auckland<br />
</p>
<p><span><strong>Abstract:</strong></span> It is widely accepted that in the sport of cricket, batting is more difficult early in a player’s innings, but becomes easier as a player familiarizes themselves with the local conditions. Here we develop a Bayesian survival analysis method to predict and quantify the Test Match batting abilities for international cricketers, at any stage of a player’s innings. The model is applied in two stages, firstly to individual players, allowing us to quantify players’ initial and equilibrium batting abilities, and the rate of transition between the two. The results indicate that most players begin a Test match innings batting with between a quarter and a half of their potential batting ability. The model is then implemented using a hierarchical structure, providing us with more general inference concerning a selected group of opening batsmen from New Zealand. Using this hierarchical structure we are able to make predictions for the batting abilities of the next opening batsman to debut for New Zealand. These results are considered in conjunction with other performance based metrics, allowing us to identify players who excel in the role of opening the batting, which has practical implications in terms of batting order and team selection policy.</p>
<p><span><strong>Keywords:</strong></span> Bayesian survival analysis, hierarchical modelling, cricket</p>
<p><span><strong>References:</strong></span></p>
Stevenson, O.G. and Brewer, B.J. (2017). Bayesian survival anaylsis of opening batsmen in Test cricket <em>Journal of Quantitative Analysis in Sports</em>, <em>13</em>(1), 25-36.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_142">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:40 OGGB5 (260-051)
</p>
</div>
</div>
<div id="covariate-discretisation-on-big-data" class="section level2 unnumbered">
<h2>Covariate Discretisation On Big Data</h2>
<p style="text-align:center">
Hon Hwang<sup>1</sup>, Stephen Wright<sup>2</sup>, and Louise Ryan<sup>1</sup><br /> <sup>1</sup>University of Technology Sydney<br /> <sup>2</sup>Australian Red Cross Blood Service<br />
</p>
<p><span><strong>Abstract:</strong></span> Distributed Computing Systems such as Hadoop and Spark allow statistical analysis to be performed on arbitrary large datasets. However, when performing statistical analysis on these systems, the data communication between the nodes of a distributed computing system can become a major performance bottleneck. In this work, we outline a novel combination of statistical and computation techniques to address this issue. We first apply data reduction technique such as coarsening (interval-censoring) on large data sets using a distributed computing system. We then perform statistical analysis on the coarsened data. However, performing analysis using coarsened data potentially introduces biases in the results. To address this, we use the Expectation-Maximisation (EM) algorithm to recover the complete (non-coarsened) data model. Our work draws on methods for the analysis of data involving coarsened co-variates using EM by methods of weights. We explore different coarsening strategies (e.g., rounding, quantile and quintile) and discuss how our methods can scale to very large data settings. Through simulation studies, we find our method works especially well when data is coarsened from a wide interval, where there are more loss of information. Compared with naïvely using the coarsened data, our method is able to estimate regression coefficients that are closer to estimates obtained from using the complete data. In addition, the standard errors from our method reflect more accurately the uncertainty arising from using coarsened data.</p>
<span><strong>Keywords:</strong></span> EM algorithm, coarsened data, regression, big data
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_092">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:40 Case Room 2 (260-057)
</p>
</div>
</div>
<div id="big-sir-a-sliced-inverse-regression-approach-for-massive-data" class="section level2 unnumbered">
<h2>BIG-SIR A Sliced Inverse Regression Approach For Massive Data</h2>
<p style="text-align:center">
Benoit Liquet<sup>1</sup> and Jerome Saracco<sup>2</sup><br /> <sup>1</sup>Queensland University of Technology<br /> <sup>2</sup>University of Bordeaux<br />
</p>
<p><span><strong>Abstract:</strong></span> In a massive data setting, we focus on a semiparametric regression model involving a real dependent variable <span class="math inline">\(Y\)</span> and a <span class="math inline">\(p\)</span>-dimensional covariable <span class="math inline">\(X\)</span>. This model includes a dimension reduction of X via an index <span class="math inline">\(X&#39;\beta\)</span>. The Effective Dimension Reduction (EDR) direction <span class="math inline">\(\beta\)</span> cannot be directly estimated by the Sliced Inverse Regression (SIR) method due to the large volume of the data. To deal with the main challenges of analysing massive datasets which are the storage and computational efficiency, we propose a new SIR estimator of the EDR direction by following the “divide and conquer” strategy. The data is divided into subsets. EDR directions are estimated in each subset which is a small dataset. The recombination step is based on the optimisation of a criterion which assesses the proximity between the EDR directions of each subset. Computations are run in parallel with no communication among them. The consistency of our estimator is established and its asymptotic distribution is given. Extensions to multiple indices models, <span class="math inline">\(q\)</span>-dimensional response variable and/or SIR<span class="math inline">\(_{\alpha}\)</span>-based methods are also discussed. A simulation study using our <code>edrGraphicalTools</code> R package shows that our approach enables us to reduce the computation time and conquer the memory constraint problem posed by massive datasets. A combination of <code>foreach</code> and <code>bigmemory</code> R packages are exploited to offer efficiency of execution in both speed and memory. Results are visualised using the bin-summarise-smooth approach through the <code>bigvis</code> R package. Finally, we illustrate our proposed approach on a massive airline data set.</p>
<p><span><strong>Keywords:</strong></span> High performance computing, Effective Dimension Reduction (EDR), Parallel programming, R software, Sliced Inverse Regression (SIR)</p>
<p><span><strong>References:</strong></span></p>
Liquet, B., &amp; Saracco, J. (2016), <em>BIG-SIR a Sliced Inverse Regression Approach for Massive Data</em>, Statistics and Its Interface. Vol 9, 509-520.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_087">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:40 Case Room 3 (260-055)
</p>
</div>
</div>
<div id="symbolic-data-analytical-approach-to-unauthorized-access-logs" class="section level2 unnumbered">
<h2>Symbolic Data Analytical Approach To Unauthorized-Access Logs</h2>
<p style="text-align:center">
Hiroyuki Minami and Masahiro Mizuta<br /> Hokkaido University<br />
</p>
<p><span><strong>Abstract:</strong></span> We have been annoyed by tons of unwilling accesses in many Internet applications including SSH (Secure SHell) known as a typical remote access tool and E-mail delivery protocols. An attacked server put a report according to the configuration and the log files have grown day by day.</p>
<p>Bad accesses might be caused by computer virus and so-called <span><em>zombie</em></span>, a hi-jacked computer. We assume that the actions would have their own trends. For example, we sometimes find that a few attacks come simultaneously from only 1 site, however, we also find several attacks from a set of the sites within 1 minute or 1 day. The IP-Addresses might be variable, however, within the assigned range. It suggests that the victimizer is just one but gets an IP-Address allocation so many times.</p>
<p>To analyze the log files and give an interpretation to them, we introduce Symbolic Data Analysis (SDA) to adopt its main idea <span><em>concept</em></span>. If we configure an appropriate <span><em>concept</em></span> whose elements (<span> <em>individuals</em></span> in SDA) are IP-Address, port-numbers and attack time span, we can reveal some relationship between <span><em>concepts</em></span> and classify them into perspective. The results would give us some useful information to protect our Internet environment.</p>
<p>We discuss how we get them and the interpretation appropriately through some practical examples.</p>
<p><span><strong>Keywords:</strong></span> Invalid network access, Firewall, Massive Data Analysis</p>
<p><span><strong>References:</strong></span></p>
<p>Collins, M. (2014). <em>Network Security Through Data Analysis</em>. O’Reilly.</p>
Minami, H. and Mizuta, M. (2016). A study on the Analysis of the refused logs by Internet Firewall. <em>Proceedings of 2016 International Conference for JSCS 30th Anniversary in Seattle</em>.
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>
<div id="talk_192">
<p class="contribBanner">
Monday 11<sup>th</sup> 16:40 Case Room 4 (260-009)
</p>
</div>
</div>
<div id="my-knee-still-hurts-the-statistical-pathway-to-the-development-of-a-clinical-decision-aid" class="section level2 unnumbered">
<h2>My Knee Still Hurts; The Statistical Pathway To The Development Of A Clinical Decision Aid</h2>
<p style="text-align:center">
Robert Borotkanics<br /> Auckland University of Technology<br />
</p>
<p><span><strong>Abstract:</strong></span> Total knee arthroplasty (TKA) is considered an effective intervention to improve physical function and reduce joint pain in those with end stage knee arthritis, yet up to 34 What is reported herein is the methodological approaches applied to tease out the various nuances of developing such a clinical decision aid. By way of summary, a series of logistic regression models were developed and refined to identify predictors chronic postoperative pain, where pain was reported using the numerical pain rating scale (NRS). Self-reported NRS pain was dichotomized based on functional status using a defined statistical approach. Multivariate models were developed using a stepwise selection approach, accounting for interaction and collinearity. The effect of changing collinearity thresholds on information criterion is illustrated. The sensitivity and specificity were calculated, along with receiver operating characteristic (ROC) analyses for each logistic model. Final models were chosen by a combination of superior area under the curve (AUC) and Akaike Information Criterion (AIC). The stability of β coefficients across the top performing models is reported, along with model goodness-of-fit using the Hosmer and Lameshow methodology. Cut-point analyses are reported on models performances, including the effect of changing pain thresholds on model accuracy. Finally, the conversion of a superior logistic model into a probabilistic function, of potential utility for clinicians is illustrated.</p>
<span><strong>Keywords:</strong></span> total knee arthroplasty; logistic regression; clinical decision support
<p style="text-align: right">
<a href = "programme-at-a-glance.html#Monday-tbl">Return to Programme</a><br/><br/>
</p>
<p class="pagebreak">
</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="indexbysubmission.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="Tuesday.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["NZSA-IASC-Programme.epub"],
"toc": {
"collapse": "subsection",
"collapsed": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
