<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017</title>
  <meta name="description" content="Programme and abstracts for NZSA-IASC-ARS Conference 2017">
  <meta name="generator" content="bookdown 0.5 and GitBook 2.6.7">

  <meta property="og:title" content="New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="pictures/logo.png" />
  <meta property="og:description" content="Programme and abstracts for NZSA-IASC-ARS Conference 2017" />
  <meta name="github-repo" content="jmcurran/NZSA-IASC-Prog" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017" />
  
  <meta name="twitter:description" content="Programme and abstracts for NZSA-IASC-ARS Conference 2017" />
  <meta name="twitter:image" content="pictures/logo.png" />

<meta name="author" content="Auckland New Zealand">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="programme-at-a-glance.html">
<link rel="next" href="abstract.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html"><i class="fa fa-check"></i>Programme At A Glance</a><ul>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#monday"><i class="fa fa-check"></i>Monday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#tuesday"><i class="fa fa-check"></i>Tuesday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#wednesday"><i class="fa fa-check"></i>Wednesday</a></li>
<li class="chapter" data-level="" data-path="programme-at-a-glance.html"><a href="programme-at-a-glance.html#thursday"><i class="fa fa-check"></i>Thursday</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-monday-11th-of-december.html#programme-and-abstracts-for-monday-11th-of-december">Programme And Abstracts For Monday 11<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html"><i class="fa fa-check"></i>Effect of area level deprivation on body mass index: analysis of NZ Health Surveys</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#calendar-based-graphics-for-visualising-peoples-daily-schedules"><i class="fa fa-check"></i>Calendar-based graphics for visualising peopleâ€™s daily schedules</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#nonparametric-test-for-volatility-in-clustered-multiple-time-series-abstract"><i class="fa fa-check"></i>Nonparametric Test for Volatility in Clustered Multiple Time Series Abstract</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#igess-a-statistical-approach-to-integrating-individual-level-genotype-data-and-summary-statistics-in-genome-wide-association-studies"><i class="fa fa-check"></i>IGESS: A Statistical Approach to Integrating Individual Level Genotype Data and Summary Statistics in Genome Wide Association Studies</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#clustering-using-nonparametric-mixtures-and-mode-identification"><i class="fa fa-check"></i>Clustering using Nonparametric Mixtures and Mode Identification</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#bayesian-curve-fitting-for-discontinuous-function-using-overcomplete-representation-with-multiple-kernels"><i class="fa fa-check"></i>Bayesian curve fitting for discontinuous function using overcomplete representation with multiple kernels</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#estimation-of-a-semiparametric-spatiotemporal-models-with-mixed-frequency"><i class="fa fa-check"></i>Estimation of a Semiparametric Spatiotemporal Models with Mixed Frequency</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#lsmm-a-statistical-approach-to-integrating-functional-annotations-with-genome-wide-association-studies"><i class="fa fa-check"></i>LSMM: A statistical approach to integrating functional annotations with genome-wide association studies</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#estimation-of-a-high-dimensional-covariance-matrix"><i class="fa fa-check"></i>Estimation of a High-Dimensional Covariance Matrix</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#innovative-bayesian-estimation-in-the-von-mises-distribution"><i class="fa fa-check"></i>Innovative Bayesian Estimation in the von-Mises Distribution</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#evidence-of-climate-change-from-nonparametric-change-point-analysis"><i class="fa fa-check"></i>Evidence of Climate Change from Nonparametric Change-point Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#joint-analysis-of-individual-level-genotype-data-and-summary-statistics-by-leveraging-pleiotropy"><i class="fa fa-check"></i>Joint Analysis of Individual Level Genotype Data and Summary Statistics by Leveraging Pleiotropy</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#genetic-map-estimation-using-hidden-markov-models-in-the-presence-of-partially-observed-information"><i class="fa fa-check"></i>Genetic map estimation using hidden Markov models in the presence of partially observed information</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#a-simple-method-for-grouping-patients-based-on-historical-doses"><i class="fa fa-check"></i>A simple method for grouping patients based on historical doses</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#semiparametric-mixed-analysis-of-covariance-model"><i class="fa fa-check"></i>Semiparametric Mixed Analysis of Covariance Model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#adaptive-false-discovery-rate-regression-with-application-in-integrative-analysis-of-large-scale-genomic-data"><i class="fa fa-check"></i>Adaptive False Discovery Rate regression with application in integrative analysis of large-scale genomic data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#vector-generalized-linear-time-series-models"><i class="fa fa-check"></i>Vector Generalized Linear Time Series Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#local-canonical-correlation-analysis-for-multimodal-labeled-data"><i class="fa fa-check"></i>Local Canonical Correlation Analysis for Multimodal Labeled Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#canonical-covariance-analysis-for-mixed-numerical-and-categorical-three-way-three-mode-data"><i class="fa fa-check"></i>Canonical Covariance Analysis for Mixed Numerical and Categorical Three-way Three-mode Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#variable-selection-algorithms"><i class="fa fa-check"></i>Variable Selection Algorithms</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#estimating-causal-structures-for-continuous-and-discrete-variables"><i class="fa fa-check"></i>Estimating causal structures for continuous and discrete variables</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#incorporating-genetic-networks-into-case-control-association-studies-with-high-dimensional-dna-methylation-data"><i class="fa fa-check"></i>Incorporating genetic networks into case-control association studies with high-dimensional DNA methylation data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#adaptive-model-checking-for-functional-single-index-models"><i class="fa fa-check"></i>Adaptive Model Checking for Functional Single-Index Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#testing-for-genetic-associations-in-arbitrarily-structured-populations"><i class="fa fa-check"></i>Testing for genetic associations in arbitrarily structured populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#on-optimal-group-testing-designs-prevalence-estimation-cost-considerations-and-dilution-effects"><i class="fa fa-check"></i>On Optimal Group Testing Designs: Prevalence Estimation, Cost Considerations, and Dilution Effects</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#the-use-of-bayesian-networks-in-grape-yield-prediction"><i class="fa fa-check"></i>The Use of Bayesian Networks in Grape Yield Prediction</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="abstract.html"><a href="abstract.html"><i class="fa fa-check"></i><b>1</b> Abstract</a><ul>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#pattern-prediction-for-time-series-data-with-change-points"><i class="fa fa-check"></i>Pattern Prediction for Time Series Data with Change Points</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#test-for-genomic-imprinting-effects-on-the-x-chromosome"><i class="fa fa-check"></i>Test for Genomic Imprinting Effects on the X Chromosome</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#fluctuation-reduction-of-value-at-risk-estimation-and-its-applications"><i class="fa fa-check"></i>Fluctuation Reduction of Value-at-Risk Estimation and its Applications</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#meta-analytic-principal-component-analysis-in-integrative-omics-application"><i class="fa fa-check"></i>Meta-analytic Principal Component Analysis in Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#estimation-of-animal-density-from-acoustic-detections"><i class="fa fa-check"></i>Estimation of animal density from acoustic detections</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#mixed-models-for-complex-survey-data"><i class="fa fa-check"></i>Mixed models for complex survey data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#regression-with-random-effects-for-analysing-correlated-survival-data-application-to-disease-recurrences"><i class="fa fa-check"></i>Regression with random effects for analysing correlated survival data: application to disease recurrences</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#genetic-predictors-underlying-long-term-cognitive-recovery-following-mild-traumatic-brain-injury"><i class="fa fa-check"></i>Genetic predictors underlying long-term cognitive recovery following mild traumatic brain injury</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#bayesian-structure-selection-for-vector-autoregression-model"><i class="fa fa-check"></i>Bayesian Structure Selection for Vector Autoregression Model</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#phylogenetic-tree-based-microbiome-association-test"><i class="fa fa-check"></i>Phylogenetic Tree-based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#talk-data-to-me"><i class="fa fa-check"></i>Talk Data To Me</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#smooth-nonparametric-regression-under-shape-restrictions"><i class="fa fa-check"></i>Smooth Nonparametric Regression under Shape Restrictions</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#elastic-band-transform-a-new-approach-to-multiscale-visualization"><i class="fa fa-check"></i>Elastic-Band Transform: A New Approach to Multiscale Visualization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#a-computational-tool-for-detecting-copy-number-variations-from-whole-genome-and-targeted-exome-sequencing"><i class="fa fa-check"></i>A Computational Tool for Detecting Copy Number Variations from Whole Genome and Targeted Exome Sequencing</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#flight-to-relative-safety-learning-from-a-no-arbitrage-network-of-yield-curves-model-of-the-euro-area"><i class="fa fa-check"></i>Flight to relative safety: Learning from a no-arbitrage network of yield curves model of the euro area</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series"><i class="fa fa-check"></i>Wavelet-based power transformation of non-Gaussian long memory time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#robustness-of-temperature-reconstruction-for-past-500-years"><i class="fa fa-check"></i>Robustness of Temperature Reconstruction for Past 500 Years</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#nonparametric-causal-inference-by-the-kernel-method"><i class="fa fa-check"></i>Nonparametric causal inference by the kernel method</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#a-unified-regularized-group-pls-algorithm-scalable-to-big-data"><i class="fa fa-check"></i>A Unified Regularized Group PLS Algorithm Scalable to Big Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#meta-analysis-with-symbolic-data-analysis-and-its-application-for-clinical-data"><i class="fa fa-check"></i>Meta-analysis with symbolic data analysis and its application for clinical data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#scoring-rules-for-prediction-and-classification-challenges"><i class="fa fa-check"></i>Scoring rules for prediction and classification challenges</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#real-time-transit-network-modelling-for-improved-arrival-time-predictions"><i class="fa fa-check"></i>Real-time transit network modelling for improved arrival time predictions</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#sparse-group-subgroup-partial-least-squares-with-application-to-genomic-data"><i class="fa fa-check"></i>Sparse Group-subgroup Partial Least Squares with Application to Genomic Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#big-sir-a-sliced-inverse-regression-approach-for-massive-data"><i class="fa fa-check"></i>BIG-SIR a Sliced Inverse Regression Approach for Massive Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#comparison-of-tests-of-mean-difference-in-longitudinal-data-based-on-block-resampling-methods"><i class="fa fa-check"></i>Comparison of Tests of Mean Difference in Longitudinal Data Based on Block Resampling Methods</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#modeling-of-document-abstraction-using-association-rule-based-characterization"><i class="fa fa-check"></i>Modeling of Document Abstraction Using Association Rule Based Characterization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#bayesian-survival-analysis-of-batsmen-in-test-cricket"><i class="fa fa-check"></i>Bayesian survival analysis of batsmen in Test cricket</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#covariate-discretisation-on-big-data"><i class="fa fa-check"></i>Covariate Discretisation on Big Data</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#visualization-and-statistical-modeling-of-financial-big-data"><i class="fa fa-check"></i>Visualization and Statistical Modeling of Financial Big Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#analysis-of-spatial-data-with-a-gaussian-mixture-markov-random-field-model"><i class="fa fa-check"></i>Analysis of Spatial Data with a Gaussian Mixture Markov Random Field Model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html#my-knee-still-hurts-the-statistical-pathway-to-the-development-of-a-clinical-decision-aid"><i class="fa fa-check"></i>My knee still hurts; the statistical pathway to the development of a clinical decision aid</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#programme-and-abstracts-for-tuesday-12th-of-december">Programme And Abstracts For Tuesday 12<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html"><i class="fa fa-check"></i>R&amp;D policy regimes in France: New evidence from a spatio-temporal analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#analysing-scientific-collaborations-of-new-zealand-institutions-using-scopus-bibliometric-data"><i class="fa fa-check"></i>Analysing Scientific Collaborations of New Zealand Institutions using Scopus Bibliometric Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#a-practitioners-guide-to-deep-learning-for-predictive-analytics-on-structured-data"><i class="fa fa-check"></i>A Practitioners Guide to Deep Learning for Predictive Analytics on Structured Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#analysis-of-multivariate-binary-longitudinal-data-metabolic-syndrome-during-menopausal-transition"><i class="fa fa-check"></i>Analysis of Multivariate Binary Longitudinal Data: Metabolic Syndrome During Menopausal Transition</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#clustering-of-curves-on-a-spatial-domain-using-a-bayesian-partitioning-model"><i class="fa fa-check"></i>Clustering of curves on a spatial domain using a Bayesian partitioning model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#the-uncomfortable-entrepreneurs-bad-working-conditions-and-entrepreneurial-commitment"><i class="fa fa-check"></i>The Uncomfortable Entrepreneurs: Bad Working Conditions and Entrepreneurial Commitment</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#spatial-surveillance-with-scan-statistics-by-controlling-the-false-discovery-rate"><i class="fa fa-check"></i>Spatial Surveillance with Scan Statistics by Controlling the False Discovery Rate</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#statistical-models-for-the-source-attribution-of-zoonotic-diseases-a-study-of-campylobacteriosis"><i class="fa fa-check"></i>Statistical models for the source attribution of zoonotic diseases: A study of campylobacteriosis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#family-structure-and-academic-achievements-of-high-school-students-in-tonga"><i class="fa fa-check"></i>Family structure and academic achievements of high school students in Tonga</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#identifying-clusters-of-patients-with-diabetes-using-a-markov-birth-death-process"><i class="fa fa-check"></i>Identifying Clusters Of Patients With Diabetes Using a Markov Birth-Death Process</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-temporal-density-estimation-using-autoregressive-species-sampling-models"><i class="fa fa-check"></i>Bayesian Temporal Density Estimation using Autoregressive Species Sampling Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#intensity-estimation-of-spatial-point-processes-based-on-area-aggregated-data"><i class="fa fa-check"></i>Intensity Estimation of Spatial Point Processes Based on Area-Aggregated Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-inference-for-population-attributable-measures"><i class="fa fa-check"></i>Bayesian Inference for Population Attributable Measures</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#latent-variable-models-and-multivariate-binomial-data"><i class="fa fa-check"></i>Latent variable models and multivariate binomial data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#analysis-of-a-brief-telephone-intervention-for-problem-gambling-and-examining-the-impact-on-co-existing-depression"><i class="fa fa-check"></i>Analysis of a brief telephone intervention for problem gambling and examining the impact on co-existing depression?</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#prior-based-bayesian-information-criterion"><i class="fa fa-check"></i>Prior-based Bayesian Information Criterion</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#correlated-defaults-with-a-distance-to-default"><i class="fa fa-check"></i>Correlated Defaults with a Distance to Default</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#geographically-weighted-principal-component-analysis-for-spatio-temporal-statistical-dataset"><i class="fa fa-check"></i>Geographically Weighted Principal Component Analysis for Spatio-temporal Statistical Dataset</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#dimensionality-reduction-of-multivariate-data-for-bayesian-analysis"><i class="fa fa-check"></i>Dimensionality Reduction of Multivariate Data for Bayesian Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#an-information-criterion-for-prediction-with-auxiliary-variables-under-covariate-shift"><i class="fa fa-check"></i>An information criterion for prediction with auxiliary variables under covariate shift</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#adjusting-for-linkage-bias-in-the-analysis-of-record-linked-data"><i class="fa fa-check"></i>Adjusting for Linkage Bias in the Analysis of Record-Linked Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-semiparametric-hierarchical-models-for-longitudinal-data-analysis-with-application-to-dose-response-studies"><i class="fa fa-check"></i>Bayesian semiparametric hierarchical models for longitudinal data analysis with application to dose-response studies</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#optimizing-junior-rugby-weight-limits"><i class="fa fa-check"></i>Optimizing Junior Rugby Weight Limits</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#spatial-scan-statistics-for-matched-case-control-data"><i class="fa fa-check"></i>Spatial scan statistics for matched case-control data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#whitebait-in-all-its-varieties-one-fish-two-fish-three-four-five-fish."><i class="fa fa-check"></i>Whitebait in all its varieties: one fish, two fish, three, four, five fish.</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#an-advanced-approach-for-time-series-forecasting-using-deep-learning"><i class="fa fa-check"></i>An Advanced Approach for Time Series Forecasting using Deep Learning</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#asking-about-sex-in-general-health-surveys-comparing-the-methods-and-findings-of-the-2010-health-survey-for-england-with-those-of-the-third-national-survey-of-sexual-attitudes-and-lifestyles"><i class="fa fa-check"></i>Asking about Sex in General Health Surveys: Comparing the Methods and Findings of the 2010 Health Survey for England with Those of the Third National Survey of Sexual Attitudes and Lifestyles</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#tolerance-limits-for-the-reliability-of-semiconductor-devices-using-longitudinal-data"><i class="fa fa-check"></i>Tolerance limits for the reliability of semiconductor devices using longitudinal data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#model-checking-for-regressions-a-local-smoothing-based-global-smoothing-test"><i class="fa fa-check"></i>Model-checking for regressions: a local smoothing-based global smoothing test</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#breeding-value-estimation-in-partially-genotyped-populations"><i class="fa fa-check"></i>Breeding Value Estimation in Partially-Genotyped Populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bivas-a-scalable-bayesian-method-for-bi-level-variable-selection"><i class="fa fa-check"></i>BIVAS: A scalable Bayesian method for Bi-level Variable Selection</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#ranking-potential-shoplifters-in-real-time"><i class="fa fa-check"></i>Ranking Potential Shoplifters in Real Time</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#two-stage-approach-to-data-driven-subgroup-identification-in-clinical-trials"><i class="fa fa-check"></i>Two stage approach to data-driven subgroup identification in clinical trials</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#estimation-and-prediction-for-a-distribution-with-bathtub-shape-under-progressive-first-failure-censoring"><i class="fa fa-check"></i>Estimation and prediction for a distribution with bathtub shape under progressive first failure censoring</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#inverse-regression-for-multivariate-functional-data"><i class="fa fa-check"></i>Inverse Regression for Multivariate Functional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#including-covariate-estimation-error-when-predicting-species-distributions-a-simulation-exercise-using-template-model-builder"><i class="fa fa-check"></i>Including covariate estimation error when predicting species distributions: a simulation exercise using Template Model Builder</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#adjusted-adaptive-index-model-for-binary-response"><i class="fa fa-check"></i>Adjusted adaptive index model for binary response</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#factors-influencing-on-growth-of-garments-industry-in-bangladesh"><i class="fa fa-check"></i>Factors Influencing on Growth of Garments Industry in Bangladesh</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#comparison-of-exact-and-approximate-testing-procedures-in-clinical-trials-with-multiple-binary-endpoints"><i class="fa fa-check"></i>Comparison of Exact and Approximate Testing Procedures in Clinical Trials with Multiple Binary Endpoints</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#bayesian-optimum-warranty-length-under-type-ii-unified-hybrid-censoring-scheme"><i class="fa fa-check"></i>Bayesian optimum warranty length under Type-II unified hybrid censoring scheme</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#multiple-function-on-function-linear-regression-with-application-to-weather-forecast-calibration"><i class="fa fa-check"></i>Multiple function-on-function linear regression with application to weather forecast calibration</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#modelling-the-distribution-of-lifetime-using-compound-time-homogenous-poisson-process"><i class="fa fa-check"></i>Modelling the distribution of lifetime using compound time-homogenous Poisson process</a></li>
<li><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#detecting-change-points-in-the-stress-strength-reliability-px"><span class="toc-section-number">1.1</span> Detecting Change-Points in the Stress-Strength Reliability P(X<Y) {-}
<p style="text-align:center"></a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#new-zealand-crime-and-victims-survey-filling-the-knowledge-gap"><i class="fa fa-check"></i>New Zealand Crime and Victims Survey: filling the knowledge gap</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-tuesday-12th-of-december.html"><a href="programme-and-abstracts-for-tuesday-12th-of-december.html#missing-data-in-randomised-control-trials-stepped-multiple-imputation"><i class="fa fa-check"></i>Missing Data in Randomised Control Trials: Stepped Multiple Imputation</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#programme-and-abstracts-for-wednesday-13th-of-december">Programme And Abstracts For Wednesday 13<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html"><i class="fa fa-check"></i>Promoting your R package</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-smoothing-filter-modelling-approach-for-time-series"><i class="fa fa-check"></i>A smoothing filter modelling approach for time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#online-learning-for-bayesian-nonparametrics-weakly-conjugate-approximation"><i class="fa fa-check"></i>Online Learning for Bayesian Nonparametrics: Weakly Conjugate Approximation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#improving-the-production-cycle-at-stats-nz-with-rstudio"><i class="fa fa-check"></i>Improving the production cycle at Stats NZ with RStudio</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#sparse-common-component-analysis"><i class="fa fa-check"></i>Sparse Common Component Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#random-search-global-optimization-using-random-forests"><i class="fa fa-check"></i>Random Search Global Optimization using Random Forests</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#gridsvg-then-and-now"><i class="fa fa-check"></i>gridSVG: Then and Now</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#probabilistic-outlier-detection-and-visualization-of-smart-meter-data"><i class="fa fa-check"></i>Probabilistic outlier detection and visualization of smart meter data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#the-joint-models-for-nonlinear-longitudinal-and-time-to-event-data-using-penalized-splines-a-bayesian-approach"><i class="fa fa-check"></i>The joint models for nonlinear longitudinal and time-to-event data using penalized splines: A Bayesian approach</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-a-powerful-analysis-tool-to-improve-official-statistics-in-romania"><i class="fa fa-check"></i>R â€“ A powerful analysis tool to improve Official Statistics in Romania</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#robust-principal-expectile-component-analysis"><i class="fa fa-check"></i>Robust Principal Expectile Component Analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bringing-multimix-from-fortran-to-r"><i class="fa fa-check"></i>Bringing Multimix from Fortran to R</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#specification-of-garch-model-under-asymmetric-error-innovations"><i class="fa fa-check"></i>SPECIFICATION OF GARCH MODEL UNDER ASYMMETRIC ERROR INNOVATIONS</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#the-joint-models-for-nonlinear-longitudinal-and-time-to-event-data-using-penalized-splines-a-bayesian-approach-1"><i class="fa fa-check"></i>The joint models for nonlinear longitudinal and time-to-event data using penalized splines: A Bayesian approach</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#statistical-disclosure-control-with-r-traditional-methods-and-synthetic-data"><i class="fa fa-check"></i>Statistical Disclosure Control with R: Traditional Methods and Synthetic Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-max-type-multivariate-two-sample-baumgartner-statistic"><i class="fa fa-check"></i>A Max-Type Multivariate Two-Sample Baumgartner Statistic</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#deep-learning-high-dimensional-covariance-matrices"><i class="fa fa-check"></i>Deep Learning High-Dimensional Covariance Matrices</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-in-industry-application-on-pipe-renewal-planning"><i class="fa fa-check"></i>R in Industry â€“ Application on Pipe Renewal Planning</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#empirical-comparison-of-some-algorithms-for-automatic-univariate-arma-modeling-using-rcmdrplugin.spss"><i class="fa fa-check"></i>Empirical comparison of some algorithms for automatic univariate ARMA modeling using RcmdrPlugin.SPSS</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bayesian-analyses-of-non-homogeneous-gaussian-hidden-markov-models"><i class="fa fa-check"></i>Bayesian Analyses of Non-homogeneous Gaussian Hidden Markov Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#imputation-of-the-2016-economic-census-for-business-activity-in-japan"><i class="fa fa-check"></i>Imputation of the 2016 Economic Census for Business Activity in Japan</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#simultaneous-test-for-mean-vectors-and-covariance-matrices-in-high-dimensional-settings"><i class="fa fa-check"></i>Simultaneous test for mean vectors and covariance matrices in high-dimensional settings</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#applying-active-learning-procedure-to-drug-consumption-data"><i class="fa fa-check"></i>Applying Active Learning Procedure to Drug Consumption Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-for-everything"><i class="fa fa-check"></i>R for Everything</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#r-package-for-new-two-stage-methods-in-forecasting-time-series-with-multiple-seasonality"><i class="fa fa-check"></i>R Package for New Two-Stage Methods in Forecasting Time Series with Multiple Seasonality</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#performance-of-bayesian-credible-interval-for-binomial-proportion-using-logit-transformation"><i class="fa fa-check"></i>Performance of Bayesian Credible Interval for Binomial Proportion using Logit Transformation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#analysis-of-official-microdata-using-secure-statistical-computation-system"><i class="fa fa-check"></i>Analysis of Official Microdata Using Secure Statistical Computation System</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#high-dimensional-asymptotics-for-the-naive-canonical-correlation-coefficient"><i class="fa fa-check"></i>High Dimensional Asymptotics for the Naive Canonical Correlation Coefficient</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#presenting-flexi-a-statistical-program-for-fitting-variance-models"><i class="fa fa-check"></i>Presenting Flexi, A Statistical Program for Fitting Variance Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#cluster-wise-regression-models-combined-by-a-quasi-linear-function"><i class="fa fa-check"></i>Cluster-wise regression models combined by a quasi-linear function</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#hierarchical-structural-component-analysis-of-gene-environment-interactions"><i class="fa fa-check"></i>Hierarchical Structural Component Analysis of Gene-Environment Interactions</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series-1"><i class="fa fa-check"></i>Wavelet-based power transformation of non-Gaussian long memory time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#cross-covariance-estimation-for-integration-of-multi-omics-data"><i class="fa fa-check"></i>Cross covariance estimation for integration of multi-omics data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#relationships-between-linguistic-characteristics-and-the-use-of-maori-loanwords-in-new-zealand-english."><i class="fa fa-check"></i>Relationships between linguistic characteristics and the use of MÄori loanwords in New Zealand English.</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#transfer-regression-and-predictive-distributions"><i class="fa fa-check"></i>Transfer Regression and Predictive Distributions</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#fluctuation-reduction-of-value-at-risk-estimation-and-its-applications-1"><i class="fa fa-check"></i>Fluctuation Reduction of Value-at-Risk Estimation and its Applications</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#testing-for-genetic-associations-in-arbitrarily-structured-populations-1"><i class="fa fa-check"></i>Testing for genetic associations in arbitrarily structured populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bayesian-analysis-for-fitting-zero-inflated-count-data-with-data-augmentation"><i class="fa fa-check"></i>Bayesian analysis for fitting zero-inflated count data with data augmentation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#regularized-noise-reduction-methodology-for-high-dimensional-data"><i class="fa fa-check"></i>Regularized Noise-Reduction Methodology for High-Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#computation-of-influence-functions-for-robust-statistics"><i class="fa fa-check"></i>Computation of influence functions for robust statistics</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#adaptive-model-averaging-in-high-dimensional-linear-regression"><i class="fa fa-check"></i>Adaptive Model Averaging in High-Dimensional Linear Regression</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#meta-analytic-principal-component-analysis-in-integrative-omics-application-1"><i class="fa fa-check"></i>Meta-analytic Principal Component Analysis in Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#model-based-clustering-for-multivariate-categorical-data-with-dimension-reduction"><i class="fa fa-check"></i>Model-based clustering for multivariate categorical data with dimension reduction</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#selecting-the-number-of-principal-components"><i class="fa fa-check"></i>Selecting the number of principal components</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#rolling-survival-extrapolation-algorithm-for-estimating-life-years-lost-among-subjects-exposed-to-long-term-air-pollution"><i class="fa fa-check"></i>Rolling survival extrapolation algorithm for estimating life years lost among subjects exposed to long-term air pollution</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#enhancing-the-flexibility-of-regression-modeling-by-liquid-association"><i class="fa fa-check"></i>Enhancing the flexibility of regression modeling by liquid association</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#flight-to-relative-safety-learning-from-a-no-arbitrage-network-of-yield-curves-model-of-the-euro-area-1"><i class="fa fa-check"></i>Flight to relative safety: Learning from a no-arbitrage network of yield curves model of the euro area</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#phylogenetic-tree-based-microbiome-association-test-1"><i class="fa fa-check"></i>Phylogenetic Tree-based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#towards-a-sparse-scalable-and-stably-positive-definite-inverse-covariance-estimator"><i class="fa fa-check"></i>Towards a sparse, scalable, and stably positive definite (inverse) covariance estimator</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#high-mortality-predictions-with-lines-or-curves-fitted-to-over-dispersed-exposure-mortality-data"><i class="fa fa-check"></i>High Mortality Predictions with Lines or Curves Fitted to Over-Dispersed Exposure-Mortality Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#lattice-polytope-samplers"><i class="fa fa-check"></i>Lattice Polytope Samplers</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#statistical-modelling-and-analysis-of-cosmic-microwave-background-data"><i class="fa fa-check"></i>Statistical Modelling and Analysis of Cosmic Microwave Background Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#measure-of-departure-from-marginal-average-point-symmetry-for-two-way-contingency-tables-with-ordered-categories"><i class="fa fa-check"></i>Measure of Departure from Marginal Average Point-Symmetry for Two-Way Contingency Tables with Ordered Categories</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#sparse-estimates-from-dense-precision-matrix-posteriors"><i class="fa fa-check"></i>Sparse estimates from dense precision matrix posteriors</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#dimension-reduction-strategies-for-modeling-bi-clustered-high-dimensional-data"><i class="fa fa-check"></i>Dimension Reduction Strategies for Modeling Bi-clustered High Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#computing-entropies-with-nested-sampling"><i class="fa fa-check"></i>Computing Entropies with Nested Sampling</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#spline-based-drift-models-for-high-temperature-operating-life-tests"><i class="fa fa-check"></i>Spline-based Drift Models for High Temperature Operating Life Tests</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-new-approach-to-distribution-free-tests-in-contingency-tables"><i class="fa fa-check"></i>A new approach to distribution free tests in contingency tables</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#bayesian-continuous-space-time-model-of-burglaries"><i class="fa fa-check"></i>Bayesian continuous space-time model of Burglaries</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#modified-gene-shaving-algorithm---a-dimension-reduction-and-clustering-method"><i class="fa fa-check"></i>Modified Gene Shaving Algorithm - A Dimension REduction and Clustering Method</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#a-simple-method-to-construct-confidence-bands-in-functional-linear-regression"><i class="fa fa-check"></i>A simple method to construct confidence bands in functional linear regression</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#separation-of-symmetry-for-square-contingency-tables-with-ordinal-categories"><i class="fa fa-check"></i>Separation of symmetry for square contingency tables with ordinal categories</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#an-overview-of-the-correspondence-analysis-family"><i class="fa fa-check"></i>An overview of the correspondence analysis family</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-wednesday-13th-of-december.html"><a href="programme-and-abstracts-for-wednesday-13th-of-december.html#testing-for-presence-of-clustering-effect-in-multilevel-model-with-high-dimensional-predictors"><i class="fa fa-check"></i>TESTING FOR PRESENCE OF CLUSTERING EFFECT IN MULTILEVEL MODEL WITH HIGH-DIMENSIONAL PREDICTORS</a></li>
</ul></li>
<li><a href="programme-and-abstracts-for-thursday-14th-of-december.html#programme-and-abstracts-for-thursday-14th-of-december">Programme And Abstracts For Thursday 14<sup>th</sup> Of December</a><ul>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html"><i class="fa fa-check"></i>Penalized Vector Generalized Additive Models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#author-name-identification-for-evaluating-research-performance-of-institutes"><i class="fa fa-check"></i>Author Name Identification for Evaluating Research Performance of Institutes</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#cross-covariance-estimation-for-integration-of-multi-omics-data-1"><i class="fa fa-check"></i>Cross covariance estimation for integration of multi-omics data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#evaluation-of-spatial-cluster-detection-method-based-on-all-geographical-linkage-patterns"><i class="fa fa-check"></i>Evaluation of spatial cluster detection method based on all geographical linkage patterns</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#bayesian-static-parameter-inference-for-partially-observed-stochastic-systems"><i class="fa fa-check"></i>Bayesian Static Parameter Inference for Partially Observed Stochastic Systems</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#testing-for-genetic-associations-in-arbitrarily-structured-populations-2"><i class="fa fa-check"></i>Testing for genetic associations in arbitrarily structured populations</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#consistency-of-linear-mixed-effects-model-selection-with-inconsistent-covariance-parameter-estimators"><i class="fa fa-check"></i>Consistency of Linear Mixed-Effects Model Selection with Inconsistent Covariance Parameter Estimators</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#a-study-of-the-influence-of-articles-in-the-large-scale-citation-network"><i class="fa fa-check"></i>A Study of the Influence of Articles in the Large-Scale Citation Network</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#regularized-noise-reduction-methodology-for-high-dimensional-data-1"><i class="fa fa-check"></i>Regularized Noise-Reduction Methodology for High-Dimensional Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#genetic-approach-and-statistical-approach-for-association-study-on-dna-data"><i class="fa fa-check"></i>Genetic Approach and Statistical Approach for Association Study on DNA Data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#ssrem-a-summary-statistics-based-random-effect-model-to-estimating-heritability-co-heritability-and-effect-sizes-in-gwas-data-analysis"><i class="fa fa-check"></i>SSREM: A Summary-Statistics-based Random Effect Model to Estimating heritability, co-heritability and effect sizes in GWAS data analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#meta-analytic-principal-component-analysis-in-integrative-omics-application-2"><i class="fa fa-check"></i>Meta-analytic Principal Component Analysis in Integrative Omics Application</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#selecting-generalised-linear-models-under-inequality-constraints"><i class="fa fa-check"></i>Selecting Generalised Linear Models Under Inequality Constraints</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#clustering-of-research-subject-based-on-stochastic-block-model"><i class="fa fa-check"></i>Clustering of research subject based on stochastic block model</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#selecting-the-number-of-principal-components-1"><i class="fa fa-check"></i>Selecting the number of principal components</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#symbolic-data-analytical-approach-to-unauthorized-access-logs"><i class="fa fa-check"></i>Symbolic data analytical approach to unauthorized-access logs</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#forward-selection-in-regression-models-based-on-robust-estimation"><i class="fa fa-check"></i>Forward selection in regression models based on robust estimation</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#phylogenetic-tree-based-microbiome-association-test-2"><i class="fa fa-check"></i>Phylogenetic Tree-based Microbiome Association Test</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#statistical-generalized-derivative-applied-to-the-profile-likelihood-estimation-in-a-mixture-of-semiparametric-models"><i class="fa fa-check"></i>Statistical generalized derivative applied to the profile likelihood estimation in a mixture of semiparametric models</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#structure-of-members-in-the-organization-to-induce-innovation-quantitatively-analyze-the-capability-of-the-organization"><i class="fa fa-check"></i>Structure of Members in the Organization to Induce Innovation: Quantitatively Analyze the Capability of the Organization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#towards-a-sparse-scalable-and-stably-positive-definite-inverse-covariance-estimator-1"><i class="fa fa-check"></i>Towards a sparse, scalable, and stably positive definite (inverse) covariance estimator</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#statistical-challenges-for-proteogenomic-data-analysis"><i class="fa fa-check"></i>Statistical challenges for proteogenomic data analysis</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#feature-selection-in-high-dimensional-models-with-complex-block-structures"><i class="fa fa-check"></i>Feature selection in high-dimensional models with complex block structures</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#wavelet-based-power-transformation-of-non-gaussian-long-memory-time-series-2"><i class="fa fa-check"></i>Wavelet-based power transformation of non-Gaussian long memory time series</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#dissimilarities-between-groups-of-data"><i class="fa fa-check"></i>Dissimilarities between groups of data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#a-package-for-multiple-precision-floating-point-computation-on-r"><i class="fa fa-check"></i>A package for multiple precision floating-point computation on R</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#cluster-wise-regression-models-combined-by-a-quasi-linear-function-1"><i class="fa fa-check"></i>Cluster-wise regression models combined by a quasi-linear function</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#mobile-learning-in-teaching-bioinformatics-for-medical-doctors"><i class="fa fa-check"></i>Mobile Learning in teaching Bioinformatics for medical doctors</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#interactive-visualization-of-aggregated-symbolic-data"><i class="fa fa-check"></i>Interactive visualization of aggregated symbolic data</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#an-incomplete-data-fisher-scoring-with-an-acceleration-method"><i class="fa fa-check"></i>An incomplete-data Fisher scoring with an acceleration method</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#clusterwise-low-rank-correlation-analysis-based-on-majorization"><i class="fa fa-check"></i>Clusterwise low-rank correlation analysis based on majorization</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#towards-an-informal-test-for-goodness-of-fit"><i class="fa fa-check"></i>Towards an informal test for goodness-of-fit</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#an-ewma-chart-for-monitoring-covariance-matrix-based-on-dissimilarity-index"><i class="fa fa-check"></i>An EWMA chart for monitoring covariance matrix based on dissimilarity index</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#improvement-of-computation-for-nonlinear-multivariate-methods"><i class="fa fa-check"></i>Improvement of Computation for Nonlinear Multivariate Methods</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-thursday-14th-of-december.html"><a href="programme-and-abstracts-for-thursday-14th-of-december.html#three-dimensional-data-visualization-education-with-virtual-reality"><i class="fa fa-check"></i>Three-Dimensional Data Visualization Education with Virtual Reality</a></li>
<li class="chapter" data-level="" data-path="programme-and-abstracts-for-monday-11th-of-december.html"><a href="programme-and-abstracts-for-monday-11th-of-december.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">New Zealand Statistical Association and the International Associate of Statistical Computing (Asian Region) Joint Conference 2017</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="programme-and-abstracts-for-monday-11th-of-december" class="section level1 unnumbered">
<h1>Programme And Abstracts For Monday 11<sup>th</sup> Of December</h1>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 10:30 OGGB4 (260-073)
</p>
<div id="effect-of-area-level-deprivation-on-body-mass-index-analysis-of-nz-health-surveys" class="section level2 unnumbered">
<h2>Effect of area level deprivation on body mass index: analysis of NZ Health Surveys</h2>
<p style="text-align:center">
Andrew Adiguna Halim, Arindam Basu, and Raymond Kirk<br /> <br />
</p>
<p>Title: Area level deprivation is independently associated with body mass index: Analysis of NZ Health Surveys</p>
<p>Author: Andrew Adiguna Halim Arindam Basu Ray Kirk</p>
<p>Abstract: Obesity is a growing public health problem in New Zealand but the trends of its determinants are unclear. We obtained the confidentialised unit record files (CURF) of the New Zealand Health Surveys (NZHS) from the Statistics New Zealand, containing multiple sets of anonymised individual level data from 2002/03 to 2014/15. We assessed the association between deprivation quintile and compliance with the dietary guideline, and the prevalence of overweight/obesity. For adults, we converted Body Mass Index (BMI) variable into tertiles. Then we regressed the BMI tertiles on deprivation level, ethnicity, age, sex, physical activity, education, smoking status, fruit guideline, vegetable guideline, and household income variables using stepwise ordinal logistic regression with complex survey design. We regressed the BMI categories on deprivation level, ethnicity, age, sex, household income, education, fruit guideline, vegetable guideline, soft drink consumption, and fast food consumption in the child data. We found that people living in the highest deprivation quintile were more likely to be in the higher BMI tertile in adults and BMI category in children compared with those living in the lowest deprivation quintile after adjusting for other confounding variables. For adults and children the ORs (95</p>
<p>Keywords: obesity, BMI, dietary guideline, deprivation, r statistics, proportional odds regression, survey complex design.</p>
Contact Details: Andrew Adiguna Halim <a href="mailto:aha164@uclive.ac.nz">aha164@uclive.ac.nz</a> 0211412636
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 10:30 OGGB5 (260-051)
</p>
</div>
<div id="calendar-based-graphics-for-visualising-peoples-daily-schedules" class="section level2 unnumbered">
<h2>Calendar-based graphics for visualising peopleâ€™s daily schedules</h2>
<p style="text-align:center">
Earo Wang, Dianne Cook, and Rob Hyndman<br /> <br />
</p>
<p><span><strong>Calendar-based graphics for visualising peopleâ€™s daily schedules</strong></span></p>
<p>Earo Wang<span class="math inline">\(^1\)</span>, Dianne Cook<span class="math inline">\(^1\)</span> and Rob J Hyndman<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Econometrics and Business Statistics, Monash University, VIC 3800, Australia</p>
<p><span><strong>Abstract</strong></span>. This paper describes a <code>frame_calendar</code> function that organises and displays temporal data, collected on sub-daily resolution, into a calendar layout. Calendars are broadly used in society to display temporal information, and events. The <code>frame_calendar</code> uses linear algebra on the date variable to create the layout. It utilises the grammar of graphics to create the plots inside each cell, and thus synchronises neatly with <span><strong>ggplot2</strong></span> graphics. The motivating application is studying pedestrian behaviour in Melbourne, Australia, based on counts which are captured at hourly intervals by sensors scattered around the city. Faceting by the usual features such as day and month, was insufficient to examine the behaviour. Making displays on a monthly calendar format helps to understand pedestrian patterns relative to events such as work days, weekends, holidays, and special events. The layout algorithm has several format options and variations. It is implemented in the R package <span><strong>sugrrants</strong></span>.</p>
<p><span><strong>Keywords</strong></span>. data visualisation, statistical graphics, time series, R package, grammar of graphics</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Van Wijk JJ, Van Selow ER (1999). Cluster and Calendar Based Visualization of Time Series Data. In <em>Information Visualization, 1999.(Info Visâ€™ 99) Proceedings</em>. 4â€“9.</p>
<p>Wickham H (2009). <em>ggplot2: Elegant Graphics for Data Analysis.</em> Springer-Verlag New York, New York, NY.</p>
Wickham H, Hofmann H, Wickham C, Cook D (2012). Glyph-maps for Visually Exploring Temporal Patterns in Climate Data and Models. <em>Environmetrics</em>, <strong>23</strong>(5), 382â€“393.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 10:30 Case Room 2 (260-057)
</p>
</div>
<div id="nonparametric-test-for-volatility-in-clustered-multiple-time-series-abstract" class="section level2 unnumbered">
<h2>Nonparametric Test for Volatility in Clustered Multiple Time Series Abstract</h2>
<p style="text-align:center">
Paolo Victor Redondo and Erniel Barrios<br /> <br />
</p>
<p>Testing Volatility in Clustered Multiple Time Series: A Nonparametric Approach</p>
<p>Erniel B. Barrios and Paolo Victor T. Redondo School of Statistics, University of the Philippines Diliman Abstract</p>
<p>We proposed a test for volatility in clustered multiple time series based on sieve bootstrap. Clustering of observations is intended to capture contagion effect in multiple time series data, assumed to be present in the data generating process where the test is based from. We designed a simulation study to evaluate the test procedure. The method is further illustrated using data on global stock prices and rice production among Asian countries. The test is potentially robust to some distributional assumption but is possibly affected by the nature of volatility.</p>
Keywords: multiple time series; volatility; nonparametric test; Sieve Bootstrap
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 10:30 Case Room 3 (260-055)
</p>
</div>
<div id="igess-a-statistical-approach-to-integrating-individual-level-genotype-data-and-summary-statistics-in-genome-wide-association-studies" class="section level2 unnumbered">
<h2>IGESS: A Statistical Approach to Integrating Individual Level Genotype Data and Summary Statistics in Genome Wide Association Studies</h2>
<p style="text-align:center">
Mingwei Dai<sup>1</sup>, Jingsi Ming<sup>2</sup>, Mingxuan Cai<sup>2</sup>, Jin Liu<sup>3</sup>, Can Yang<sup>4</sup>, Xiang Wan<sup>2</sup>, and Zongben Xu<sup>1</sup><br />
</p>
<p><span><strong>LSMM: A statistical approach to integrating functional annotations with genome-wide association studies</strong></span></p>
<p>Jingsi Ming<span class="math inline">\(^1\)</span>, Mingwei Dai<span class="math inline">\(^{2,5}\)</span>, Mingxuan Cai<span class="math inline">\(^1\)</span>, Xiang Wan<span class="math inline">\(^3\)</span>, Jin Liu<span class="math inline">\(^4\)</span> and Can Yang<span class="math inline">\(^5\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Mathematics, Hong Kong Baptist University, Hong Kong</p>
<p><span class="math inline">\(^2 \;\)</span> School of Mathematics and Statistics, Xiâ€™an Jiaotong University, Xiâ€™an, China</p>
<p><span class="math inline">\(^3 \;\)</span> Department of Computer Science, Hong Kong Baptist University, Hong Kong</p>
<p><span class="math inline">\(^4 \;\)</span> Centre for Quantitative Medicine, Duke-NUS Medical School, Singapore</p>
<p><span class="math inline">\(^5 \;\)</span> Department of Mathematics, The Hong Kong University of Science and Technology, Hong Kong</p>
<span><strong>Abstract</strong></span>. Thousands of risk variants underlying complex phenotypes have been identified in genome-wide association studies (GWAS). However, there are two major challenges towards fully characterizing the biological basis of complex diseases. First, many complex traits are suggested to be highly polygenic, whereas a large proportion of risk variants with small effects remains unknown. Second, the functional roles of the majority of GWAS hits in the non-coding region is largely unclear. In this paper, we propose a latent sparse mixed model (LSMM) to address the challenges by integrating functional annotations with summary statistics from GWAS. An efficient variational expectation-maximization (EM) algorithm is developed. We conducted comprehensive simulation studies and then applied it to 30 GWAS of complex phenotypes integrating 9 genic annotation categories and 127 tissue-specific functional annotations from the Roadmap project. The results demonstrate that LSMM is not only able to increase the statistical power to identify risk variants, but also provide a deeper understanding of genetic architecture of complex traits by detecting relevant functional annotations. <span><strong>Keywords</strong></span>. GWAS, functional annotations, variational inference
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 10:50 OGGB4 (260-073)
</p>
</div>
<div id="clustering-using-nonparametric-mixtures-and-mode-identification" class="section level2 unnumbered">
<h2>Clustering using Nonparametric Mixtures and Mode Identification</h2>
<p style="text-align:center">
Shengwei Hu and Yong Wang<br /> <br />
</p>
<p><span><strong>Clustering using Nonparametric Mixtures and Mode Identification</strong></span></p>
<p>Shengwei Hu<span class="math inline">\(^1\)</span> and Yong Wang<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, the University of Auckland, New Zealand</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Statistics, the University of Auckland, New Zealand</p>
<p><span><strong>Abstract</strong></span>. Clustering aims to partition a set of observations into a proper number of clusters with similar objects allocated to the same group. Current partitioning methods mainly include those based on some measure of distance or probability distribution. Here we propose a mode-based clustering methodology motivated via density estimation and mode identification procedures. The idea is to estimate the data-generating probability distribution using a nonparametric density estimator and then locate the modes of the density obtained. In the nonparametric mixture models, each mode and the observations ascend to it correspond to a single cluster. Thus, the problem of determining the number of clusters can be recast as a mode merging problem. A criterion of measuring the separability between modes is also addressed in this work. The most similar modes would be merged sequentially until the optimal number of clusters is reached. The performance of the proposed method is investigated on both simulated and real datasets.</p>
<p><span><strong>Keywords</strong></span>. Clustering, Nonparametric mixtures, Mode identification</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Wang, X. and Wang, Y.: <em>Nonparametric multivariate density estimation using mixtures</em>. Stat. Comput. <strong>25</strong>, 349â€“-364 (2015).</p>
Li, J., Ray S. and Lindsay B.G.: <em>A nonparametric statistical approach to clustering via mode identification</em>. Journal of Machine Learning Research. <strong>8</strong>, 1687â€“-1723 (2007).
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 10:50 OGGB5 (260-051)
</p>
</div>
<div id="bayesian-curve-fitting-for-discontinuous-function-using-overcomplete-representation-with-multiple-kernels" class="section level2 unnumbered">
<h2>Bayesian curve fitting for discontinuous function using overcomplete representation with multiple kernels</h2>
<p style="text-align:center">
Youngseon Lee<sup>1</sup>, Shuhei Mano<sup>2</sup>, and Jaeyong Lee<sup>1</sup><br />
</p>
<p><span><strong>Bayesian curve fitting for discontinuous function using overcomplete representation with multiple kernels</strong></span></p>
<p>Youngseon Lee<span class="math inline">\(^1\)</span>, Shuhei Mano<span class="math inline">\(^2\)</span> and Jaeyong Lee<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, College of Natural Science, Seoul National University, 56-1 Mountain, Sillim-dong, Gwanak-gu, Seoul, Korea</p>
<p><span class="math inline">\(^2 \;\)</span> The Institute of Statistical Mathematics, 10-3 Midori-cho, Tachikawa, Tokyo 190-8562, Japan</p>
<p><span><strong>Abstract</strong></span>. We propose a new Bayesian methodology for estimating discontinuous functions. In this model, the estimated function is expressed by the overcomplete representation with multiple kernels. Therefore, the complex shape of functions can be expressed by the much smaller number of parameters due to the nature of the sparseness. It does not need any assumptions about the location of discontinuities, the smoothness of the function, the number of features. The form of the function taking all of these into account is determined naturally by the random Levy measure. Simulation data and real data analysis show that this model is suitable for fitting discontinuous functions. We also proved theoretical properties about the support of the function space having jumps in this paper.</p>
<p><span><strong>Keywords</strong></span>. Bayesian, nonparametric regression, discontinuous curve fitting, overcomplete, multiple kernel, Levy random field</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Chu, J. H., Clyde, M. A., and Liang, F. (2009). Bayesian function estimation using continuous wavelet dictionaries, <em>Statistica Sinica</em>, 1419â€“1438</p>
<p>Clyde, M. A., and Wolpert, R. L. (2007). Nonparametric function estimation using overcomplete dictionaries, <em>Bayesian Statistics</em>, <strong>8</strong>, 91â€“114.</p>
<p>Green, Peter J. (1995). Reversible jump Markov chain Monte Carlo computation and Bayesian model determination, <em>Biometrika</em>, <strong>82(4)</strong>, 711â€“732.</p>
<p>Khinchine, Alexander Ya and L<span>Ã©</span>vy, Paul (1936). Sur les lois stables, <em>CR Acad. Sci. Paris</em>, <strong>202</strong>, 374â€“376.</p>
<p>M<span>Ã¼</span>ller, P., and Quintana, F. A. (2004). Nonparametric Bayesian data analysis, <em>Statistical science</em>, 95â€“110</p>
<p>Pillai, N. S., Wu, Q., Liang, F., Mukherjee, S., and Wolpert, R. L. (2007). Characterizing the function space for Bayesian kernel models, <em>Journal of Machine Learning Research</em>, <strong>8</strong>, 1769â€“1797.</p>
<p>Qiu, Peihua (2011). <em>Jump Regression Analysis</em>. Springer.</p>
Wolpert, R. L., Clyde, M. A., and Tu, C. (2011). Stochastic expansions using continuous dictionaries: L<span>Ã©</span>vy adaptive regression kernels, The <em>Annals of Statistics</em>, 1916â€“1962.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 10:50 Case Room 2 (260-057)
</p>
</div>
<div id="estimation-of-a-semiparametric-spatiotemporal-models-with-mixed-frequency" class="section level2 unnumbered">
<h2>Estimation of a Semiparametric Spatiotemporal Models with Mixed Frequency</h2>
<p style="text-align:center">
Vladimir Malabanan, Erniel Barrios, and Joseph Ryan Lansangan<br /> <br />
</p>
<p>Estimation of a Semiparametric Spatiotemporal Models with Mixed Frequency</p>
<p>Vladimir A. Malabanan, Erniel B. Barrios, Joseph Ryan G. Lansangan School of Statistics, University of the Philippines Diliman Abstract</p>
<p>A semiparametric spatiotemporal model is postulated with data measured at varying frequency. The model optimizes utilization of information from variables measured at higher frequency by estimating its nonparametric effect on the response through the backfitting algorithm. Simulation studies support the optimality of the model over simple generalized additive model with aggregation of high frequency data. The method is then used in analyzing the spatiotemporal dynamics of corn yield based on some remotely-sensed data as covariates.</p>
Keywords: spatiotemporal model, semiparametric model, backfitting, mixed frequency
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 10:50 Case Room 3 (260-055)
</p>
</div>
<div id="lsmm-a-statistical-approach-to-integrating-functional-annotations-with-genome-wide-association-studies" class="section level2 unnumbered">
<h2>LSMM: A statistical approach to integrating functional annotations with genome-wide association studies</h2>
<p style="text-align:center">
Jingsi Ming<sup>1</sup>, Mingwei Dai<sup>2</sup>, Mingxuan Cai<sup>1</sup>, Xiang Wan<sup>1</sup>, Jin Liu<sup>3</sup>, and Can Yang<sup>4</sup><br />
</p>
<p><span><strong>LSMM: A statistical approach to integrating functional annotations with genome-wide association studies</strong></span></p>
<p>Jingsi Ming<span class="math inline">\(^1\)</span>, Mingwei Dai<span class="math inline">\(^{2,5}\)</span>, Mingxuan Cai<span class="math inline">\(^1\)</span>, Xiang Wan<span class="math inline">\(^3\)</span>, Jin Liu<span class="math inline">\(^4\)</span> and Can Yang<span class="math inline">\(^5\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Mathematics, Hong Kong Baptist University, Hong Kong</p>
<p><span class="math inline">\(^2 \;\)</span> School of Mathematics and Statistics, Xiâ€™an Jiaotong University, Xiâ€™an, China</p>
<p><span class="math inline">\(^3 \;\)</span> Department of Computer Science, Hong Kong Baptist University, Hong Kong</p>
<p><span class="math inline">\(^4 \;\)</span> Centre for Quantitative Medicine, Duke-NUS Medical School, Singapore</p>
<p><span class="math inline">\(^5 \;\)</span> Department of Mathematics, The Hong Kong University of Science and Technology, Hong Kong</p>
<span><strong>Abstract</strong></span>. Thousands of risk variants underlying complex phenotypes have been identified in genome-wide association studies (GWAS). However, there are two major challenges towards fully characterizing the biological basis of complex diseases. First, many complex traits are suggested to be highly polygenic, whereas a large proportion of risk variants with small effects remains unknown. Second, the functional roles of the majority of GWAS hits in the non-coding region is largely unclear. In this paper, we propose a latent sparse mixed model (LSMM) to address the challenges by integrating functional annotations with summary statistics from GWAS. An efficient variational expectation-maximization (EM) algorithm is developed. We conducted comprehensive simulation studies and then applied it to 30 GWAS of complex phenotypes integrating 9 genic annotation categories and 127 tissue-specific functional annotations from the Roadmap project. The results demonstrate that LSMM is not only able to increase the statistical power to identify risk variants, but also provide a deeper understanding of genetic architecture of complex traits by detecting relevant functional annotations. <span><strong>Keywords</strong></span>. GWAS, functional annotations, variational inference
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:10 OGGB4 (260-073)
</p>
</div>
<div id="estimation-of-a-high-dimensional-covariance-matrix" class="section level2 unnumbered">
<h2>Estimation of a High-Dimensional Covariance Matrix</h2>
<p style="text-align:center">
Xiangjie Xue and Yong Wang<br /> <br />
</p>
<p><span><strong>Estimation of a High-Dimensional Covariance Matrix</strong></span></p>
<p>Xiangjie Xue<span class="math inline">\(^*\)</span> and Yong Wang<span class="math inline">\(^*\)</span></p>
<p><span class="math inline">\(^* \;\)</span> Department of Statistics, The University of Auckland, New Zealand.</p>
<p><span><strong>Abstract</strong></span>. The estimation of covariance or precision (inverse covariance) matrices plays a prominent role in multivariate analysis. The usual estimator, the sample covariance matrix, is known to be unstable and ill-conditioned in high-dimensional setting. In the past two decades, various methods have been developed to give a stable and well-conditioned estimator and they have their own advantages and disadvantages. We will review some of the most popular methods and describe a new method to estimate the correlation matrix and hence the covariance matrix using the empirical Bayes method. Similar to many element-wise methods in the literature, we also assume that the elements in a correlation matrix are independent of each other. We use the fact that the elements in a sample correlation matrix can be approximated by the same one-parameter normal distribution with unknown means , along with the non-parametric maximum likelihood estimation to give a new estimator of the correlation matrix. Preliminary simulation results show that the new estimator has some advantages over various thresholding methods in estimating sparse covariance matrices.</p>
<p><span><strong>Keywords</strong></span>. Big Data, Multivariate Analysis, Statistical Inference</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Efron, B., 2010. <em>Correlated <span class="math inline">\(z\)</span>-values and the accuracy of large-scale statistical estimates</em>. J Am Stat Assoc <strong>105</strong>, 1042 - 1055.</p>
<p>Fan, J., Liao, Y., Liu, H., 2016. <em>An overview of the estimation of large covariance and precision matrices</em>. Econometrics Journal <strong>19</strong>, C1 - C32.</p>
Wang, Y., 2007. <em>On fast computation of the non-parametric maximum likelihood estimate of a mixing distribution</em>. Journal of the Royal Statistical Society: Series B <strong>69</strong>, 185 - 198.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:10 OGGB5 (260-051)
</p>
</div>
<div id="innovative-bayesian-estimation-in-the-von-mises-distribution" class="section level2 unnumbered">
<h2>Innovative Bayesian Estimation in the von-Mises Distribution</h2>
<p style="text-align:center">
Yuta Kamiya<sup>1</sup>, Toshinari Kamakura<sup>1</sup>, and Takemi Yanagimoto<sup>2</sup><br />
</p>
<p><span><strong>Innovative Bayesian Estimation in the von-Mises Distribution</strong></span></p>
<p>Yuta Kamiya<span class="math inline">\(^1\)</span> , Toshinari Kamakura<span class="math inline">\(^2\)</span> and Takemi Yanagimoto<span class="math inline">\(^3\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Graduate School of Industrial and Systems Engineering, Chuo University, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Industrial and Systems Engineering, Chuo University, Japan</p>
<p><span class="math inline">\(^3 \;\)</span> Institute of Statistical Mathematics, Japan</p>
<p><span><strong>Abstract</strong></span>. In spite of recent growing interest in applying the von-Mises distribution to circular data in various scientific fields, researches on the parameter estimation are surprisingly sparse. The standard estimators are the MLE and the maximum marginal likelihood estimator (Schou 1978). Although Bayesian estimators are promising, it looks that they have not been fully developed. We propose the posterior mean of the canonical parameter, instead of the mean parameter, under the reference prior. This estimator satisfies an optimality property, and performs favorably for wide ranges of true parameters. Extensive simulation studies yield that the risks of the proposed estimator are significantly small, compared with the existing estimators. An interesting finding is that the estimating function for the dispersion parameter behaves reasonably. Notable advantages of the present approach are its straightforward extensions to various procedures, including Bayesian estimator under an informative prior based on the reference prior. The proposed estimator is examined by applying to practical datasets.</p>
<p><span><strong>Keywords</strong></span>. von-Mises distribution, bayesian estimation, canonical parameter</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Fisher, Nicholas I. <span><em>Statistical analysis of circular data.</em></span> Cambridge University Press, 1995.</p>
Schou, Geert. â€œEstimation of the concentration parameter in von Misesâ€“Fisher distributions.â€ Biometrika 65.2 (1978): 369-377.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:10 Case Room 2 (260-057)
</p>
</div>
<div id="evidence-of-climate-change-from-nonparametric-change-point-analysis" class="section level2 unnumbered">
<h2>Evidence of Climate Change from Nonparametric Change-point Analysis</h2>
<p style="text-align:center">
Angela Nalica, Paolo Redondo, Erniel Barrios, and Stephen Villejo<br /> <br />
</p>
<p>Evidence of Climate Change from Nonparametric Change-point Analysis</p>
<p>Stephen Jun Villejo, Paolo Victor Redondo, Angela Nalica, Erniel Barrios University of the Philippines Diliman</p>
Abstract Suppose that the time series data is sufficiently explained by a model, e.g., autoregressive model, transfer function model. A change-point is considered to exist if any of the model parameters is substantially different in two or more regimes. We proposed a test for existence of a change-point (assuming that location of the change is known) based on nonparametric bootstrap. The method is used in verifying whether the southern oscillation index exhibits change-point which is taken as an evidence of climate change. There is indeed an evidence of climate change in the period. Keywords: change-point analysis, block bootstrap, southern oscillation index (SOI)
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:10 Case Room 3 (260-055)
</p>
</div>
<div id="joint-analysis-of-individual-level-genotype-data-and-summary-statistics-by-leveraging-pleiotropy" class="section level2 unnumbered">
<h2>Joint Analysis of Individual Level Genotype Data and Summary Statistics by Leveraging Pleiotropy</h2>
<p style="text-align:center">
Mingwei Dai<sup>1</sup>, Jin Liu<sup>2</sup>, and Can Yang<sup>3</sup><br />
</p>
<p><span><strong>Joint Analysis of Individual Level Genotype Data and Summary Statistics by Leveraging Pleiotropy</strong></span></p>
<p>Mingwei Dai<span class="math inline">\(^1\)</span>, Can Yang<span class="math inline">\(^2\)</span> and Jin Liu<span class="math inline">\(^3\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> School of Mathematics and Statistics, Xiâ€™an Jiaotong University, Xiâ€™an, China</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Mathematics, Hong Kong University of Science and Technology, Hong Kong</p>
<p><span class="math inline">\(^3 \;\)</span> Centre of Quantitative Medicine, Duke-NUS Medical School, Singapore</p>
<p><span><strong>Abstract</strong></span>. Results from Genome-wide association studies (GWAS) suggest that a complex phenotype is often affected by many variants with small effects, known as â€œpolygenicityâ€. Tens of thousands of samples are often required to ensure statistical power of identifying these variants with small effects. However, it is often the case that a research group can only get approval for the access to individual-level genotype data with a limited sample size (e.g., a few hundreds or thousands). Meanwhile, pleiotropy is a pervasive phenomenon in genetics whereby a DNA variant influences multiple traits, and summary statistics for genetically related traits (e.g., autoimmune diseases or psychiatric disorders) are becoming publicly available. The sample sizes associated with the summary statistics data sets are usually quite large. How to make the most efficient use of existing abundant data resources largely remains an open problem.</p>
<p>In this study, we propose a statistical approach, LEP, to increasing statistical power of identifying risk variants and improving accuracy of risk prediction by integrating individual level genotype data and summary statistics by veraging leiotropy. An efficient algorithm based on variational inference is developed to handle the genome-wide analysis. Through comprehensive simulation studies, we demonstrated the advantages of LEP over the methods which take either individual-level data or summary statistics data as input. We applied LEP to perform integrative analysis of several auto-immune diseases from WTCCC and summary statistics from other studies. LEP was able to significantly increase the statistical power of identifying risk variants and improve the risk prediction accuracy by jointly analyzing autoimmune diseases.</p>
<p><span><strong>Keywords</strong></span>. GWAS, pleiotropy, polygenicity, summary statistics, variational inference</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Solovieff N, Cotsapas C, Lee P H, et al. (2013) Pleiotropy in complex traits: challenges and strategies In: <em>Nature reviews. Genetics</em> 14(7): 483.</p>
<p>Carbonetto P, Stephens M. (2012) Scalable variational inference for Bayesian variable selection in regression, and its accuracy in genetic association studies In: <em>Bayesian analysis</em> 7(1): 73-108.</p>
<p>Chung D, Yang C, Li C, et al. (2014). GPA: a statistical approach to prioritizing GWAS results by integrating pleiotropy and annotation In: <em>PLoS genetics</em></p>
Dai M, Ming J, Cai M, et al. (2017). IGESS: a statistical approach to integrating individual-level genotype data and summary statistics in genome-wide association studies. In: <em>Bioinformatics</em>
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:30 OGGB4 (260-073)
</p>
</div>
<div id="genetic-map-estimation-using-hidden-markov-models-in-the-presence-of-partially-observed-information" class="section level2 unnumbered">
<h2>Genetic map estimation using hidden Markov models in the presence of partially observed information</h2>
<p style="text-align:center">
Timothy Bilton<sup>1,2</sup>, Matthew Schofield<sup>1</sup>, Ken Dodds<sup>2</sup>, and Michael Black<sup>1</sup><br />
</p>
<p><span><strong>Genetic map estimation using hidden Markov models in the presence of partially observed information</strong></span></p>
<p>Timothy P. Bilton<sup>1,2</sup>, Matthew R. Schofield<sup>1</sup>, Ken G. Dodds<sup>2</sup> and Michael A. Black<sup>3</sup><br />
<span class="math inline">\(^1 \;\)</span> Department of Mathematics and Statistics, University of Otago, Dunedin, New Zealand</p>
<p><span class="math inline">\(^2 \;\)</span> Invermay Agricultural Centre, AgResearch, Mosgiel, New Zealand</p>
<p><span class="math inline">\(^3 \;\)</span> Department of Biochemistry, University of Otago, Dunedin, New Zealand</p>
<p><span><strong>Abstract</strong></span>. A genetic linkage map shows the relative position of and genetic distance between markers, positions of the genome that exhibit variation, and underpins the study of speciesâ€™ genomes in a number of scientific applications. Genetic maps are constructed by tracking the transmission of genetic information from individuals to their offspring, which is frequently modelled using a hidden Markov model (HMM) since only the expression and not the transmission of genetic information is observed. However, constructing genetic maps with data generated using the latest sequencing technology is complicated by the fact that some observations are only partially observed which, if unaccounted for, typically results in inflated estimates. We extend the HMM to model partially observed information by including an additional layer of latent variables. In addition, we investigate several different approaches for computing confidence intervals of the genetic map estimates obtained from the extended HMM. Results show that our model is able to produce accurate genetic map estimates, even in situations where a large proportion of the data is only partially observed. Our methodology has been implemented in the R package GusMap.</p>
<span><strong>Keywords</strong></span>. hidden Markov models, linkage mapping, partially observed data, confidence intervals
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:30 OGGB5 (260-051)
</p>
</div>
<div id="a-simple-method-for-grouping-patients-based-on-historical-doses" class="section level2 unnumbered">
<h2>A simple method for grouping patients based on historical doses</h2>
<p style="text-align:center">
Shengli Tzeng<br /> <br />
</p>
<p><span><strong>A simple method for grouping patients based on historical doses</strong></span></p>
<p>ShengLi Tzeng<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Public Health, China Medical University, Taichung, 40402, Taiwan</p>
<p><span><strong>Abstract</strong></span></p>
<p>Monitoring dose patterns over time helps physicians and patients learn more about metabolic change, disease evolution, etc. One way to turn such longitudinal data into clinically useful information is through cluster analysis, which aims to separate the â€œprofiles of dosesâ€ among patients into homogeneous subgroups. Different doses patterns reflect heterogeneity in patientsâ€™ characteristics and effectiveness of therapy. However, not all patients were prescribed at regular time points, and missing values seems ubiquitous if one aligns records at distinct time points. Moreover, a few outliers may heavily influence the estimation for within and/or between variations of clusters, making the distinction among clusters blurred. In this study, a simple method based on a novel pairwise dissimilarity is proposed, which also serves as a screen tool to detect potential outliers. We use smoothing splines, handling data observed either at regular or irregular time points, and measure the dissimilarity between patients based on pairwise varying curve estimates with commutation of smoothing parameters. It takes into account the estimation uncertainty and is not strongly affected by outliers. The effectiveness of our proposal is shown by simulations comparing it to other dissimilarity measures and by a real application to methadone dosage maintenance levels.</p>
<p><span><strong>Keywords</strong></span>. Clustering, longitudinal data, smoothing splines, outliers</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Lin, Chien-Ju, Christian Hennig, and Chieh-Liang Huang. (2016). Clustering and a dissimilarity measure for methadone dosage time series. In <em>Analysis of Large and Complex Data</em>, 31-41. Springer, Switzerland.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:30 Case Room 2 (260-057)
</p>
</div>
<div id="semiparametric-mixed-analysis-of-covariance-model" class="section level2 unnumbered">
<h2>Semiparametric Mixed Analysis of Covariance Model</h2>
<p style="text-align:center">
Virgelio Alao, Erniel Barrios, and Joseph Ryan Lansangan<br /> <br />
</p>
<p>Semiparametric Mixed Analysis of Covariance Model</p>
<p>Virgelio M. Alao Visayas State University Erniel B. Barrios Joseph Ryan G. Lansangan University of the Philippines Diliman</p>
<p>ABSTRACT</p>
<p>A semiparametric mixed analysis of covariance model is postulated and estimated using the two procedures: based on an imbedded restricted maximum likelihood (REML) and nonparametric regression (smoothing splines) estimation into the backfitting framework (ARMS); and infusing bootstrap into the ARMS (B-ARMS). The heterogeneous effect of covariates across the groups is postulated to affect the response through a nonparametric function to mitigate overparameterization. Using simulation studies, we exhibited the capability of the postulated model (and estimation procedures) in increasing predictive ability and stabilizing variance components estimates even for small sample size and with minimal covariate effect, and regardless of whether the model is correctly specified or there is misspecification error.</p>
Keywords: mixed ANCOVA model, nonparametric regression, backfitting, bootstrap, random effects, variance components
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:30 Case Room 3 (260-055)
</p>
</div>
<div id="adaptive-false-discovery-rate-regression-with-application-in-integrative-analysis-of-large-scale-genomic-data" class="section level2 unnumbered">
<h2>Adaptive False Discovery Rate regression with application in integrative analysis of large-scale genomic data</h2>
<p style="text-align:center">
Can Yang<br /> <br />
</p>
<p><span><strong>Adaptive False Discovery Rate regression with application in integrative analysis of large-scale genomic data</strong></span></p>
<p>Can YANG<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Mathematics, The Hong Kong University of Science and Techonolgy, Clear Water Bay, Hong Kong.</p>
<p><span><strong>Abstract</strong></span>. Recent international projects, such as the Encyclopedia of DNA Elements (ENCODE) project, the Roadmap project and the Genotype-Tissue Expression (GTEx) project, have generated vast amounts of genomic annotation data, e.g., epigenome and transcriptome. There is great demanding of effective statistical approaches to integrate genomic annotations with the results from genome-wide association studies. In this talk, we introduce a statistical framework, named AdaFDR, for integrating multiple annotations to characterize functional roles of genetic variants that underlie human complex phenotypes. For a given phenotype, AdaFDR can adaptively incorporates relevant annotations for prioritization of genetic risk variants, allowing nonlinear effects among these annotations, such as interaction effects between genomic features. Specifically, we assume that the prior probability of a variant associated with the phenotype is a function of its annotations <span class="math inline">\(F(X)\)</span>, where <span class="math inline">\(X\)</span> is the collection of the annotation status and <span class="math inline">\(F(X)\)</span> is an ensemble of decision trees, i.e., <span class="math inline">\(F(X) = \sum_k f_k(X)\)</span> and <span class="math inline">\(f_k(X)\)</span> is a shallow decision tree. We have developed an efficient EM-Boosting algorithm for model fitting, where a shallow decision tree grows in a gradient-Boosting manner (Friedman J. 2001) at each EM-iteration. Our framework inherits the nice property of gradient boosted trees: (1) The gradient accent property of the Boosting algorithm naturally guarantees the convergence of our EM-Boosting algorithm. (2) Based on the fitted ensemble <span class="math inline">\(\hat{F}(X)\)</span>, we are able to rank the importance of annotations, measure the interaction among annotations and visualize the model via partial plots (Friedman J. 2008). Using AdaFDR, we performed integrative analysis of genome-wide association studies on human complex phenotypes and genome-wide annotation resources, e.g., Roadmap epigenome. The analysis results revealed interesting regulatory patterns of risk variants. These findings deepen our understanding of genetic architectures of complex phenotypes. The statistical framework developed here is also broadly applicable to many other areas for integrative analysis of rich data sets.</p>
<p><span><strong>Keywords</strong></span>. False Discovery Rate, integrative analysis, functional annotation, genomic data</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Friedman, Jerome H (2001). Greedy function approximation: a gradient boosting machine, <em>Annals of statistics</em>, <strong>29:5</strong>,1189â€“1232.</p>
Jerome H. Friedman and Bogdan E. Popescu (2008) Predictive Learning via Rule Ensembles <em>The Annals of Applied Statistics</em>, <strong>2:3</strong>, 916â€“954
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:50 OGGB4 (260-073)
</p>
</div>
<div id="vector-generalized-linear-time-series-models" class="section level2 unnumbered">
<h2>Vector Generalized Linear Time Series Models</h2>
<p style="text-align:center">
Victor Miranda and Thomas Yee<br /> <br />
</p>
<p><span><strong>Vector Generalized Linear Time Series Models</strong></span></p>
<p>Victor Miranda<span class="math inline">\(^1\)</span> and Thomas Yee<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, University of Auckland, Auckland, NZ.</p>
<p><span><strong>Abstract</strong></span>. Since the introduction of the ARMA class in the early 1970s many time series (TS) extensions have been proposed, e.g., vector ARMA and GARCH-type models for heteroscedasticity. The result has been a plethora of models having pockets of substructure but little overriding framework. In this talk we propose a class of TS models called Vector Generalized Linear Time Series Models (VGLTSM), which can be thought of as multivariate generalized linear models directed towards time series data. The crucial VGLM ideas are constraint matrices, vector responses and covariate-specific linear predictors, and estimation by iteratively reweighted least squares and Fisher scoring. The only addition to the VGLM framework is a log-likelihood that depends on past values. We show how several popular sub-classes of TS models are accommodated as special cases of VGLMs, as well as new work that broadens TS modelling even more. Algorithmic details of its implementation in , and properties such as stationarity, parameters depending on covariates, expected information matrices and cointegrated TS are surveyed.</p>
<p><span><strong>Keywords</strong></span>. VGLM, time series, Fisher scoring.</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Yee, T. W. (2015) <em>Vector Generalized Linear and Additive Models: With an Implementation in R.</em> New York, USA: Springer.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 11:50 OGGB5 (260-051)
</p>
</div>
<div id="local-canonical-correlation-analysis-for-multimodal-labeled-data" class="section level2 unnumbered">
<h2>Local Canonical Correlation Analysis for Multimodal Labeled Data</h2>
<p style="text-align:center">
Seigo Mizutani and Hiroshi Yadohisa<br /> <br />
</p>
<p><span><strong>Local Canonical Correlation Analysis for Multimodal Labeled Data</strong></span></p>
<p>Seigo Mizutani<span class="math inline">\(^1\)</span> and Hiroshi Yadohisa<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Graduate School of Culture and Information Science, Doshisha University, Kyoto, JAPAN</p>
<p><span class="math inline">\(^2 \;\)</span> Faculty of Culture and Information Science, Doshisha University, Kyoto, JAPAN</p>
<p><span><strong>Abstract</strong></span></p>
<p>In supervised learning, canonical correlation analysis (CCA) is widely used for dimension reduction problems. When using dimension reduction methods, researchers should always aim to preserve the data structure in a low dimensional space. However, if the obtained data are assumed to be multimodal labeled data, that is, each cluster can be subdivided into several latent clusters, CCA is rarely able to preserve the data structure in a low dimensional space.</p>
<p>In this study, we propose local CCA (LCCA) for multimodal labeled data. This method is based on local Fisher discriminant analysis (LFDA) (Sugiyama, 2007). We do not employ the same local covariance matrix of the explanatory variables as under LFDA, which uses a local between-group variance matrix and a local within-group variance matrix. Instead, in our proposed method, we use a covariance matrix of the explanatory variables as well as a weighted affinity matrix. The usefulness of LCCA in data visualization and clustering is then demonstrated by simulation studies.</p>
<p><span><strong>Keywords</strong></span>. Supervised learning, Dimension reduction, Local Fisher discriminant analysis (LFDA), Weighted affinity matrix</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Sugiyama, M. (2007). Dimensionality reduction of multimodal labeled data by local Fisher discriminant analysis. <em>Journal of Machine Learning Research</em>, <strong>8</strong>, 1027-1061.</p>
<p>Hastie, T. and Buja, A. and Tibshirani, R. (1995) Penalized discriminant analysis., 73-102.</p>
Hotelling, H. (1936). Relations between two sets of variates. <em>Biometrika</em>, <strong>28</strong>, 321-377.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:10 098 Lecture Theatre (260-098)
</p>
</div>
<div id="canonical-covariance-analysis-for-mixed-numerical-and-categorical-three-way-three-mode-data" class="section level2 unnumbered">
<h2>Canonical Covariance Analysis for Mixed Numerical and Categorical Three-way Three-mode Data</h2>
<p style="text-align:center">
Jun Tsuchida and Hiroshi Yadohisa<br /> <br />
</p>
<p><span><strong>Canonical Covariance Analysis for Mixed Numerical and Categorical Three-way Three-mode Data</strong></span></p>
<p>Jun Tsuchida<span class="math inline">\(^1\)</span> and Hiroshi Yadohisa<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Graduate School of Culture and Information Science, Doshisha University, 1-3 Tataramiyakodani, Kyotanave, Kyoto, Japan</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Culture and Information Science, Doshisha University, 1-3 Tataramiyakodani, Kyotanave, Kyoto, Japan</p>
<p><span><strong>Abstract</strong></span>. Three-mode three-way data (objects <span class="math inline">\(\times\)</span> variable <span class="math inline">\(\times\)</span> conditions) have been observed in many areas of research. For example, panel data often include values for the same objects and variables at different times. Given two three-mode three-way data sets, we often investigate two types of factors: common factors, which show the relationships between the two data sets, and unique factors, which represent the uniqueness of each data set. In light of this, canonical covariance analysis has been proposed. However, these datasets often have numerical and categorical variables simultaneously. Many multivariate methods for two three-mode thee-way data sets assume that the data has numerical variables only. To overcome this problem, we propose three-mode three-way canonical covariance analysis with numerical and categorical variables. We use an optimal scaling method (for example, Yong (1987)) for the quantification of categorical data because the values of a categorical variable could not be compared with the value of a numerical variable.</p>
<p><span><strong>Keywords</strong></span>. Alternative least squares, Dimensional reduction, Optimal scaling,Quantification method</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
Young, F. W. (1981). Quantitative analysis of qualitative data. <em>Psychometrika</em>, <strong>46</strong>, pp.Â 357â€“388
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:10 OGGB4 (260-073)
</p>
</div>
<div id="variable-selection-algorithms" class="section level2 unnumbered">
<h2>Variable Selection Algorithms</h2>
<p style="text-align:center">
Fangayo Li<sup>1</sup>, Christopher Triggs<sup>1</sup>, Bogdan Dumitrescu<sup>2</sup>, and Ciprian Giurcaneanu<sup>1</sup><br />
</p>
<p><span><strong>Variable Selection Algorithms</strong></span></p>
<p>Fangyao Li<span class="math inline">\(^1\)</span>, Christopher M. Triggs<span class="math inline">\(^1\)</span>, Bogdan Dumitrescu<span class="math inline">\(^2\)</span>, Ciprian Doru GiurcÄƒneanu<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, University of Auckland, Auckland 1142, New Zealand</p>
<p><span class="math inline">\(^2 \;\)</span> Department of Automatic Control and Computers, University Politehnica of Bucharest, 060042 Bucharest, Romania</p>
<p><span><strong>Abstract</strong></span>. The matching pursuit algorithm (MPA) is an efficient solution for high dimensional variable selection (BÃ¼hlmann and van de Geer, 2011). There is, however, no widely accepted stopping rule for MPA. (Li et al., 2017) have given novel stopping rules based on information theoretic criteria (ITC). All of these ITC are based on the degrees of freedom (df) of the hat matrix which maps the data vector to its estimate. We derive some properties of the hat matrix when MPA is used. These allow us to give an upper bound on the possible increase in df between successive MPA iterations. A simulation study with data generated from different models compares the mean integrated square error of the different ITC and cross validation (Sancetta, 2016).</p>
<p><span><strong>Keywords</strong></span>. Matching pursuit algorithm, degrees of freedom, hat matrix</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>A.Sancetta (2016). <em>Greedy algorithms for prediction</em>. Bernoulli, vol.Â 22, pp.Â 1227 - 1277.</p>
<p>P.BÃ¼hlmann and S.van de Geer (2011). <em>Statistics for high-dimensional data. Methods, theory and applications</em>. Springer Science &amp; Business Media.</p>
F.Li, C.Triggs, B.Dumitrescu, and C.D.GiurcÄƒneanu (2017). <em>On the number of iterations for the matching pursuit algorithm</em> . Proceedings of the 25th European Signal Processing Conference (EUSIPCO), pp.Â 191 - 195. (to appear)
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:10 OGGB5 (260-051)
</p>
</div>
<div id="estimating-causal-structures-for-continuous-and-discrete-variables" class="section level2 unnumbered">
<h2>Estimating causal structures for continuous and discrete variables</h2>
<p style="text-align:center">
Mako Yamayoshi and Hiroshi Yadohisa<br /> <br />
</p>
<p><span><strong>Estimating causal structures for continuous and discrete variables</strong> </span></p>
<p>Mako Yamayoshi<span class="math inline">\(^1\)</span> and Hiroshi Yadohisa<span class="math inline">\(^2\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Graduate school of Culture and Information Science, Doshisha University, Kyotanabe, Kyoto, JAPAN</p>
<p><span class="math inline">\(^2 \;\)</span> Faculty of Culture and Information Science, Doshisha University, Kyotanabe, Kyoto, JAPAN</p>
<p><span><strong>Abstract</strong></span>.</p>
<p>Structural equation models have been used extensively for continuous variable data to find causal structures. In such a framework, the Linear Non- Gaussian Acyclic Model (LiNGAM) could enable finding a whole causal model (Shimizu et al., 2006). However, in many desciplines, the data include both continuous and discrete variables. LiNGAM could fail to capture the actual causal relationship for such data because it handles both discrete and continuous variables as continuous. Therefore, it is necessary to improve the estimation method for causal structures in such conditions.</p>
<p>In this study, we propose a method to find causal structures for continuous and discrete variables. To overcome the problems of the existing method, we use the Link function. Using simulation studies, we show that the proposed method performs more efficiently for data that includes continuous and discrete variables.</p>
<p><span><strong>Keywords</strong></span>. Causal direction, Latent variables, Link function, SEM, LiNGAM</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Barnett, J.A., Payne, R.W. and Yarrow, D. (1990). <em>Yeasts: Characteristics and identification: Second Edition.</em> Cambridge: Cambridge University Press.</p>
<p>S. Shimizu, P.O. Hoyer, A. HyvÃ¤rinen, and A. Kerminen (2006). A linear non-Gaussian acyclic model for causal discovery. <em>The Journal of<br />
Machine Learning Research</em>, vol.Â 7, pp.Â 2003-2030.</p>
<p>(ed.) Barnett, V., Payne, R. and Steiner, R. (1995). <em>Agricultural Sustainability: Economic, Environmental and Statistical Considerations</em>. Chichester: Wiley.</p>
<p>Payne, R.W. (1997). <em>Algorithm AS314 Inversion of matrices Statistics</em>, <strong>46</strong>, 295â€“298.</p>
Payne, R.W. and Welham, S.J. (1990). A comparison of algorithms for combination of information in generally balanced designs. In: <em>COMPSTAT90 Proceedings in Computational Statistics</em>, 297â€“302. Heidelberg: Physica-Verlag.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:10 Case Room 2 (260-057)
</p>
</div>
<div id="incorporating-genetic-networks-into-case-control-association-studies-with-high-dimensional-dna-methylation-data" class="section level2 unnumbered">
<h2>Incorporating genetic networks into case-control association studies with high-dimensional DNA methylation data</h2>
<p style="text-align:center">
Hokeun Sun<br /> <br />
</p>
<p><span><strong>Incorporating genetic networks into case-control association studies with high-dimensional DNA methylation data</strong></span></p>
<p>Hokeun Sun<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, Pusan National University, Busan 46241, Korea</p>
<p><span><strong>Abstract</strong></span>. In human genetic association studies with high-dimensional microarray data, it has been well known that statistical methods utilizing prior biological network knowledge such as genetic pathways and signaling pathways can outperform other methods that ignore genetic network structures. In recent epigenetic research on case-control association studies, relatively many statistical methods have been proposed to identify cancer-related CpG sites and the corresponding genes from high-dimensional DNA methylation data. However, most of existing methods are not able to utilize genetic networks although methylation levels among linked genes in the networks tend to be highly correlated with each other. In this article, we propose new approach that combines independent component analysis with network-based regularization to identify outcome-related genes for analysis of high-dimensional DNA methylation data. The proposed approach first captures gene-level signals from multiple CpG sites using independent component analysis and then regularizes them to perform gene selection according to given biological network information. In simulation studies, we demonstrated that the proposed approach overwhelms other statistical methods that do not utilize genetic network information in terms of true positive selection. We also applied it to the 450K DNA methylation array data of the four breast invasive carcinoma cancer subtypes from The Cancer Genome Atlas (TCGA) project.</p>
<span><strong>Keywords</strong></span>. Independent component analysis, network-based regularization, genetic network, DNA methylation, high-dimensional data
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:10 Case Room 3 (260-055)
</p>
</div>
<div id="adaptive-model-checking-for-functional-single-index-models" class="section level2 unnumbered">
<h2>Adaptive Model Checking for Functional Single-Index Models</h2>
<p style="text-align:center">
Feifei Chen<sup>1</sup>, Qing Jiang<sup>2</sup>, and Zhenghui Feng<sup>3</sup><br />
</p>
<p><span><strong>Adaptive Model Checking for Functional Single-Index Models</strong></span></p>
<p>Feifei CHEN<span class="math inline">\(^1\)</span>, Qing JIANG<span class="math inline">\(^2\)</span>, and Zhenghui FENG<span class="math inline">\(^3\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> School of Statistics, The Renmin University, Beijing, China</p>
<p><span class="math inline">\(^2 \;\)</span> School of Statistics, Beijing Normal University, Beijing, China</p>
<p><span class="math inline">\(^3 \;\)</span> School of Economics and the Wang Yanan Institute for Studies in Economics, Xiamen University, Xiamen, China</p>
<p><span><strong>Abstract</strong></span>. In this paper, a model-adaptive test statistic is proposed to do model checking for functional single-index models. Dimension reduction methods are included to handle the curse of dimensionality. The test statistic consists of two parts: the first term is a naive one, and the second term is adaptive to the model as if the model were univariate. It is consistent and can detect local alternative at a fast rate. Monte Carlo method is used to find the critical value under null hypothesis. Simulation studies show the performance of our proposed test procedure.</p>
<span><strong>Keywords</strong></span>. Functional single-index models, dimension reduction, model checking
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:10 Case Room 4 (260-009)
</p>
</div>
<div id="testing-for-genetic-associations-in-arbitrarily-structured-populations" class="section level2 unnumbered">
<h2>Testing for genetic associations in arbitrarily structured populations</h2>
<p style="text-align:center">
Minsun Song<br /> <br />
</p>
<p><span><strong>Testing for genetic associations in arbitrarily structured populations</strong></span></p>
<p>Minsun Song<span class="math inline">\(^1\)</span></p>
<p><span class="math inline">\(^1 \;\)</span> Department of Statistics, Sookmyung Womenâ€™s University, Seoul, 04310, Korea</p>
<p><span><strong>Abstract</strong></span>.We present a new statistical test of association between a trait and genetic markers, which we theoretically and practically prove to be robust to arbitrarily complex population structure. The statistical test involves a set of parameters that can be directly estimated from large-scale genotyping data, such as those measured in genome-wide associations studies. We also derive a new set of methodologies, called a genotype-conditional association test, shown to provide accurate association tests in populations with complex structures, manifested in both the genetic and non-genetic contributions to the trait. We demonstrate the proposed method on a large simulation study and on the real data. Our proposed framework provides a substantially different approach to the problem from existing methods.</p>
<span><strong>Keywords</strong></span>. Genome-wide association studies, Latent variable, Population structure
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:30 098 Lecture Theatre (260-098)
</p>
</div>
<div id="on-optimal-group-testing-designs-prevalence-estimation-cost-considerations-and-dilution-effects" class="section level2 unnumbered">
<h2>On Optimal Group Testing Designs: Prevalence Estimation, Cost Considerations, and Dilution Effects</h2>
<p style="text-align:center">
Shih-Hao Huang<br /> <br />
</p>
<p><span><strong>On Optimal Group Testing Designs: Prevalence Estimation, Cost Considerations, and Dilution Effects</strong></span></p>
<p>Shih-Hao Huang</p>
<p>Institute of Statistical Science, Academia Sinica, Taipei 11529, Taiwan</p>
<p><span><strong>Abstract</strong></span>. Group testing has been used for decades to estimate the prevalence of a rare disease when samples from multiple subjects can be pooled and tested as a group. A group testing design is specified by the support points (distinct group sizes) and their corresponding frequencies. In this series of works, we construct locally optimal approximate designs for group testing with uncertain error rates, where the goal is to maximize the precision of the prevalence estimate. We also provide a guaranteed algorithm based on the approximate theory for constructing exact designs for practical use. Our simulated examples based on a Chlamydia study in the United States show that the proposed design outperforms competing designs, and its performance is quite stable to the working parameters. We then extend the framework to accommodate two features likely to be encountered in real-world studies. We develop optimal budgeted-constrained designs, where both subjects and tests incur costs, and the error rates of the the assay are linked to the group sizes, allowing dilution effects to reduce the test performance. (Work done jointly with M.-N. L. Huang, K. Shedden, and W. K. Wong.)</p>
<p><span><strong>Keywords</strong></span>. Budget-constrained design, dilution effect, <span class="math inline">\(D_s\)</span>-optimality, group testing, sensitivity, specificity</p>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<p>Huang, S.-H., Huang, M.-N. L., Shedden, K. and Wong, W. K. (in press). Optimal group testing designs for estimating prevalence with uncertain testing errors. <em>Journal of the Royal Statistical Society: Series B.</em> DOI: 10.1111/rssb.12223.</p>
Huang, S.-H., Huang, M.-N. L. and Shedden, K. (manuscript). Cost considerations for efficient group testing studies.
<p class="pagebreak">
</p>
<p style="background-color:#ccccff;text-align:center">
Monday 11<sup>th</sup> 14:30 OGGB4 (260-073)
</p>
</div>
<div id="the-use-of-bayesian-networks-in-grape-yield-prediction" class="section level2 unnumbered">
<h2>The Use of Bayesian Networks in Grape Yield Prediction</h2>
<p style="text-align:center">
Rory Ellis, Daniel Gerhard, and Elena Moltchanova<br /> <br />
</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="programme-at-a-glance.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="abstract.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["NZSA-IASC-Programme.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
